---
layout: talk_reveal3
title: Forecasting the Future of Flu
venue: Chicago, IL
event: Options IX
author: Trevor Bedford
---

<!--
Outline:
* Weather forecasting
  - Weather forecast analogy
* Vaccine updates and antigenic drift
  - Drifted clades quickly take over
  - Spindly tree and rapid clade growth indicates adaptive evolution
  - Rapid surveillence and analysis necessary
* nextflu
* Antigenic cartography and tree-based antigenic analysis
  - Model validation
  - Phylogenetic interpolation of titer and identification of missing clades
* Fitness models and forecasting
* How to identify high fitness viruses
  - Sequence mutations
  - Clade behavior
  - Antigenic evolution
* Projections and model validation
  - When does the forecast fail? Emerging clades.
* New mutations difficult
  - Interpolation vs extrapolation of genotype-phenotype
* The world today and projections
  - "Barring substantial changes in other clades, we predict the (HA1:171K, HA2:77V/155E) variant to dominate."
* Future improvements
  - Extend to other seasonal viruses
  - Extend to NA
  - Better phenotype: integration of neutralization, egg adaptation
  - Geographic origins of antigenic variants
* Looking past seasonal influenza
  - Ebola / Zika
  - Avian flu / swine flu
  - General surveillence tool
-->

<!-- Color ramp: ["#5097BA", "#5BA6A7", "#69B091", "#79B77D", "#8BBB6A", "#9EBE5A", "#B2BD4D", "#C4B945", "#D4B13F", "#DFA43B"] -->

<!--
<section data-background="#000000">
	<ul class="title" style="list-style-type:none">
		<li>Trevor Bedford, PhD</li>
		<li>Assistant Member</li>
		<li>Fred Hutchinson Cancer Research Center, Seattle, WA, USA</li>
	</ul>
	<p>&nbsp;<p>
	<h3 class="title">
		I have no financial relationships with commercial interests to disclose
	</h3>
	<h3 class="title">
		My presentation does not include discussion of off-label or investigational use
	</h3>
</section>
-->

<section data-background="#000000">
<!-- data-background-iframe="https://earth.nullschool.net/#current/wind/surface/level/orthographic=-112.41,39.56,1182" -->
	<h2 class="title">{{ page.title }}</h2>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<!--
<section data-background="/images/talks/giphy_possible.gif">
</section>
-->

<!--
<section>
	<h3>What would it look like to have forecast for flu?</h3>
</section>
-->

<section>
	<img class="stretch" data-src="/images/talks/forecasting_weather.png">
	<div class="citation">
		<a href="https://www.wunderground.com/cgi-bin/findweather/getForecast?query=chicago">Weather Underground</a>
	</div>
</section>

<section>
	<img class="stretch" data-src="/images/talks/forecasting_politics.png">
	<div class="citation">
		<a href="http://projects.fivethirtyeight.com/2016-election-forecast/">Five Thirty Eight</a>
	</div>
</section>

<section>
	<h3>Want to forecast the make up of the future flu population from the population that exists today</h3>
	<p>Real-time updates as new information rolls in</p>
</section>

<section>
	<h3>Population turnover (in H3N2) is extremely rapid</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<section>
	<h3>Clades emerge, die out and take over</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clades show rapid turnover</h3>
	<img class="stretch" data-src="/images/talks/prediction_combined_trajectories.png">
</section>

<section>
	<h3>Dynamics driven by antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>Drift variants emerge and rapidly take over in the virus population</h3>
	<br>
	<h3>This causes the side effect of evading existing vaccine formulations</h3>
</section>

<section>
	<h3>Drift necessitates vaccine updates</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates.png">
</section>

<section>
	<h3>Timely surveillance and rapid analysis essential to vaccine strain selection</h3>
</section>

<section>
	<h1>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h1>
	<p>Project to provide a real-time view of the evolving influenza population</p>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population</p>
	<p>All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch" data-src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate clade frequencies</li>
		<li>Infer antigenic phenotypes</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section>

	<section>
		<h3>Up-to-date analysis publicly available at:</h3>
		<h2>
			<a href="http://nextflu.org">
				<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
			</a>
		</h2>
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_1.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_2.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_3.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_4.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_5.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_6.png">
	</section>

</section>

<!-- Demo focusing on the emergence and spread of 3c2a and 3c3a viruses -->

<section data-background="#5097BA">
	<h2 class="title">Antigenic analysis</h2>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" data-src="/images/talks/hi_assay.png">
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" data-src="/images/talks/hi_cross_reactivity.jpg">
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" data-src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<div class="left" style="float:left;width:63%">
		<h3>Antigenic cartography compresses HI measurements into an interpretable diagram</h3>
	</div>
	<div class="right" style="float:right;width:33%">
		<img data-src="/images/talks/hi_smith_map.jpg">
	</div>
	<div class="citation">
		<a href="http://www.sciencemag.org/content/305/5682/371.abstract">Smith et al 2004</a>
	</div>
</section>

<section>
	<h3>Instead of a geometric model, we sought a phylogenetic model of HI titer data</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_example.png">
	<div class="citation">
		<a href="http://bedford.io/publications/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Identify phylogeny branches associated with drops in HI titer</h3>
	<img class="stretch" data-src="/images/talks/titer_model_with_table.png">
	<div class="citation">
		<a href="http://bedford.io/publications/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Model can be used to interpolate across tree and predict phenotype of untested viruses</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_model.png">
	<div class="citation">
		<a href="http://bedford.io/publications/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" data-src="/images/talks/titers_validation.png">
	<div class="citation">
		<a href="http://bedford.io/publications/neher-titers/">Neher et al 2016</a>
	</div>
</section>

<section>
	<h3>Recent HI data from London WHO Collaborating Center</h3>
</section>

<section>

	<section>
		<h3>Up-to-date analysis at:</h3>
		<h2>
			<a href="http://nextflu.org">
				<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
			</a>
		</h2>
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_7.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_8.png">
	</section>

	<section>
		<img class="stretch" data-src="/images/talks/nextflu_aug2016_9.png">
	</section>

</section>

<!-- Demo of titer tool using H3N2 / 12y. Look at measurements and antigenic clusters surrounding vaccine strains. -->

<section data-background="#69B091">
	<h2 class="title">Forecasting</h2>
</section>

<section>
	<h2><p><q>"The future is here, it's just not evenly <br> distributed yet"</q> &mdash; William Gibson</p></h2>
</section>

<section data-background="/images/talks/giphy_mnemonic.gif">
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" data-src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" data-src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" data-src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>The question of forecasting becomes: how do we accurately estimate fitnesses of circulating viruses?</h3>
</section>

<section>
	<h3>Fortunately, there's lots of training data and previously successful strains have had:</h3>
	<br>
	<ol>
		<li class="larger">Amino acid changes at epitope sites</li>
		<li class="larger">Antigenic novelty based on HI</li>
		<li class="larger">Rapid phylogenetic growth</li>
	</ol>
</section>

<!--
<section>
	<h3>"Trunk" branches show an excess of non-synonymous change at epitope sites</h3>
	<img class="stretch" data-src="/images/talks/prediction_mutation_table.png">
	<div class="citation">
		<a href="https://biologydirect.biomedcentral.com/articles/10.1186/1745-6150-1-34">
			Wolf et al. 2006
		</a>
	</div>
</section>
-->

<!--
<section>
	<h3>"Trunk" branches show an excess of non-synonymous change at epitope sites</h3>
	<img class="stretch" data-src="/images/talks/prediction_mutation_tree.png">
	<div class="citation">
		<a href="http://www.genetics.org/content/192/2/671">
			Strelkowa and L&auml;ssig 2012
		</a>
	</div>
</section>
-->

<!-- REMAKE FIGURE FROM NEW TITER TOOL -->
<!--
<section>
	<h3>"Trunk" is antigenically advanced relative to the population average</h3>
	<img class="stretch" data-src="/images/talks/bmds_h3n2_diffusion_drift.png">
	<div class="citation">
		<a href="/publications/bedford-flux/">
			Bedford et al. 2014
		</a>
	</div>
</section>
-->

<section>
	<h3>Predictor: calculate HI drop from ancestor, <br> drifted clades have high fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_antigenic_drift.png">
</section>

<!--
<section>
	<h3>Branching rate gives an estimate of viral fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_neher.jpg">
	<div class="citation">
		<a href="http://elifesciences.org/content/elife/3/e03568">Neher et al 2014</a>
	</div>
</section>

<section data-transition="slide-in fade-out">
	<h3>"Local branching index" correlates with fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_lbi_3c2a.png">
</section>

<section data-transition="fade-in slide-out">
	<h3>"Local branching index" correlates with fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_lbi_171k.png">
</section>
-->

<section>
	<h3>Predictor: project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_frequency_trajectories.png">
</section>

<section>
	<h3>We predict fitness based on a simple formula</h3>
	<br>
	<p>where the fitness $f$ of virus $i$ is estimated as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{HI} \, f_i^\mathrm{HI} + \beta^\mathrm{freq} \, f_i^\mathrm{freq}$$</p>
	<p>where $f_i^\mathrm{HI}$ measures antigenic drift via HI and $f_i^\mathrm{freq}$ measures clade growth/decline</p>
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>

<!--
<section>
	<h3>Validation of 1 year clade frequency change</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_trajectories.png">
</section>
-->

<section>
	<h3>Clade growth rate is well predicted (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section data-transition="slide-in zoom-out" data-transition-speed="slow">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_grid.png">
</section>

<section data-transition="zoom-in slide-out">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_zoom.png">
</section>

<section>
	<h3>This model is similar in formulation and performance to &#321;uksza and L&auml;ssig</h3>
	<!--
	<br>
	<p>This model estimates the fitness $f$ of virus $i$ as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{ep} \, f_i^\mathrm{ep} + \beta^\mathrm{ne} \, f_i^\mathrm{ne}$$</p>
	<p>where $f_i^\mathrm{ep}$ measures cross-immunity via substitutions at epitope sites and $f_i^\mathrm{ep}$ measures mutational load via substitutions at non-epitope sites.</p>
	-->
	<img class="stretch" data-src="/images/talks/prediction_ll.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<!--
<section>
	<h3>We find that HI is preferred over epitope mutations</h3>
	<br>
	<table>
		<thead>
			<tr class="smaller">
				<th width="24%">Model</th>
				<th width="19%">Ep coefficient</th>
				<th width="19%">HI coefficient</th>
				<th width="19%">Freq error</th>
				<th width="19%">Growth corr</th>
			</tr>
		</thead>
		<tbody>
			<tr>
				<td>Epitope only</td>
				<td>2.36</td>
				<td>--</td>
				<td>0.10</td>
				<td>0.57</td>
			</tr>
			<tr>
				<td>HI only</td>
				<td>--</td>
				<td>2.05</td>
				<td>0.08</td>
				<td>0.63</td>
			</tr>
			<tr>
				<td>Epitope + HI</td>
				<td>-0.11</td>
				<td>2.15</td>
				<td>0.08</td>
				<td>0.67</td>
			</tr>
		</tbody>
	</table>
</section>
-->

<section>
	<h3>When does the forecast fail?</h3>
	<br>
	<p>Emerging clades are difficult to forecast: little antigenic data and little evidence of "past performance"</p>
	<p>Models work well for clades at >10%, but less well for clades <5%</p>
</section>

<section>
	<h3>New mutations difficult</h3>
	<br>
	<p>Models can project forward from circulating strains, but cannot foresee the appearance of new mutations</p>
	<p>Intrinsically limits the timescale of forecasting to ~1 year</p>
</section>

<section>
	<h3>Model is only as good as the data</h3>
	<br>
	<p>Requires rapid shipping of samples, rapid sequencing and rapid antigenic characterization</p>
</section>

<section data-background="#8BBB6A">
	<h2 class="title">Current projections</h2>
</section>

<section>
	<h3>Issuing reports online in Feb and Sep</h3>
	<img class="stretch" data-src="/images/talks/prediction_aug2016_reports.png">
</section>

<section>
	<p>In February we stated</p>
	<h3><p><q>"Barring substantial changes in other clades, we predict the (HA1:171K, HA2:77V/155E) variant to dominate"</q></p></h3>
	<p>Let's see how we did</p>
	<div class="citation">
		<a href="http://nextflu.org/reports/feb-2016/">Bedford and Neher 2016</a>
	</div>
</section>

<section>
	<h3>The (H3N2) world today</h3>
	<img class="stretch" data-src="/images/talks/prediction_aug2016_clade_tree.png">
</section>

<section>
	<h3>3c2.a viruses continue to predominate (except in the USA)</h3>
	<img class="stretch" data-src="/images/talks/prediction_aug2016_clade_freq.png">
</section>

<section>
	<h3>Within 3c2.a clades are emerging, in particular (HA1:171K, HA2:77V/155E)</h3>
	<img class="stretch" data-src="/images/talks/prediction_aug2016_mut_tree.png">
</section>

<section>
	<h3>The 171K clade has recently risen in frequency</h3>
	<img class="stretch" data-src="/images/talks/prediction_aug2016_mut_freq.png">
</section>

<section>
	<h3>We predict the 171K clade will continue to be successful (unless supplanted by a novel mutant)</h3>
	<br>
	<ul>
		<li class="larger">Has an amino acid change at predicted epitope site</li>
		<li class="larger">Has some evidence of antigenic novelty based on HI</li>
		<li class="larger">Shows recent rapid expansion</li>
	</ul>
</section>

<section data-background="#B2BD4D">
	<h2 class="title">Looking forward</h2>
</section>

<section>
	<h3>Further improvements to predictive modeling</h3>
	<br>
	<ol>
		<li>Extend to other seasonal viruses</li>
		<li>Forecast NA evolution</li>
		<li>Integrate neutralization (FRA) assay data</li>
		<li>Model effects of egg adaptation</li>
		<li>Incorporate an explicit geographic model</li>
	</ol>
</section>

<section>
	<h3>Phylogeny of H3 with geographic history</h3>
	<img class="stretch" data-src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/publications/bedford-global-circulation/">Bedford et al 2015</a>
	</div>
</section>

<section>
	<h3>Geographic location of phylogeny trunk</h3>
	<img class="stretch" data-src="/images/talks/flu_geo_h3_trunk.png">
	<div class="citation">
		<a href="http://bedford.io/publications/bedford-global-circulation/">Bedford et al 2015</a>
	</div>
</section>

<section>
	<h3>More generally real-time analyses may be useful for other viruses</h3>
</section>

<section>
	<h3>Ebola at <a href="http://ebola.nextstrain.org/"><span class="text-gray">ebola.nextstrain.org</span></a></h3>
	<img class="stretch" data-src="/images/talks/nextstrain_ebola.png">
</section>

<section>
	<h3>Zika at <a href="http://nextstrain.org/zika/"><span class="text-gray">nextstrain.org/zika/</span></a></h3>
	<img class="stretch" data-src="/images/talks/nextstrain_zika.png">
</section>

<section>
	<h3>Major opportunity to track evolution non-human influenza viruses</h3>
</section>

<section>
	<h3>Adaptation to avian flu viruses by Yujia Zhou and Justin Bahl</h3>
	<img class="stretch" data-src="/images/talks/nextstrain_h9.png">
</section>

<section>
	<h3>All tools are completely open source and we encourage other groups to get involved and push the project forward</h3>
	<img class="stretch" data-src="/images/talks/github_nextflu.png">
	<div class="citation">
		<a href="https://github.com/blab/nextflu">github.com/blab/nextflu</a>
	</div>
</section>

<section data-background="#D4B13F">
	<h2 class="title">General purpose genomic surveillance tool</h2>
</section>

<section>
	<h3>Acknowledgements</h3>
	<p class="smaller">Analysis: Richard Neher, Colin Russell, Charlton Callender, Colin Megill, Andrew Rambaut,
		Charles Cheung, Marc Suchard, Steven Riley, Philippe Lemey, Gytis Dudas, Boris Shraiman,
		Marta &#321;uksza, Michael L&auml;ssig</p>
	<p class="smaller">GISRS: Ian Barr, Shobha Broor, Mandeep Chadha, Nancy Cox, Rod Daniels, Becky Garten,
		Palani Gunasekaran, Aeron Hurt, Anne Kelso, Jackie Katz, Nicola Lewis, Xiyan Li, John McCauley,
		Takato Odagiri, Varsha Potdar, Yuelong Shu, Eugene Skepner, Masato Tashiro, Dayan Wang, Dave Wentworth,
		Xiyan Xu</p>
	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="40%" data-src="/images/talks/logo_fred_hutch.png">
		&nbsp;
		<img width="13%" data-src="/images/talks/logo_midas.jpg">
		&nbsp; &nbsp;
		<img width="13%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="13%" data-src="/images/talks/logo_nih.jpg">
	</div>
</section>

<section data-background="#000000">
	<h3 class="title">Contact</h3>
	<br>
	<p>
	<ul class="title">
		<li>Website: bedford.io</li>
		<li>Twitter: @trvrb</li>
		<li>Slides: bedford.io{{ page.url }}</li>
	</ul>
</section>
