---
layout: talk
title: Real-time tracking of virus evolution
venue: Genome Sciences, University of Washington
event: Combi Seminar
author: Trevor Bedford
---

<!-- Color ramp: ["#559EB1", "#6DB28B", "#8CBB68", "#AFBD4F", "#CFB541", "#E39B39", "#E56E30"] -->

<section data-background="#000000">
	<h2 class="title">{{ page.title }}</h2>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section data-background="#000000">
	<h3 class="title">Slides at bedford.io/talks/</h3>
</section>

<section data-background="#559EB1">
	<h2 class="title">Phylogenies</h2>
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_tudors.jpg">
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_haeckel.jpg">
	<div class="citation">
		Haeckel 1879
	</div>
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_pace.png">
	<div class="citation">
		<a href="http://www.sciencemag.org/content/276/5313/734.short">Pace 1997</a>
	</div>
</section>

<section>
	<h3>Phylogenies reveal process</h3>
	<img class="stretch" src="/images/talks/phylogenies_darwin.png">
	<div class="citation">
		Darwin 1859
	</div>
</section>

<section data-background="#6DB28B">
	<h2 class="title">Epidemics</h2>
</section>

<section>
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4.png">
</section>

<section>
	<h3>Middle Eastern MERS-CoV phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_mers.jpg">
	<div class="citation">
		<a href="http://www.thelancet.com/journals/lancet/article/PIIS0140-6736(13)61887-5/abstract">Cotten et al 2013</a>
	</div>
</section>

<section>
	<h3>West African Ebola phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_ebola.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>Global influenza phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al 2015</a>
	</div>
</section>

<section>
	<h3>Applications of evolutionary analysis for influenza vaccine strain selection and charting outbreak spread</h3>
</section>

<section data-background="#8CBB68">
	<h2 class="title">Influenza</h2>
</section>

<section>
	<h3>Influenza virion</h3>
	<img class="stretch" src="/images/talks/flu_virion.jpg">
</section>

<section>
	<h3>Influenza H3N2 vaccine updates</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_updates.png">
</section>

<section>
	<h3>H3N2 phylogeny showing antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>H3N2 phylogeny showing antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<section data-background="#559EB1">
	<h3 class="title">Drift variants rapidly take over the virus population</h3>
	<br>
	<h3 class="title">Timely surveillance and rapid analysis essential to understand ongoing influenza evolution</h3>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population
	<p class="fragment" data-fragment-index="1">All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch fragment" data-fragment-index="1" src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate frequencies</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section>
	<h3>Up-to-date analysis publicly available at:</h3>
	<h2>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section>

<section data-background="#AFBD4F">
	<h2 class="title">Antigenic evolution</h2>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" src="/images/talks/hi_assay.png">
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" src="/images/talks/hi_cross_reactivity.jpg">
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<h3>Fit HI titer drops to phylogeny branches</h3>
	<img class="stretch" src="/images/talks/titer_model_with_table.png">
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" src="/images/talks/titers_validation.png">
</section>

<!--
<section>
	<h3>Broad patterns agree with cartographic analyses</h3>
	<img class="stretch" src="/images/talks/titers_h3n2_drift_comparison.png">
</section>
-->

<section>
	<h3>Recent HI data from WHO CC London annual and interim reports</h3>
</section>

<section>
	<h3>Up-to-date analysis at:</h3>
	<h2>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section>

<section data-background="#CFB541">
	<h2 class="title">Forecasting</h2>
</section>

<section>
	<h3><p><q>The future is here, it's just not evenly distributed yet</q> &mdash; William Gibson</p></h3>
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<div style="vertical-align: bottom; display: table-cell;">
		<img width="49%" src="/images/talks/prediction_frequency_start.png">
		&nbsp;
		<img class="fragment" width="49%" src="/images/talks/prediction_frequency_stop.png">
	</div>
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>Predictive fitness models</h3>
	<br>
	<p>A simple predictive model estimates the fitness $f$ of virus $i$ as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{ep} \, f_i^\mathrm{ep} + \beta^\mathrm{ne} \, f_i^\mathrm{ne}$$</p>
	<p>where $f_i^\mathrm{ep}$ measures cross-immunity via substitutions at epitope sites and $f_i^\mathrm{ep}$ measures mutational load via substitutions at non-epitope sites</p>
	<img class="stretch" src="/images/talks/prediction_ll.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>We implement a similar model based on two predictors</h3>
	<br>
	<ol>
		<li class="larger">Clade frequency change</li>
		<li class="larger">Antigenic advancement</li>
	</ol>
</section>

<!--
<section>
	<h3><q>The best predictor of future behavior is past behavior</q></h3>
</section>
-->

<section>
	<h3>Project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_frequency_trajectories.png">
</section>

<section>
	<h3>Calculate HI drop from ancestor, <br> drifted clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_antigenic_drift.png">
</section>

<section>
	<h3>Fitness model parameterization</h3>
	<br>
	<p>Our predictive model estimates the fitness $f$ of virus $i$ as</p>
	<br>
	<p>$$\hat{f}_i = \beta^\mathrm{freq} \, f_i^\mathrm{freq} + \beta^\mathrm{HI} \, f_i^\mathrm{HI}$$</p>
	<br>
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>

<!--
<section>
	<h3>Validation of 1 year clade frequency change</h3>
	<img class="stretch" src="/images/talks/prediction_validation_trajectories.png">
</section>
-->

<section>
	<h3>Clade growth rate is well predicted</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 83% of cases</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section>
	<h3>Clade error increases steadily over time</h3>
	<img class="stretch" src="/images/talks/prediction_trajectories_envelope.png">
</section>

<section>
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" src="/images/talks/prediction_trajectories_grid.png">
</section>

<section>
	<h3>Formalizes intuition about drivers of influenza dynamics</h3>
	<br>
	<table>
		<thead>
			<tr class="smaller">
				<th width="24%">Model</th>
				<th width="19%">Ep coefficient</th>
				<th width="19%">HI coefficient</th>
				<th width="19%">Freq error</th>
				<th width="19%">Growth corr</th>
			</tr>
		</thead>
		<tbody>
			<tr>
				<td>Epitope only</td>
				<td>2.36</td>
				<td>--</td>
				<td>0.10</td>
				<td>0.57</td>
			</tr>
			<tr>
				<td>HI only</td>
				<td>--</td>
				<td>2.05</td>
				<td>0.08</td>
				<td>0.63</td>
			</tr>
			<tr>
				<td>Epitope + HI</td>
				<td>-0.11</td>
				<td>2.15</td>
				<td>0.08</td>
				<td>0.67</td>
			</tr>
		</tbody>
	</table>
</section>

<!--
<section>
	<h3>Prototype up at</h3>
	<h2>
		<a href="http://dev.nextflu.org">
			<span class="text-gray">dev.</span><span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section>
-->

<section>
	<h3>Further work on predictive modeling</h3>
	<br>
	<ol>
		<li>Integrate data predictors and data sources, e.g. plan to investigate a geographic predictor</li>
		<li>Possible to build predictive models for H1N1 and B and to forecast NA evolution</li>
	</ol>
</section>

<section>
	<h3>Evolutionary analyses can inform influenza vaccine strain selection</h3>
	<br>
	<p>Analyses must be rapid and widely available</p>
	<p>Predictive models can flag clades for experimental follow-up and creation of vaccine candidates</p>
</section>

<section data-background="#E39B39">
	<h2 class="title">Ebola</h2>
</section>

<section>
	<h3>Epidemic nearly contained, but resulted in >28,000 confirmed cases and >11,000 deaths</h3>
	<img class="stretch" src="/images/talks/ebola_timeseries.png">
</section>

<section>
	<h3>Outbreaks are independent spillovers from the animal reservoir</h3>
	<img class="stretch" src="/images/talks/ebola_between_outbreak_tree.png">
	<div class="citation">
		<a href="https://www.sciencemag.org/content/345/6202/1369.full">Gire et al 2014</a>
	</div>
</section>

<section>
	<h3>Person-to-person spread in the early West African outbreak</h3>
	<img class="stretch" src="/images/talks/ebola_early_outbreak_tree.png">
	<div class="citation">
		<a href="https://www.sciencemag.org/content/345/6202/1369.full">Gire et al 2014</a>
	</div>
</section>

<section>
	<h3>Continued spread through Dec 2014</h3>
	<img class="stretch" src="/images/talks/ebola_mid_outbreak_mst.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>At epidemic height, geographic spread of particular interest</h3>
	<img class="stretch" src="/images/talks/ebola_mid_outbreak_geo_tree.png">
	<div class="citation">
		Rambaut 2015
	</div>
</section>

<section>
	<h3>Later on, tracking transmission clusters of primary importance</h3>
	<img class="stretch" src="/images/talks/ebola_late_outbreak_tree.png">
	<div class="citation">
		<a href="http://virological.org/t/update-on-ebola-virus-circulating-in-sierra-leone-and-guinea/151">Rambaut 2015</a>
	</div>
</section>

<section>
	<h3>Tracking epidemic spread in real-time:</h3>
	<h2>
		<a href="http://ebola.nextstrain.org">
			<span class="text-gray">ebola.nextstrain.org</span>
		</a>
	</h2>
</section>

<section data-background="#E56E30">
	<h3 class="title">Middle East respiratory syndrome coronavirus (MERS-CoV)</h3>
</section>

<section>
	<h3>Cases concentrated in the Arabian Peninsula with occasional exports</h3>
	<img class="stretch" src="/images/talks/mers_map.jpg">
	<div class="citation">
		<a href="http://epidemic.bio.ed.ac.uk/coronavirus_background">Rambaut 2015</a>
	</div>
</section>

<section>
	<h3>No evidence of epidemic growth, spill-over transmission clusters</h3>
	<img class="stretch" src="/images/talks/mers_timeseries.png">
	<div class="citation">
		<a href="http://epidemic.bio.ed.ac.uk/coronavirus_background">Rambaut 2015</a>
	</div>
</section>

<section data-background="/images/talks/mers_camels.jpg">
</section>

<section>
	<h2>Bats &#10142; Camels &#10142; Humans</h2>
</section>

<section>
	<h3>Tracking spillover events in real-time:</h3>
	<h2>
		<a href="http://mers.nextstrain.org">
			<span class="text-gray">mers.nextstrain.org</span>
		</a>
	</h2>
</section>

<section>
	<h3>Moving forward, genetically-informed outbreak response requires:</h3>
	<br>
	<p>
	<ul>
		<li>Rapid sharing of sequence data, <i>genetic context critical</i></li>
		<li>Technologies to rapidly conduct phylogenetic inference</li>
		<li>Technologies to explore genetic relationships and inform epidemiological investigation</li>
	</ul>
</section>

<section>
	<h3>Acknowledgements</h3>
	<br>
	<p class="smaller">Richard Neher (Max Planck T&uuml;bingen), Andrew Rambaut (University of Edinburgh),
		Colin Russell (Cambridge University), Michael L&auml;ssig (University of Cologne),
		Marta &#321;uksza (Institute for Advanced Study), Gytis Dudas (University of Edinburgh),
		Pardis Sabeti (Harvard University), Danny Park (Harvard University), Nick Loman (University of Birmingham)
		Matthew Cotten (Sanger Institute), Paul Kellam (Sanger Institute),
		WHO Global Influenza Surveillance Network, GISAID
	</p>
	<div style="vertical-align: bottom; display: table-cell;">
		<img width="33%" src="/images/talks/logo_royal_society.jpg">
		&nbsp;
		<img width="33%" src="/images/talks/logo_fred_hutch.png">
		&nbsp;
		<img width="11%" src="/images/talks/logo_midas.jpg">
	</div>
</section>

<section data-background="#000000">
	<h3 class="title">Contact</h3>
	<br>
	<p>
	<ul class="title">
		<li>Website: bedford.io</li>
		<li>Twitter: @trvrb</li>
		<li>Slides: bedford.io{{ page.url }}</li>
	</ul>
</section>
