---
layout: talk
title: Tracking influenza antigenic evolution and geographic circulation
venue: Korea University College of Medicine
event: Microbiology Department Seminar
author: Trevor Bedford
---

<!-- Standard 1hr total talk length. Aim for 45 min + questions. -->
<!-- Color ramp: ["#559EB1", "#6DB28B", "#8CBB68", "#AFBD4F", "#CFB541", "#E39B39", "#E56E30"] -->

<section data-background="#000000">
	<h2 class="title">{{ page.title }}</h2>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section data-background="#000000">
	<h3 class="title">Slides at bedford.io/talks/</h3>
</section>

<section>
	<h3>Influenza virus</h3>
	<img class="stretch" src="/images/talks/flu_virion.jpg">
</section>

<section>
	<h3>Influenza H3N2 vaccine updates</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_updates.png">
</section>

<section>
	<h3>Gene flow and population turnover</h3>
	<img class="stretch" src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Turnover can be seen in the flu phylogeny</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<section>
	<h3>Flu pandemics caused by host switch events</h3>
	<img class="stretch" src="/images/talks/flu_pandemics.png">
</section>

<section>
	<h3>Influenza B does not have pandemic potential</h3>
	<img class="stretch" src="/images/talks/flu_pandemics_b.png">
</section>

<section>
	<h3>Phylogenetic trees of different influenza lineages</h3>
	<img class="stretch" src="/images/talks/flu_trees.png">
</section>

<section>
	<h3>Antigenic evolution drives viral dynamics</h3>
	<img class="stretch" src="/images/talks/canalization_dynamics.jpg">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-canalization/">Bedford et al 2012</a>
	</div>
</section>

<section data-background="#559EB1">
	<h2 class="title">Antigenic drift</h2>
	<p class="title">with Andrew Rambaut, Marc Suchard and others</p>
	<br>
	<p class="smaller title">
		<a class="title" href="http://bedford.io/papers/bedford-flux/">
			Bedford et al 2014. Integrating influenza antigenic dynamics <br> with molecular evolution. eLife.
		</a>
	</p>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" src="/images/talks/hi_assay.png">
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" src="/images/talks/hi_cross_reactivity.jpg">
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<h3>Compiled HI data difficult to work with</h3>
	<img class="stretch" src="/images/talks/hi_full_table.png">
</section>

<section>
	<h3>Antigenic cartography positions viruses and sera to recapitulate titer values</h3>
	<div style="vertical-align: bottom; display: table-cell;">
		<img width="40%" src="/images/talks/hi_toy_distances.png">
		&nbsp;
		<img width="40%" src="/images/talks/hi_full_table.png">
	</div>
	<div class="citation">
		<a href="http://www.sciencemag.org/content/305/5682/371.abstract">Smith et al 2004</a>
	</div>
</section>

<section data-transition="none">
	<h3>Antigenic cartography positions viruses and sera to recapitulate titer values</h3>
	<div style="vertical-align: bottom; display: table-cell;">
		<img width="40%" src="/images/talks/hi_toy_map.png">
		&nbsp;
		<img width="20%" src="/images/talks/hi_smith_map.jpg">
	</div>
	<div class="citation">
		<a href="http://www.sciencemag.org/content/305/5682/371.abstract">Smith et al 2004</a>
	</div>
</section>

<section>
	<h3>Schematic HI table and antigenic map</h3>
	<img class="stretch" src="/images/talks/hi_schematic.png">
</section>

<section>
	<h3>Bayesian multidimensional scaling (BMDS)</h3>
	<img class="stretch" src="/images/talks/bmds_likelihood.png">
</section>

<section>
	<h3>Integration through Markov chain Monte Carlo (MCMC)</h3>
	<img class="stretch" src="/images/talks/beast_mcmc.jpg">
</section>

<section>
	<h3>Antigenic map of H3N2 influenza from 1968 to 2011</h3>
	<img class="stretch" src="/images/talks/bmds_h3n2_map.jpg">
</section>

<section>
	<h3>Antigenic drift of H3N2 influenza</h3>
	<img class="stretch" src="/images/talks/bmds_h3n2_drift.jpg">
</section>

<section>
	<h3>Factoring the BMDS model</h3>
	<br>
	<img class="stretch" src="/images/talks/bmds_basic_model.png">
</section>

<section>
	<h3>Phylogeny of H3N2 virus sequences</h3>
	<img class="stretch" src="/images/talks/bmds_h3n2_tree.png">
</section>

<section>
	<h3>Modeling continuous characters via Brownian motion</h3>
	<img class="stretch" src="/images/talks/beast_continuous_trait.jpg">
	<div class="citation">
		<a href="http://mbe.oxfordjournals.org/content/27/8/1877.short">
			Lemey et al 2010
		</a>
	</div>
</section>

<section>
	<h3>Including diffusion in MDS model</h3>
	<img class="stretch" src="/images/talks/bmds_h3n2_map.jpg">
	<img class="stretch" src="/images/talks/bmds_h3n2_diffusion_map.png">
</section>

<section>
	<h3>Factoring the BMDS diffusion model</h3>
	<br>
	<img class="stretch" src="/images/talks/bmds_diffusion_model.png">
</section>

<section data-background="#6DB28B">
	<h2 class="title">Antigenic cluster transitions</h2>
	<p class="title">with Charles Cheung, Andrew Rambaut and Marc Suchard</p>
	<br>
	<p class="smaller title">
		Cheung et al 2015. Detailed antigenic dynamics of influenza virus revealed by Bayesian phylogenetic clustering. <i>In prep</i>.
	</p>
</section>

<section data-background="/images/talks/clustering_simple_model.jpg">
</section>

<section>
	<h3>Factoring the BMDS discrete transition model</h3>
	<br>
	<img class="stretch" src="/images/talks/bmds_transition_model.png">
</section>

<section>
	<h3>H3N2 discrete antigenic dynamics</h3>
	<img class="stretch" src="/images/talks/clustering_h3n2_dynamics.png">
</section>

<section>
	<h3>Association map of H3N2 antigenic clustering</h3>
	<img class="stretch" src="/images/talks/clustering_h3n2_association.png">
</section>

<section data-background="/images/talks/clustering_driver_model.jpg">
</section>

<section>
	<h3>H3N2 driver mutations</h3>
	<br>
	<p class="smaller">Sites: 135, 144, 145, 155, 156, 157, 158, 159, 189, 223 , 225, 226</p>
	<img class="stretch" src="/images/talks/clustering_driver_sites.png">
</section>

<section>
	<h3>H3N2 clustering with driver mutations</h3>
	<img class="stretch" src="/images/talks/clustering_driver_phylogeny.png">
</section>

<section data-background="#8CBB68">
	<h2 class="title">Drift across viruses</h2>
	<br>
	<p class="smaller title">
		<a class="title" href="http://bedford.io/papers/bedford-flux/">
			Bedford et al 2014. Integrating influenza antigenic dynamics <br> with molecular evolution. eLife.
		</a>
	</p>
</section>

<section>
	<h3>Phylogenetic trees of different influenza lineages</h3>
	<img class="stretch" src="/images/talks/flu_trees.png">
</section>

<section>
	<h3>Antigenic phenotype across lineages</h3>
	<img class="stretch" src="/images/talks/bmds_lineages_map.png">
</section>

<section>
	<h3>Antigenic drift across lineages</h3>
	<img class="stretch" src="/images/talks/bmds_lineages_drift.png">
</section>

<section data-transition="none">
	<h3>Antigenic drift across lineages</h3>
	<img class="stretch" src="/images/talks/bmds_lineages_drift_stats.png">
</section>

<section data-background="#AFBD4F">
	<h2 class="title">Geographic circulation</h2>
	<p class="title">with Colin Russell, Philippe Lemey and many others</p>
	<br>
	<p class="smaller title">
		<a class="title" href="http://bedford.io/papers/bedford-global-circulation/">
			Bedford et al 2015. Global circulation patterns of seasonal influenza viruses vary with rates of antigenic drift. Nature.
		</a>
	</p>
</section>

<section>
	<h3>Seasonality in influenza</h3>
	<img class="stretch" src="/images/talks/flu_seasonality.png">
	<div class="citation">
		<a href="http://www.nature.com/nrg/journal/v8/n3/abs/nrg2053.html">
			Nelson and Holmes 2007
		</a>
	</div>
</section>

<section>
	<h3>Hypotheses of influenza circulation patterns</h3>
	<img class="stretch" src="/images/talks/flu_circulation_hypotheses.png">
</section>

<section>
	<h3>Influenza H3 genealogy for NY state viruses</h3>
	<img class="stretch" src="/images/talks/flu_tree_ny_state.png">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v453/n7195/abs/nature06945.html">
			Rambaut et al 2008
		</a>
	</div>
</section>

<section>
	<h3>Hypotheses of influenza circulation patterns</h3>
	<img class="stretch" src="/images/talks/flu_circulation_hypotheses_remaining.png">
</section>

<section>
	<h3>Sample H3N2 from around the world</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_sample_map.png">
</section>

<section>
	<h3>Treating geographic state as an evolving character</h3>
	<img class="stretch" src="/images/talks/beast_geo_ctmc.png">
</section>

<section>
	<h3>Phylogeny of H3 with geographic history</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
</section>

<section>
	<h3>Infer geographic transition matrix</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_map.png">
</section>

<section data-background="/images/talks/flight_network.jpg">
</section>

<section>
	<h3>Air travel predicts migration rates</h3>
	<img class="stretch" src="/images/talks/flu_geo_predictors.png">
	<div class="citation">
		<a href="http://bedford.io/papers/lemey-air-travel/">
			Lemey et al 2014
		</a>
	</div>
</section>

<section>
	<h3>Geographic location of phylogeny trunk</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_trunk.png">
</section>

<section>
	<h3>Region-specific ancestry</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_hist.png">
</section>

<section>
	<h3>Phylogenies across subtypes / lineages</h3>
	<img class="stretch" src="/images/talks/flu_geo_lineages_trees.jpg">
</section>

<section>
	<h3>H3N2 phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
</section>

<section>
	<h3>H1N1 phylogeny</h3>
	<img class="stretch" src="/images/talks/flu_geo_h1_tree.png">
</section>

<section>
	<h3>B/Vic phylogeny</h3>
	<img class="stretch" src="/images/talks/flu_geo_vic_tree.png">
</section>

<section>
	<h3>B/Yam phylogeny</h3>
	<img class="stretch" src="/images/talks/flu_geo_yam_tree.png">
</section>

<section>
	<h3>Ancestry patterns across lineages</h3>
	<img class="stretch" src="/images/talks/flu_geo_lineages_histories.png">
</section>

<section>
	<h3>Regional persistence patterns</h3>
	<img class="stretch" src="/images/talks/flu_geo_lineages_persistence.png">
</section>

<section>
	<h3>How to explain these differences?</h3>
</section>

<section>
	<h3>Age distribution across viruses</h3>
	<img class="stretch" src="/images/talks/flu_geo_age_of_infection.png">
</section>

<section>
	<h3>Air travel differences between adults and children</h3>
	<img class="stretch" src="/images/talks/flu_geo_air_travel_by_age.png">
</section>

<section>
	<h3>Epidemiological model of varying rates of antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_geo_model_dynamics.jpg">
</section>

<section>
	<h3>Results of varying antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_geo_model_results.png">
</section>

<section>
	<h3>Interaction between virus evolution, epidemiology and human behavior drives migration rate differences</h3>
</section>

<section>
	<h3>Circulating antigenic variants</h3>
	<img class="stretch" src="/images/talks/flu_geo_vic_tree_map.png">
</section>

<section data-background="#CFB541">
	<h2 class="title">Vaccine strain prediction</h2>
</section>

<section>
	<h3>Antigenic evolution in H3N2</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Predictive models</h3>
	<br>
	<p>A simple predictive model estimates the fitness $f$ of virus $i$ as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{ep} \, f_i^\mathrm{ep} + \beta^\mathrm{ne} \, f_i^\mathrm{ne}$$</p>
	<p>where $f_i^\mathrm{ep}$ measures cross-immunity via substitutions at epitope sites and $f_i^\mathrm{ep}$ measures mutational load via substitutions at non-epitope sites.</p>
	<img class="stretch" src="/images/talks/prediction_ll.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h3>Predictive models</h3>
	<br>
	<p>Another approach quantifies phylogenetic branching patterns</p>
	<img class="stretch" src="/images/talks/prediction_neher.jpg">
	<div class="citation">
		<a href="http://elifesciences.org/content/elife/3/e03568">Neher et al 2014</a>
	</div>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population
	<p class="fragment" data-fragment-index="1">All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch fragment" data-fragment-index="1" src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Align sequences</li>
		<li>More filtering</li>
		<li>Build tree</li>
		<li>Estimate frequencies</li>
		<li>Export JSON for visualization</li>
	</ol>
</section>

<section>
	<h1>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h1>
</section>

<section>
	<h3>Acknowledgements</h3>
	<br>
	<p class="smaller">Richard Neher (Max Planck T&uuml;bingen), Andrew Rambaut (University of Edinburgh),
		Colin Russell (Cambridge University), Charles Cheung (Fred Hutch),
		Marc Suchard (UCLA), Steven Riley (Imperial College),
		Philippe Lemey (Philippe Lemey (KU Leuven), Gytis Dudas (University of Edinburgh)
		<br>
		WHO Global Influenza Surveillance Network / GISAID: Ian Barr, Shobha Broor, Mandeep Chadha, Nancy Cox, Rod Daniels, Palani 
Gunasekaran, Aeron Hurt, Anne Kelso, Alexander Klimov, Nicola Lewis, Xiyan Li, John McCauley, Takato Odagiri, Varsha Potdar, Yuelong Shu, Eugene Skepner, Masato Tashiro, Dayan Wang, Xiyan Xu
	</p>
	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="50%" src="/images/talks/logo_fred_hutch.png">
		&nbsp;
		<img width="25%" src="/images/talks/logo_midas.jpg">
	</div>
</section>

<section data-background="#000000">
	<h3 class="title">Contact</h3>
	<br>
	<p>
	<ul class="title">
		<li>Website: bedford.io</li>
		<li>Twitter: @trvrb</li>
		<li>Slides: bedford.io{{ page.url }}</li>
	</ul>
</section>
