---
layout: talk
title: Real-time tracking of virus evolution
venue: Incheon, Republic of Korea
event: Federation Meeting of Korean Basic Medical Scientists
author: Trevor Bedford
---

<!-- 40 minutes total for talk. Aim for ~33 min + questions. -->
<!-- Color ramp: ["#5097BA", "#5BA6A7", "#69B091", "#79B77D", "#8BBB6A", "#9EBE5A", "#B2BD4D", "#C4B945", "#D4B13F", "#DFA43B"] -->

<section data-background="#000000">
	<h2 class="title">{{ page.title }}</h2>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}	
</section>

<section data-background="#000000">
	<h3 class="title">Slides at bedford.io/talks/</h3>
</section>

<section data-background="#5097BA">
	<h2 class="title">Phylogenies</h2>
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_tudors.jpg">
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_haeckel.jpg">
	<div class="citation">
		Haeckel 1879
	</div>
</section>

<section>
	<h3>Phylogenies describe history</h3>
	<img class="stretch" src="/images/talks/phylogenies_pace.png">
	<div class="citation">
		<a href="http://www.sciencemag.org/content/276/5313/734.short">Pace 1997</a>
	</div>
</section>

<section>
	<h3>Phylogenies reveal process</h3>
	<img class="stretch" src="/images/talks/phylogenies_darwin.png">
	<div class="citation">
		Darwin 1859
	</div>	
</section>

<section data-background="#5BA6A7">
	<h2 class="title">Epidemics</h2>
</section>

<section>
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4.png">
</section>

<section>
	<h3>Localized Middle Eastern MERS-CoV phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_mers.jpg">
	<div class="citation">
		<a href="http://www.thelancet.com/journals/lancet/article/PIIS0140-6736(13)61887-5/abstract">Cotten et al 2013</a>
	</div>
</section>

<section>
	<h3>Regional West African Ebola phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_ebola.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>Global influenza phylogeny</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al 2015</a>
	</div>
</section>

<section>
	<h3>Applications of evolutionary analysis for <b>vaccine strain selection</b> and <b>charting outbreak spread</b></h3>
</section>

<section data-background="#69B091">
	<h2 class="title">Influenza</h2>
</section>

<section>
	<h3>Influenza virion</h3>
	<img class="stretch" src="/images/talks/flu_virion.jpg">
</section>

<section>
	<h3>Influenza H3N2 vaccine updates</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_updates.png">
</section>

<section>
	<h3>H3N2 phylogeny showing antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree.png">				
</section>

<section>
	<h3>H3N2 phylogeny showing antigenic drift</h3>
	<img class="stretch" src="/images/talks/flu_h3n2_tree_vaccines.png">				
</section>

<section data-background="#559EB1">
	<h3 class="title">Drift variants rapidly take over the virus population</h3>
	<br>
	<h3 class="title">Timely surveillance and rapid analysis essential to understand ongoing influenza evolution</h3>
</section>

<section>
	<h2>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population
	<p class="fragment" data-fragment-index="1">All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch fragment" data-fragment-index="1" src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>		
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate frequencies</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section>
	<h3>Up-to-date analysis publicly available at:</h3>
	<h2>
		<a href="http://nextflu.org">
			<span class="c2">n</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span><span class="text-gray">.org</span>
		</a>
	</h2>
</section>

<section data-background="#79B77D">
	<h2 class="title">Forecasting</h2>
</section>

<section>
	<h3><p><q>The future is here, it's just not evenly distributed yet</q> &mdash; William Gibson</p></h3>
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" src="/images/talks/clonal_interferance.jpg">		
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>	
</section>

<section>
	<h3>Predictive fitness models</h3>
	<br>
	<p>A simple predictive model estimates the fitness $f$ of virus $i$ as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{ep} \, f_i^\mathrm{ep} + \beta^\mathrm{ne} \, f_i^\mathrm{ne}$$</p>
	<p>where $f_i^\mathrm{ep}$ measures cross-immunity via substitutions at epitope sites and $f_i^\mathrm{ep}$ measures mutational load via substitutions at non-epitope sites</p>
	<img class="stretch" src="/images/talks/prediction_ll.jpg">	
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>	
</section>

<section>
	<h3>We implement a similar model based on two predictors</h3>
	<br>
	<ol>
		<li class="larger">Clade frequency change</li>
		<li class="larger">Antigenic advancement</li>
	</ol>
</section>

<section>
	<h3>Project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_frequency_trajectories.png">
</section>

<section>
	<h3>Calculate HI drop from ancestor, <br> drifted clades have high fitness</h3>
	<img class="stretch" src="/images/talks/prediction_antigenic_drift.png">
</section>

<section>
	<h3>Fitness model parameterization</h3>
	<br>
	<p>Our predictive model estimates the fitness $f$ of virus $i$ as</p>
	<br>
	<p>$$\hat{f}_i = \beta^\mathrm{freq} \, f_i^\mathrm{freq} + \beta^\mathrm{HI} \, f_i^\mathrm{HI}$$</p>
	<br>	
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>	

<section>
	<h3>Clade growth rate is well correlated (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section>
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" src="/images/talks/prediction_trajectories_grid.png">
</section>

<section>
	<h3>Further work on predictive modeling</h3>
	<br>
	<ol>	
		<li>Integrate data predictors and data sources, e.g. geography</li>
		<li>Possible to build predictive models for H1N1 and B and to forecast NA evolution</li>
	</ol>
</section>

<section>
	<h3>Real-time analyses are <i>actionable</i> and thus, may inform influenza vaccine strain selection</h3>
</section>

<section data-background="#000000">
	<h2 class="title">Outbreak analysis</h2>
</section>

<section data-background="#8BBB6A">
	<h2 class="title">Ebola</h2>
</section>

<section>
	<h3>Epidemic basically contained, but resulted in >28,000 confirmed cases and >11,000 deaths</h3>
	<img class="stretch" src="/images/talks/ebola_timeseries.png">
</section>

<section>
	<h3>Early sequencing showed single origin of epidemic</h3>
	<img class="stretch" src="/images/talks/ebola_early_outbreak_tree.png">
	<div class="citation">
		<a href="https://www.sciencemag.org/content/345/6202/1369.full">Gire et al 2014</a>
	</div>	
</section>

<section>
	<h3>Continued spread through Dec 2014</h3>
	<img class="stretch" src="/images/talks/ebola_mid_outbreak_mst.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>At epidemic height, geographic spread of particular interest</h3>
	<img class="stretch" src="/images/talks/ebola_mid_outbreak_geo_tree.png">
	<div class="citation">
		Rambaut 2015
	</div>	
</section>

<section>
	<h3>Later on, tracking transmission clusters of primary importance</h3>
	<img class="stretch" src="/images/talks/ebola_late_outbreak_tree.png">
	<div class="citation">
		<a href="http://virological.org/t/update-on-ebola-virus-circulating-in-sierra-leone-and-guinea/151">Rambaut 2015</a>
	</div>	
</section>

<section>
	<h3>Evolutionary analyses helped to establish the degree of adaptive evolution occurring</h3>
	<img class="stretch" src="/images/talks/ebola_selection_average.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>Selective patterns differ across genome</h3>
	<img class="stretch" src="/images/talks/ebola_selection_by_gene.png">
	<div class="citation">
		<a href="http://bedford.io/papers/park-ebola-transmission/">Park et al 2015</a>
	</div>
</section>

<section>
	<h3>Phylogeographic analyses reveal detailed patterns of spatial movement</h3>
	<img class="stretch" src="/images/talks/ebola_spatial_spread_grid.png">
	<div class="citation">
		Dudas et al 2016
	</div>
</section>

<section>
	<h3>Animation by Gytis Dudas</h3>
    <iframe class="stretch" src="http://player.vimeo.com/video/156668942?title=0" ></iframe> 
	<div class="citation">
		Dudas et al 2016
	</div>
</section>

<section>
	<h3>Important analyses, let's make them more rapid and more automated</h3>
</section>

<section>
	<h3>Tracking epidemic spread in real-time:</h3>
	<h2>
		<a href="http://ebola.nextstrain.org">
			<span class="text-gray">ebola.nextstrain.org</span>
		</a>
	</h2>
</section>

<section data-background="/images/talks/nextstrain_in_sierra_leone.jpg">
	<h3 class="title">Rapid on-the-ground sequencing by Ian Goodfellow, Matt Cotten and colleagues</h3>
	<br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p>
</section>

<section>
	<h3>Deployment of MinION sequencing to Guinea by Nick Loman, Josh Quick, Lauren Cowley and colleagues</h3>
	<img class="stretch" src="/images/talks/minion_ebola_sequencing.jpg">	
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v530/n7589/full/nature16996.html">Quick et al 2016</a>
	</div>
</section>

<section data-background="#9EBE5A">
	<h2 class="title">Zika</h2>
</section>

<section>
	<h3>Virus endemic to Africa, emergence in Southeast Asia in the last century</h3>
	<img class="stretch" src="/images/talks/zika_phylogeny_africa_asia.jpg">
	<div class="citation">
		<a href="http://virologydownunder.blogspot.com.au/2016/03/detecting-zika-viruses-using-pcr-look.html">Mackay 2016</a>
	</div>	
</section>

<section>
	<h3>Spread eastward through the South Pacific</h3>
	<img class="stretch" src="/images/talks/zika_poly_12_phylogeography.png">
	<div class="citation">
		<a href="http://virological.org/t/initial-zika-phylogeography/202">Lycett 2016</a>
	</div>
</section>

<section>
	<h3>Isolated epidemics in the South Pacific</h3>
	<img class="stretch" src="/images/talks/zika_fp_timeseries.png">
	<div class="citation">
		<a href="http://biorxiv.org/content/early/2016/02/07/038588">Kucharski et al 2016</a>
	</div>
</section>

<section>
	<h3>Single arrival into the Americas in early 2014</h3>
	<img class="stretch" src="/images/talks/zika_faria_tmrca.png">
	<div class="citation">
		<a href="http://science.sciencemag.org/content/early/2016/03/23/science.aaf5036">Faria et al 2016</a>
	</div>
</section>

<section>
	<h3>Working on analysis of ongoing evolution:</h3>
	<h2>
		<a href="http://nextstrain.org/zika/">
			<span class="text-gray">nextstrain.org/zika/</span>
		</a>
	</h2>	
</section>

<section>
	<h3>ZiBRA: Project to do real-time sequencing of Zika in Brazil</h3>
	<img class="stretch" src="/images/talks/zika_zibra_logo.png">
</section>

<section>
	<h3>Road trip through northeast Brazil to collect samples and sequence</h3>
	<img class="stretch" src="/images/talks/zika_zibra_roadtrip_samples.jpg">
</section>

<section data-background="/images/talks/zika_zibra_minion.jpg">
	<h3 class="title">Sequencing on the MinION nanopore sequencer</h3>
	<br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p>
</section>

<section>
	<h3>Moving forward, genetically-informed outbreak response requires:</h3>
	<br>
	<p>
	<ul>
		<li>Rapid sharing of sequence data, <i>genetic context critical</i></li>
		<li>Technologies for rapid diagnostics and sequencing</li>
		<li>Technologies to rapidly conduct phylogenetic inference</li>
		<li>Technologies to explore genetic relationships and inform epidemiological investigation</li>
	</ul>
</section>

<section data-background="#000000">
	<h2 class="title">Future work</h2>
</section>

<section>
	<img class="stretch" src="/images/talks/all_the_things.png">
</section>

<section>
	<h3>Acknowledgements</h3>
	<br>
	<p class="smaller"><b>Influenza</b>: WHO Global Influenza Surveillance Network, Worldwide Influenza Centre 
	at the Francis Crick Institute, Richard Neher, Colin Russell, Andrew Rambaut</p>
	
	<p class="smaller"><b>Ebola</b>: data producers, Gytis Dudas, Andrew Rambaut, Philipe Lemey, Richard Neher, 
	Nick Loman, Ian Goodfellow, Paul Kellam, Danny Park, Kristian Andersen, Pardis Sabeti</p>

	<p class="smaller"><b>Zika</b>: data producers, Nick Loman, Nuno Faria, Andrew Rambaut, Oliver Pybus, Richard Neher, 
	Charlton Callender, Allison Black, Luiz Alcantara and the rest of the ZiBRA team</p>
	
	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="50%" src="/images/talks/logo_fred_hutch.png">
		&nbsp;
		<img width="25%" src="/images/talks/logo_midas.jpg">	
	</div>
</section>

<section data-background="#000000">
	<h3 class="title">Contact</h3>
	<br>
	<p>
	<ul class="title">
		<li>Website: bedford.io</li>
		<li>Twitter: @trvrb</li>
		<li>Slides: bedford.io{{ page.url }}</li>
	</ul>
</section>
