---
layout: talk_reveal3
title: Influenza evolutionary dynamics and global circulation patterns
venue: Leavenworth, WA
event: HIV Dynamics and Evolution
author: Trevor Bedford
---

<!-- 45 minutes total. Aim for 35 minutes + questions. -->

<!-- Abstract:
Owing to rapid mutation, the evolution of the influenza virus occurs on a human timescale; rather
than being forced to infer past evolutionary events, we can observe them in near real-time. While
individuals develop long-lasting immunity to particular influenza strains after infection, antigenic
mutations to the influenza virus genome result in proteins that are recognized to a lesser degree by
the human immune system, leaving individuals susceptible to future infection. Mutations are only
transiently advantageous; the virus population must keep evolving antigenically to stay ahead of
developing human immunity. Influenza shows remarkably rapid evolution on the global scale, wherin
new strains emerge and replace existing genetic diversity in just 2-5 years. The dynamics of strain
competition are fierce and only the fittest strains survive. Here, we characterize influenza
antigenic evolution and demonstrate its impact on fitness. We also investigate global circulation
patterns and reassortment dynamics. We construct a predictive fitness model to forecast future
strain frequencies. These dynamics may particularly inform approaches to the study of within-host
HIV evolution.
-->

<section data-background-iframe="/talks/viz/gradient/">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section>
	<h3>HIV vs influenza</h3>
	<img class="stretch" src="/images/talks/hiv_vs_flu_virion.png">
</section>

<section>
	<h3>HIV within-host phylogeny vs influenza between-host phylogeny</h3>
	<img class="stretch" src="/images/talks/hiv_vs_flu_grenfell.png">
	<div class="citation">
		<a href="http://science.sciencemag.org/content/303/5656/327.long">Grenfell et al. 2004. Science.</a>
	</div>
</section>

<section>
	<h2>Fundamentally driven by the same process</h2>
</section>

<section>
	<h3>Antigenic drift driven by acquired immunity</h3>
	<img class="stretch" src="/images/talks/antigenic_drift_diagram.png">
</section>

<section>
	<h3>Selection visible in diversity and divergence</h3>
	<img class="stretch" src="/images/talks/diversity_and_divergence_diagram.png">
</section>

<section>
	<h3>Systematic divergence at gp120 through time</h3>
	<img class="stretch" src="/images/talks/hiv_gp120_divergence.png">
	<div class="citation">
		<a href="https://elifesciences.org/articles/11282">Zanini et al. 2015. eLife.</a>
	</div>
</section>

<section>
	<h3>Within-host HIV shows adaptive evolution in envelope and purifying selection in enzymes</h3>
	<img class="stretch" src="/images/talks/hiv_diversity_and_divergence.png">
	<div class="citation">
		<a href="https://elifesciences.org/articles/11282">Zanini et al. 2015. eLife.</a>
	</div>
</section>

<section>
	<h3>Between-host influenza shows adaptive evolution in surface proteins and purifying selection in internal proteins</h3>
	<img class="stretch" src="/images/talks/flu_mcdonald_kreitman.png">
	<div class="citation">
		<a href="https://academic.oup.com/mbe/article/28/9/2443/1007907">Bhatt et al. 2011. Mol Biol Evol.</a>
	</div>
</section>

<section data-background="#000">
	<h1 class="title">Influenza evolution</h1>
</section>

<section>
	<h3>Population turnover is extremely rapid</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_beast.png">
</section>

<section>
	<h3>Clades emerge, die out and take over</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_clades.png">
</section>

<section>
	<h3>Clades show rapid turnover</h3>
	<img class="stretch" data-src="/images/talks/prediction_combined_trajectories.png">
</section>

<section>
	<h3>Dynamics driven by antigenic drift</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree.png">
</section>

<section>
	<h3>Drift necessitates vaccine updates</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines.png">
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates_2018.png">
</section>

<section>
	<h3>Vaccine strain selection by WHO</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h2>Problem of applied evolutionary biology</h2>
</section>

<section style="text-align: left;">
	<h3>Every paper in the field...</h3>
	<ul>
		<li>"These observations have implications for influenza surveillance and vaccine formulation" (Wolf et al 2006)</li>
		<li>"Our results have implications for the design of vaccines to combat rapidly mutating viral diseases" (Gupta et al 2006)</li>
		<li>"These results may have important implications for influenza vaccine and antiviral research" (Bhatt et al 2011)</li>
		<li>"Needless to say, these results have important implications for the updating of vaccines against influenza" (Zinder et al 2013)</li>
	</ul>
</section>

<section style="text-align: left;">
	<h3>Disconnect between evolutionary studies and information needed by WHO</h3>
	<ul>
		<li>WHO needs <i>specific</i> advice, ie this strain is likely to take off, this strain is likely to die out</li>
		<li>Problems of generality and timeliness</li>
	</ul>
</section>

<section data-background="/images/talks/kavli.jpg">
	<h3 class="title">Microbial evolution workshop at the Kavli Institute for Theoretical Physics</h3>
	<br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p><br><p>
</section>

<section style="text-align: left;">
	<h3>Decided to tackle this head on and build something that</h3>
	<ol>
		<li>Charts behavior of specific strains</li>
		<li>Can be kept continually up to date</li>
	</ol>
</section>

<!---------------------------------->

<section>
	<h1>
		<span class="c2">N</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h1>
	<p>Project to provide a real-time view of the evolving influenza population</p>
</section>

<section>
	<h2>
		<span class="c2">N</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
	</h2>
	<p>Project to provide a real-time view of the evolving influenza population</p>
	<p>All in collaboration with <a href="https://neherlab.wordpress.com/">Richard Neher</a></p>
	<img class="stretch" data-src="/images/talks/neher.jpg">
</section>

<section>
	<h3>
		<span class="c2">N</span><span class="c3">e</span><span class="c4">x</span><span class="c5">t</span><span class="c6">f</span><span class="c7">l</span><span class="c8">u</span>
		pipeline
	</h3>
	<p>
	<ol>
		<li>Download all recent HA sequences from GISAID</li>
		<li>Filter to remove outliers</li>
		<li>Subsample across time and space</li>
		<li>Align sequences</li>
		<li>Build tree</li>
		<li>Estimate clade frequencies</li>
		<li>Infer antigenic phenotypes</li>
		<li>Export for visualization</li>
	</ol>
</section>

<section>
	<h3>
		Subsequently wrapped into larger efforts to do real-time analysis across pathogens with the
		<span class="c1s">N</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
		project
	</h3>
	<br>
	<p class="smaller"><i>
		with
		<img width=40 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,
		<img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=40 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill,
		<br>
		<img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<img width=40 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<br>
		<img width=40 class="img-avatar" src="/images/team/charlton-callender.jpg"> Charlton Callender,
		<img width=40 class="img-avatar" src="/images/team/emma-hodcroft.jpg"> Emma Hodcroft
	</i></p>
</section>

<section>
	<h3>Up-to-date analysis publicly available at:</h3>
	<h2>
		<a href="http://nextstrain.org/flu">
			<span class="c1s">n</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span><span class="text-gray">.org/flu</span>
		</a>
	</h2>
</section>

<!-- Demo focusing on the emergence and spread of 3c2a and 3c3a viruses -->

<section>
	<h3>Current H3N2 diversity</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_tree_clades.png">
</section>

<section>
	<h3>Current H3N2 diversity</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_timetree_clades.png">
</section>

<section>
	<h3>Two clades have been growing rapidly</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_tree_lbi.png">
</section>

<section>
	<h3>Clade A2 more recently increasing</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_clades_regions.png">
</section>

<section>
	<h3>Reassortment event appears to drive success of clade A2</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_ha_na_reassortment.png">
</section>

<section>
	<h3>Reassortment event appears to drive success of clade A2</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_clade_A3re.png">
</section>


<!----------------------------------------->

<section data-background="#000">
	<h1 class="title">Antigenic analysis</h1>
</section>

<section>
	<h3>Influenza hemagglutination inhibition (HI) assay</h3>
	<img class="stretch" data-src="/images/talks/hi_assay.png">
</section>

<section>
	<h3>HI measures cross-reactivity across viruses</h3>
	<img class="stretch" data-src="/images/talks/hi_cross_reactivity.jpg">
</section>

<section>
	<h3>Data in the form of table of maximum inhibitory titers</h3>
	<img class="stretch" data-src="/images/talks/hi_table.png">
	<div class="citation">
		<a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/annual-and-interim-reports/">NIMR 2010</a>
	</div>
</section>

<section>
	<div class="left" style="float:left;width:63%">
		<h3>Antigenic cartography compresses HI measurements into an interpretable diagram</h3>
	</div>
	<div class="right" style="float:right;width:33%">
		<img data-src="/images/talks/hi_smith_map.jpg">
	</div>
	<div class="citation">
		<a href="http://science.sciencemag.org/content/305/5682/371">Smith et al. 2004. Science.</a>
	</div>
</section>

<section>
	<h3>Evolutionary model of HI titer data in which particular substitutions drop titer</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_example.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section>

<section>
	<h3>Model can be used to interpolate across tree and predict phenotype of untested viruses</h3>
	<img class="stretch" data-src="/images/talks/titers_tree_model.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section>

<section>
	<h3>Model is highly predictive of missing titer values</h3>
	<img class="stretch" data-src="/images/talks/titers_validation.png">
	<div class="citation">
		<a href="http://bedford.io/papers/neher-titers/">Neher et al. 2016. PNAS.</a>
	</div>
</section>

<section>
	<h2>Incorporate HI data from WHO Collaborating Centers</h2>
</section>

<section>
	<h3>Recent H3N2 antigenic evolution</h3>
	<img class="stretch" data-src="/images/talks/nextflu_apr2018_h3n2_tree_titers.png">
</section>

<!---------------------------------->

<section data-background="#000">
	<h1 class="title">Forecasting</h1>
</section>

<section>
	<h2><p><q>"The future is here, it's just not evenly distributed yet"</q> <br> &mdash; William Gibson</p></h2>
</section>

<section>
	<h3>USA music industry, 2011 dollars per capita</h3>
	<img class="stretch" data-src="/images/talks/music_industry.jpg">
</section>

<section>
	<h3>Influenza population turnover</h3>
	<img class="stretch" data-src="/images/talks/flu_turnover.png">
</section>

<section>
	<h3>Vaccine strain selection timeline</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Seek to explain change in clade frequencies over 1 year</h3>
	<img class="stretch" data-src="/images/talks/prediction_clade_frequencies.png">
</section>

<section>
	<h3>Fitness models can project clade frequencies</h3>
	<br>
	<p>Clade frequencies $X$ derive from the fitnesses $f$ and frequencies $x$ of constituent viruses, such that</p>
	<p>$$\hat{X}_v(t+\Delta t) = \sum_{i:v} x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>This captures clonal interference between competing lineages</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig 2014</a>
	</div>
</section>

<section>
	<h2>The question of forecasting becomes: how do we accurately estimate fitnesses of circulating viruses?</h2>
</section>

<section>
	<h3>Fortunately, there's lots of training data and previously successful strains have had:</h3>
	<br>
	<ol>
		<li class="larger">Amino acid changes at epitope sites</li>
		<li class="larger">Antigenic novelty based on HI</li>
		<li class="larger">Rapid phylogenetic growth</li>
	</ol>
</section>

<!-- <section>
	<h3>Predictor: calculate HI drop from ancestor, <br> drifted clades have high fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_antigenic_drift.png">
</section>

<section>
	<h3>Predictor: project frequencies forward, <br> growing clades have high fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_frequency_trajectories.png">
</section> -->

<section>
	<h3>We predict fitness based on a simple formula</h3>
	<br>
	<p>where the fitness $f$ of virus $i$ is estimated as</p>
	<p>$$\hat{f}_i = \beta^\mathrm{HI} \, f_i^\mathrm{HI} + \beta^\mathrm{freq} \, f_i^\mathrm{freq}$$</p>
	<p>where $f_i^\mathrm{HI}$ measures antigenic drift via HI and $f_i^\mathrm{freq}$ measures clade growth/decline</p>
	<p>We learn coefficients and validate model based on previous 15 H3N2 seasons</p>
</section>

<section>
	<h3>Clade growth rate is well predicted (<i>&rho;</i> = 0.66)</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates.png">
</section>

<section>
	<h3>Growth vs decline correct in 84% of cases</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_growth_rates_window.png">
</section>

<section data-transition="slide-in zoom-out" data-transition-speed="slow">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_grid.png">
</section>

<section data-transition="zoom-in slide-out">
	<h3>Trajectories show more detailed congruence</h3>
	<img class="stretch" data-src="/images/talks/prediction_trajectories_zoom.png">
</section>

<section>
	<h3>Current forecast favors drifted A1b clade</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_prediction.png">
</section>

<section>
	<h3>Unlikely to replace human intuition, but could automate serum and VCV selection</h3>
</section>

<!---------------------------------->

<section data-background="#000">
	<h1 class="title">Geographic circulation</h1>
</section>

<section>
	<h3>Geographic circulation patterns drive evolutionary outcomes</h3>
</section>

<section>
	<h3>Clades commonly show region-specific circulation patterns</h3>
	<img class="stretch" data-src="/images/talks/nextflu_feb2018_h3n2_freq_clades_regions.png">
</section>

<section>
	<h3>Phylogeny of H3 with geographic history</h3>
	<img class="stretch" src="/images/talks/pathogen_phylogeny_flu.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h3>Infer geographic transition matrix</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_map.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h3>Geographic location of phylogeny trunk</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_trunk.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h3>Region-specific ancestry</h3>
	<img class="stretch" src="/images/talks/flu_geo_h3_hist.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
</section>

<section>
	<h2>Which regions are most predictive of future outcomes?</h2>
</section>

<section>
	<h3>Correlation between clade frequencies</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_scatterplot.png">
</section>

<section>
	<h3>Correlations across regions</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_matrix_0m_lag.png">
</section>

<section>
	<h3>Correlations across regions with 6-month lag</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_matrix_6m_lag.png">
</section>

<section>
	<h3>Correlations across regions with 12-month lag</h3>
	<img class="stretch" src="/images/talks/flu_geo_corr_matrix_12m_lag.png">
</section>

<!----------------------------------------->

<section style="text-align: left;">
	<h2>Next steps</h2>
	<ul>
		<li>Systematically test fitness predictors</li>
		<li>Include geographic component to fitness model</li>
		<li>Incorporate information from other segments</li>
		<li>Integrate human serology to estimate selective landscape</li>
	<ul>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="even-smaller"><b>Bedford Lab</b>:
		<img width=50 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,
		<img width=50 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,
		<img width=50 class="img-avatar" src="/images/team/gytis-dudas.jpg"> Gytis Dudas,
		<img width=50 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,
		<br>
		<img width=50 class="img-avatar" src="/images/team/barney-potter.jpg"> Barney Potter,
		<img width=50 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,
		<img width=50 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla
		</p>

	<p class="even-smaller"><b>Influenza</b>: WHO Global Influenza Surveillance Network, GISAID, Richard Neher,
	Barney Potter, John Huddleston, James Hadfield, Colin Russell, Andrew Rambaut, Dave Wentworth, Becky Garten, Jackie Katz,
	Marta &#321;uksza, Michael L&auml;ssig, Richard Reeve</p>

	<p class="even-smaller"><b>Nextstrain</b>: Richard Neher, James Hadfield, Colin Megill, Sidney Bell,
	Charlton Callender, Barney Potter, John Huddleston, Emma Hodcroft</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="15%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_hhmi.jpg">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="15%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>

<section data-background="#000">
	<img width=100 src="/images/talks/giphy_virus.gif">
</section>
