---
layout: talk_reveal3
title: Genomic tracking of SARS-CoV-2 evolution and spread
venue: Fred Hutch
event: VIDD Seminar
author: Trevor Bedford
widescreen: true
---

<!-- Bio: Trevor Bedford is an Associate Professor in the Vaccine and Infectious Disease Division at
the Fred Hutchinson Cancer Research Center and is affiliate faculty at the University of Washington.
His research program focuses on phylodynamic analysis of pathogen sequence data with the goal of
making inferences that are actionable to public health. This research program spans a number of
viral systems including seasonal and avian influenza, Ebola, Zika, SIV, MERS-CoV, dengue, mumps and
SARS-CoV-2. Dr. Bedford's open-source Nextstrain project aims to harness the scientific and public
health potential of pathogen genome data by providing a continually-updated view of publicly
available data alongside powerful analytic and visualization tools for use by the community. This
platform has seen substantial public health adoption to improve seasonal influenza vaccines, combat
Ebola outbreaks and to understand and curb SARS-CoV-2 transmission. With the Seattle Flu Study, Dr.
Bedford lead the design and implementation of a next-generation infectious disease surveillance
system that discovered community transmission of COVID-19 in February 2020 and radically impacted
state and local response to the pandemic. -->

<!-- Abstract: Here, I present on emergence and spread of SARS-CoV-2 as uncovered by pathogen genome
sequencing. I discuss our work with the Nextstrain project to provide a platform for genomic
surveillance of SARS-CoV-2 and dissemination of phylodynamic analyses. I also discuss the Seattle
Flu Study, our uncovering of community transmission of SARS-CoV-2 and our ongoing collaboration in
local public health response. -->

<!-- Outline:
- Overview of what we've been up to since I last presented to the department 3 years ago
- A lot of work shaped by what's going on in the world, previously had been coming of of Ebola in West Africa and the Zika epidemic in the Americas
- Since then we've had the following papers:
  - MERS-Cov
  - West Nile
  - Zika
  - seasonal influenza
  - avian influenza
  - Ebola
- Briefly highlight influenza forecasting and Ebola in DRC
- SARS-CoV-2 and this crazy year
- emergence
- Nextstrain
- early pandemic spread
- SFS
- Cryptic transmission
- SCAN pivot
- Transmission in Washington State D614G
- Ongoing SARS-CoV-2 work: UW Reopening, continued SFS, Nextstrain development, etc... -->

<!-- Total time of ~40 minutes + 20 minutes for questions -->

<section data-background-iframe="/talks/viz/gradient/">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<section data-background="#000000">
	<h1 class="title">This talk</h1>
</section>

<!-- intro and research overview -->

<section>
	<h3>Start with an overview of what we've been up to since I last presented to the Division three years ago</h3>

	<p>A lot of our work is shaped by what's going on in the world, with previous focus on Ebola in West Africa and the Zika epidemic in the Americas</p>

	<div>
		<img width="14%" style="padding:5px;" src="/images/talks/tile_ebola_virus.png">
		<img width="14%" style="padding:5px;" src="/images/talks/tile_ebola_tree.png">
		<img width="14%" style="padding-top:5px; padding-right:25px; padding-bottom:5px; padding-left:5px;" src="/images/talks/tile_ebola_map.png">
		<img width="14%" style="padding-top:5px; padding-right:5px; padding-bottom:5px; padding-left:25px;" src="/images/talks/tile_zika_virus.png">
		<img width="14%" style="padding:5px;" src="/images/talks/tile_zika_tree.png">
		<img width="14%" style="padding:5px;" src="/images/talks/tile_zika_map.png">
	</div>

</section>

<section data-transition="slide-in none-out">
	<h3>Epidemic process</h3>
	<img class="stretch" src="/images/talks/infection_tree_1.png">
</section>

<section data-transition="none">
	<h3>Sample some individuals</h3>
	<img class="stretch" src="/images/talks/infection_tree_2.png">
</section>

<section data-transition="none">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_3b.png">
</section>

<section data-transition="none-in slide-out">
	<h3>Sequence and determine phylogeny</h3>
	<img class="stretch" src="/images/talks/infection_tree_4b.png">
</section>

<section data-transition="slide-in none-out">
	<h3 style="margin-top: -20px;">Non-COVID "lab-lead" papers since last review in July 2017</h3>
	<table class="even-smaller" style="line-height: 1.3em;">
		<tbody>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/dudas-mers-structure.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Dudas et al. 2018. MERS-CoV spillover at the camel-human interface. eLife.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/hadfield-nextstrain.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Hadfield et al. 2018. Nextstrain: real-time tracking of pathogen evolution. Bioinformatics.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/lee-h3n2-dms.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Lee et al. 2018. Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human H3N2 influenza variants. PNAS.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/bell-dengue-antigenic-dynamics.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Bell et al. 2019. Dengue genetic divergence generates within-serotype antigenic variation, but serotypes dominate evolutionary dynamics. eLife.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/hadfield-wnv-nextstrain.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Hadfield et al. 2019. Twenty years of West Nile virus spread and evolution in the Americas visualized by Nextstrain. PLoS Pathog.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/black-zika-in-colombia.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Black et al. 2019. Genomic epidemiology supports multiple introductions and cryptic transmission of Zika virus in Colombia. BMC Infect Dis.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/potter-h3n2-reassortment.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Potter et al. 2019. Evolution and rapid spread of a reassortant A(H3N2) virus that predominated the 2017-2018 influenza season. Virus Evol.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/dudas-genomic-horizon.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Dudas et al. 2019. The ability of single genes vs full genomes to resolve time and space in outbreak analysis. BMC Evol Biol.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/moncla-h5n1-cambodia.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Moncla et al. 2020. Quantifying within-host evolution of H5N1 influenza in humans and poultry in Cambodia. PLoS Pathog.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/black-genepi-whitepaper.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Black et al. 2020. Ten recommendations for supporting open pathogen genomic analysis in public health. Nat Med.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/hilton-dms-view.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Hilton et al. 2020. dms-view: Interactive visualization tool for deep mutational scanning data. J Open Source Softw.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/huddleston-flu-forecasting.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Huddleston et al. 2020. Integrating genotypes and phenotypes improves long-term forecasts of seasonal influenza A/H3N2 evolution. eLife.</td>
			</tr>
		</tbody>
	</table>
</section>

<section data-transition="none-in slide-out">
	<h3 style="margin-top: -20px;">Non-COVID "lab-lead" papers since last review in July 2017</h3>
	<table class="even-smaller" style="line-height: 1.3em;">
		<tbody>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/dudas-mers-structure.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Dudas et al. 2018. <b>MERS-CoV</b> spillover at the camel-human interface. eLife.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/hadfield-nextstrain.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Hadfield et al. 2018. Nextstrain: real-time tracking of pathogen evolution. Bioinformatics.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/lee-h3n2-dms.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Lee et al. 2018. Deep mutational scanning of hemagglutinin helps predict evolutionary fates of human <b>H3N2 influenza</b> variants. PNAS.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/bell-dengue-antigenic-dynamics.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Bell et al. 2019. <b>Dengue</b> genetic divergence generates within-serotype antigenic variation, but serotypes dominate evolutionary dynamics. eLife.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/hadfield-wnv-nextstrain.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Hadfield et al. 2019. Twenty years of <b>West Nile</b> virus spread and evolution in the Americas visualized by Nextstrain. PLoS Pathog.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/black-zika-in-colombia.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Black et al. 2019. Genomic epidemiology supports multiple introductions and cryptic transmission of <b>Zika</b> virus in Colombia. BMC Infect Dis.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/potter-h3n2-reassortment.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Potter et al. 2019. Evolution and rapid spread of a reassortant <b>A(H3N2)</b> virus that predominated the 2017-2018 influenza season. Virus Evol.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/dudas-genomic-horizon.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Dudas et al. 2019. The ability of single genes vs full genomes to resolve time and space in outbreak analysis <i>of <b>Ebola</b></i>. BMC Evol Biol.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/moncla-h5n1-cambodia.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Moncla et al. 2020. Quantifying within-host evolution of <b>H5N1 influenza</b> in humans and poultry in Cambodia. PLoS Pathog.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/black-genepi-whitepaper.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Black et al. 2020. Ten recommendations for supporting open pathogen genomic analysis in public health. Nat Med.</td>
			</tr>
			<tr>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/hilton-dms-view.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Hilton et al. 2020. dms-view: Interactive visualization tool for deep mutational scanning data. J Open Source Softw.</td>
				<td width="2%" style="border: none; padding: 1px;"></td>
				<td width="4%" style="border: none; padding: 1px;"><img width=50 style="vertical-align: middle;" src="/images/papers/huddleston-flu-forecasting.png"></td>
				<td width="45%" style="border: none; padding: 1px;">Huddleston et al. 2020. Integrating genotypes and phenotypes improves long-term forecasts of seasonal <b>influenza A/H3N2</b> evolution. eLife.</td>
			</tr>
		</tbody>
	</table>
</section>

<section data-background="#000000">
	<h1 class="title">Recent work with seasonal flu and Ebola</h1>
</section>

<!-- flu forecasting -->

<section>
	<h3>Integrating genotypes and phenotypes improves long-term forecasts of seasonal influenza A/H3N2 evolution</h3>
	<p class="smaller"><i>
	with <img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston, Richard Neher, Dave Wentworth, Becky Kondor, John McCauley, Hideki Hasegawa, Kanta Subbarao and others
	</i></p>
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>H3N2 vaccine updates occur every ~2 years</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_updates_2019_10.png">
</section>

<section>
	<h3>Vaccine strain selection by WHO</h3>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section>
	<h3>Select one strain from circulating diversity</h3>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines_2008_2020.png">
</section>

<section>
	<h3>Fitness models project strain frequencies</h3>
	<p>Future frequency $x_i(t+\Delta t)$ of strain $i$ derives from strain fitness $f_i$ and present day frequency $x_i(t)$, such that</p>
	<p>$$\hat{x}_i(t+\Delta t) = x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>
		Total strain frequencies at each timepoint are normalized.
		This captures clonal interference between competing lineages.
	</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig. 2014. Nature.</a>
	</div>
</section>

<section>
	<h3>Match strain forecast to retrospective circulation</h3>
	<img class="stretch" data-src="/images/talks/prediction_fitness_model_schematic.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Train in 6-year sliding windows from 1995 to 2015 with most recent years held out as test</h3>
	<img class="stretch" data-src="/images/talks/prediction_cross_validation_strategy.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Composite models favor combinations of HI drift, local branching index and non-epitope fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_performance_composite.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Model successfully predicts clade growth and best pick from model is generally close to best possible retrospective pick</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_clade_growth_ranking.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Forecasts are live at <a href="https://nextstrain.org/flu">nextstrain.org/flu</a>, but are currently fraught due to lack of recent isolates</h3>
	<img class="stretch" data-src="/images/talks/nextflu_oct2020_h3n2_tree_clades.png">
</section>

<!-- DRC Ebola epidemic -->

<section>
	<h3>Operationalizing genomic epidemiology during the Nord-Kivu Ebola outbreak, Democratic Republic of the Congo</h3>
	<p class="smaller"><i>
	with Eddy Kinganda-Lusamaki, <img width=40 class="img-avatar" src="/images/team/allison-black.jpg"> Allison Black, Daniel Mukadi, <img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield, Placide Mbala-Kingebeni, Catherine Pratt, Jean-Jacques Muyembe Tamfum, Steve Ahuka-Mundeke, Michael Wiley, Martine Peeters, Amadou Alpha Sall, Eric Delaporte, Gustavo Palacios and many others
	</i></p>
	<div class="citation">
		<a href="/papers/kinganda-lusamaki-ebola-drc/">Kinganda-Lusamaki et al. 2020. medRxiv.</a>
	</div>
</section>

<section style="text-align: left;">
	<h3>Genomic epidemiology used to track North Kivu outbreak</h3>
	<p>
	<ul>
		<li>Institut National de Recherche BiomÃ©dicale (INRB) sequenced samples in Kinshasa and Katwa</li>
		<li>Data and pipeline housed at <a href="https://github.com/inrb-drc/ebola-nord-kivu">github.com/inrb-drc/ebola-nord-kivu</a></li>
		<li>Resulting analyses deployed live to <a href="https://nextstrain.org/community/inrb-drc/ebola-nord-kivu">nextstrain.org/community/inrb-drc/ebola-nord-kivu</a></li>
	</ul>
	</p>
</section>

<section>
	<h3>792 Ebola virus genomes sequenced generally within 4 weeks of sample collection, representing 24% of confirmed cases</h3>
	<img class="stretch" src="/images/talks/ebola_drc_sampling.png">
	<div class="citation">
		<a href="/papers/kinganda-lusamaki-ebola-drc/">Kinganda-Lusamaki et al. 2020. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Outbreak sustained by locally transitory transmission chains</h3>
	<img class="stretch" src="/images/talks/ebola_drc_phylogeny.png">
	<div class="citation">
		<a href="/papers/kinganda-lusamaki-ebola-drc/">Kinganda-Lusamaki et al. 2020. medRxiv.</a>
	</div>
</section>

<section style="text-align: left;">
	<h3>Situation reports deployed as Nextstrain Narratives</h3>
	<p>
	<ul>
		<li><i>Narratives</i> are Markdown posts that allow pairing of narrative text to visualization state</li>
		<li>Situation reports describing new batches of sequenced cases shared as narratives and discussed at EOC meetings</li>
		<li>Example narrative for Ebola in the DRC here: <a href="https://nextstrain.org/narratives/inrb-ebola-example-sit-rep">nextstrain.org/narratives/inrb-ebola-example-sit-rep</a></li>
		<li>Focus on epidemiological inferences important for contact tracing</li>
	</ul>
	</p>
</section>

<section data-background="#000000">
	<h1 class="title">SARS-CoV-2</h1>
</section>

<!-- SARS-CoV-2 pandemic spread -->

<section>
	<h3>Detection and sequencing of SARS-CoV-2 in January</h3>
</section>

<section>
	<h3>Jan 11: First five genomes showed that the outbreak was caused by a novel SARS-like coronavirus</h3>
	<img class="stretch" data-src="/images/talks/ncov_sars_like_tree.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/blab/beta-cov">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Jan 19: First 12 genomes from Wuhan and Bangkok lack genetic diversity</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_2020_01_19.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>
		Single introduction into the human population between Nov 15 and Dec 15 and
		human-to-human epidemic spread from this point forward
	</h3>
</section>

<section>
	<h1>ðŸ˜§</h1>
</section>

<section>
	<h3>
		Spent the week of Jan 20 alerting public health officials, and since then have
		aimed to keep nextstrain.org up-to-date
	</h3>
</section>

<section>
	<h1>
		<span class="c1s">N</span><span class="c2s">e</span><span class="c3s">x</span><span class="c4s">t</span><span class="c5s">s</span><span class="c6s">t</span><span class="c7s">r</span><span class="c8s">a</span><span class="c9s">i</span><span class="c10s">n</span>
	</h1>
	<p>Project to conduct real-time genomic epidemiology and evolutionary analysis of emerging epidemics</p>
	<br>
	<p class="smaller"><i>
		with
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/emma-hodcroft.jpg"> Emma Hodcroft,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/ivan-aksamentov.jpg"> Ivan Aksamentov,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/moira-zuber.jpg"> Moira Zuber,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/eli-harkins.jpg"> Eli Harkins,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/misja-ilcisin.jpg"> Misja Ilcisin,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/kairsten-fay.jpg"> Kairsten Fay,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/allison-black.jpg"> Allison Black,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/sidney-bell.jpg"> Sidney Bell,</span>
		<span class="nobreak"><img width=40 class="img-avatar" src="/images/team/colin-megill.jpg"> Colin Megill</span>
	</i></p>
</section>

<section>
	<h3>Nextstrain architecture</h3>
	<p>All code open source at <a class="off" href="https://github.com/nextstrain">github.com/nextstrain</a></p>
	<p>Two central aims: (1) <i>rapid and flexible phylodynamic analysis</i> and <br> (2) <i>interactive visualization</i></p>
	<img width="65%" src="/images/talks/nextstrain_components_ncov.png">
</section>

<section style="text-align: left;">
	<h3>Rapid build pipeline for 3000 SARS-CoV-2 genomes (timings are for a laptop)</h3>
	<p>
	<ul>
		<li>Align with MAFFT <i>(~20 min)</i></li>
		<li>Build ML tree with IQTREE <i>(~40 min)</i></li>
		<li>Temporally resolve tree and geographic ancestry with TreeTime <i>(~50 min)</i></li>
		<li>Total pipeline <i>(~2 hr)</i></li>
	</ul>
	</p>
</section>

<section style="text-align: left;">
	<h3>Current data flow for SARS-CoV-2</h3>
	<p>
	<ol>
		<li>Labs contribute directly to GISAID (now have >150k full genomes)</li>
		<li>Nextstrain pulls a complete dataset from GISAID every 24 hours</li>
		<li>This triggers an automatic rebuild on Amazon Web Services</li>
		<li>We manually update new lat/longs, etc...</li>
		<li>We push this build online to nextstrain.org and tweet the update from @nextstrain</li>
	</ol>
	</p>
	We do one update per week day via Seattle and Basel.
</section>

<section>
	<h3>Sequencing and data sharing in almost real-time</h3>
	<img class="stretch" src="/images/talks/ncov_gisaid_2020_07_15_cadence.png">
	<div class="citation">
		Figure by Hadfield and Hodcroft using data from GISAID
	</div>
</section>

<section data-transition="slide-in none-out">
	<h3>Dec/Jan: Emergence from Wuhan in ~Nov 2019</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_dec_jan.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section data-transition="none">
	<h3>Jan/Feb: Spread within China and seeding elsewhere</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_jan_feb.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section data-transition="none">
	<h3>Feb/Mar: Epidemic spread within North America and Europe</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_feb_mar.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section data-transition="none-in slide-out">
	<h3>Mar/Apr: Decreasing transmission with social distancing</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_mar_apr.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Epidemic in the USA was introduced from China in late Jan and from Europe during Feb</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_usa_1.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Once in the US, virus spread rapidly</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_07_10_usa.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Single introduction at the beginning of Feb quickly shows up throughout the country</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_07_10_wa_clade.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<div class="inset-top-right">
		More recently, with ongoing mitigation <br> and decreased international travel, <br> regional clades have emerged
	</div>
	<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_global_jun_oct.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<div class="inset-top-right">
		More recently, with ongoing mitigation <br> and decreased international travel, <br> regional clades have emerged
	</div>
	<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_asia_jun_oct.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>
		Sequencing immediately useful for epidemiological understanding, but selection and functional impacts should also be studied
	</h3>
</section>

<section data-background="#59575A">
	<h3 class="title">Significant interest in spike mutation D614G</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_structure.png">
	<div class="citation">
		<a href="https://www.sciencedirect.com/science/article/pii/S0092867420308205">Korber et al. Cell.</a>
	</div>
</section>

<section>
	<h3>This mutation occurred in the initial European introduction</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_european_introduction.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>D614G is prevalent throughout Europe and mixed in US and Australia</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_map.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>D614G is increasing in frequency across states in US and Australia</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_frequencies.png">
</section>

<section>
	<h3>D614G is increasing in frequency across states in US and Australia</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_frequency_pairs.png">
</section>

<section style="text-align: left;">
	<h3>The success of D614G can be explained by either:</h3>
	<p>
	<ul>
		<li>D614G is more transmissible and has higher $R_0$</li>
		<li>founder effects and epidemiological confounding</li>
	</ul>
	</p>
</section>

<section>
	<h3>Additional evidence from Ct values of clinical specimens</h3>
	<p>
		<span style="display: inline-block; width:33%;">Sheffield, UK</span>
		<span style="display: inline-block; width:50%;">Seattle, USA</span>
	</p>
	<img width="33%" data-src="/images/talks/ncov_d614g_ct_sheffield.png">
	<img width="50%" data-src="/images/talks/ncov_d614g_ct_wa.png">
	<div class="citation">
		<a href="https://www.sciencedirect.com/science/article/pii/S0092867420308205">Korber et al. Cell,</a>
		<a href="/papers/muller-ncov-wa-d614g/">MÃ¼ller et al. 2020. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Repeated introductions to the UK suggest transmission advantage of D614G</h3>
	<img class="stretch" data-src="/images/talks/ncov_d614g_rt_introductions_uk.png">
	<div class="citation">
		<a href="https://www.medrxiv.org/content/10.1101/2020.07.31.20166082v1">Volz et al. medRxiv.</a>
	</div>
</section>

<section style="text-align: left;">
	<h3>Advancing genomic epidemiology</h3>
	<p>
	<ul>
		<li>Better methods for large datasets</li>
		<li>Distinguishing endogenous spread from importations</li>
		<li>Tying genomic epidemiology together with richer epi data to better understand local transmission</li>
		<li>Incorporating within-host variation to improve phylogenetic resolution</li>
		<li>Integrating clinical data to look for mutations that impact clinical outcomes</li>
	</ul>
	</p>
</section>

<section>
	<h3>
		Surveillance is critically important to our ability to combat epidemics and pandemics
	</h3>
</section>

<section>
	<div height=500">
		<img width="20%" style="padding-bottom: 15px; vertical-align: middle;" src="/images/talks/seattleflu_logo_y2.png">
		&nbsp;
		<span style="vertical-align: middle; font-size: 2.22em;">Seattle Flu Study</span>
	</div>
	<p class="smaller">
		<b>Lead investigators</b>: Helen Chu, Michael Boeckh, Janet Englund, Michael
		Famulare, Barry Lutz, Deborah Nickerson, Mark Rieder, Lea Starita, Matthew
		Thompson, Jay Shendure, Trevor Bedford
	</p>
	<p class="smaller">
		<b>Co-investigators</b>: Amanda Adler, Elisabeth Brandstetter, Roy Burstein,
		Shari Cho, Anne Emanuels, Kairsten Fay, Chris Frazar, Rachel Geyer, Peter
		Han, James Hadfield, Jessica Heimonen, Misja Ilcisin, Michael Jackson,
		Anahita Kiavand, Ashley Kim, Louise Kimball, Jack Henry Kotnik, Kirsten
		Lacombe, Jover Lee, Jennifer Logue, Victoria Lyon, Denise McCulloch, Jessica
		Oâ€™Hanlon, Matthew Richardson, Julia Rogers, Thomas Sibley, Monica Zigman
		Suchsland, Melissa Truong, Caitlin Wolf, Weizhi Zhong
	</p>
	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="12%" data-src="/images/talks/logo_bbi_vertical.png">
		&nbsp; &nbsp;
		<img width="15%" data-src="/images/talks/logo_uw_medicine.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_seattle_childrens.png">
	</div>
</section>

<section>
	<img class="stretch" src="/images/talks/seattleflu_overview.png">
</section>

<section>
	<img class="stretch" src="/images/talks/seattle_flu_enrollment.png">
</section>

<section>
	<h3>Phylogeny of 1150 Seattle H3N2 viruses</h3>
	<img class="stretch" src="/images/talks/seattleflu_h3n2_tree_geo.png">
</section>

<section>
	<h3>Screening of acute respiratory infections for SARS-CoV-2</h3>
	<img class="stretch" src="/images/talks/ncov_cryptic_transmission_prevalence.png">
	<div class="citation">
		<a href="https://bedford.io/papers/bedford-ncov-cryptic-transmission/">Bedford et al. Science.</a>
	</div>
</section>

<section>
	<h3>Sequencing of viruses collected prior to March 15 detects origins and rate of local spread</h3>
	<img class="stretch" src="/images/talks/ncov_cryptic_transmission_introductions.png">
	<div class="citation">
		<a href="https://bedford.io/papers/bedford-ncov-cryptic-transmission/">Bedford et al. Science.</a>
	</div>
</section>

<section>
	<h3>Sequencing of viruses collected prior to March 15 detects origins and rate of local spread</h3>
	<img class="stretch" src="/images/talks/ncov_cryptic_transmission_outbreak_clade.png">
	<div class="citation">
		<a href="https://bedford.io/papers/bedford-ncov-cryptic-transmission/">Bedford et al. Science.</a>
	</div>
</section>

<section>
	<img width="45%" data-src="/images/talks/ncov_cryptic_transmission_nyt.png">
	<img width="45%" data-src="/images/talks/ncov_cryptic_transmission_stat.png">
</section>

<section>
	<img class="stretch" src="/images/talks/ncov_scan_splash.png">
</section>

<section>
	<h3>Prevalence estimates and supplemental testing</h3>
	<img class="stretch" src="/images/talks/ncov_scan_dashboard_2020_10_20.png">
</section>

<section style="text-align: left;">
	<h3>Continued SFS activities</h3>
	<p>
	<ul>
		<li>SCAN platform for distributed testing</li>
		<li>Improvements to cost and throughput of SARS-CoV-2 testing</li>
		<li>Testing for mitigation with Husky Coronavirus Testing</li>
		<li>Continued investigation of circulation patterns across common respiratory pathogens</li>
		<li>Further integration of genome sequencing for epidemiological insights</li>
	</ul>
	</p>
</section>

<section>
	<h3>Circulation of everything except COVID-19 and rhinovirus basically disappeared</h3>
	<img class="stretch" src="/images/talks/seattleflu_pathogen_prevalence_2020_09_10.png">
</section>

<section>
	<h3>State-level transmission patterns, moving towards more detailed epi metadata</h3>
	<img class="stretch" src="/images/talks/ncov_wa_d614g_tree_county.png">
	<div class="citation">
		<a href="/papers/muller-ncov-wa-d614g/">MÃ¼ller et al. 2020. medRxiv.</a>
	</div>
</section>

<!-- Big picture closing about COVID and/or future pandemics -->

<section style="text-align: left;">
	<h3>Closing thoughts</h3>
	<p>
	<ul>
		<li>Pandemic warning systems need to be constructed on top of burden of endemic / seasonal pathogens</li>
		<li>Addressing need of endemic / seasonal diseases provides sample flows for detection of novel pathogens</li>
		<li>I don't think we need to be sequencing every acute respiratory infection, but we should be sequencing clusters and/or infections of unknown etiology</li>
		<li>Will require global surveillance / reporting to achieve</li>
	</ul>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="tiny">
		<b>Seasonal flu</b>: WHO Global Influenza Surveillance Network, GISAID,
		John Huddleston, Richard Neher, Jover Lee, Barney Potter, Dave Wentworth,
		Becky Garten
	</p>
	<p class="tiny">
		<b>Ebola in DRC</b>: James Hadfield, Allison Black, Eddy Kinganda Lusamaki,
		Placide Mbala-Kingebeni, Catherine Pratt, Mike Wiley, Jean-Jacques Muyembe Tamfum,
		Steve Ahuka-Mundeke, Daniel Mukadi, Gustavo Palacios, Amadou Sall, Ousmane Faye,
		Eric Delaporte, Martine Peeters, David Blazes, Cecile Viboud, David Spiro
	</p>
	<p class="tiny">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>
	<p class="tiny">
		<b>Seattle Flu Study</b>: Helen Chu, Michael Boeckh, Janet Englund, Michael
		Famulare, Barry Lutz, Deborah Nickerson, Mark Rieder, Lea Starita, Matthew
		Thompson, Jay Shendure, Amanda Adler, Jeris Bosua, Elisabeth Brandstetter,
		Kairsten Fay, Chris Frazar, Peter Han, Reena Gulati, James Hadfield, ShiChu
		Huang, Misja Ilcisin, Michael Jackson, Anahita Kiavand, Louise Kimball, Enos
		Kline, Kirsten Lacombe, Jover Lee, Jennifer Logue, Victoria Lyon, Kira Newman,
		Miguel Paredes, Thomas Sibley, Monica Zigman Suchsland, Cassia Wagner, Caitlin
		Wolf
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/kairsten-fay.jpg"> Kairsten Fay,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/misja-ilcisin.jpg"> Misja Ilcisin,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola MÃ¼ller,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/eli-harkins.jpg"> Eli Harkins</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>
