---
layout: talk_reveal3
title: Evolutionary dynamics of SARS-CoV-2
venue: AGBT General Meeting
author: Trevor Bedford
widescreen: true
---

<!-- Total time of 30 minutes, suggested 25 min talk with 5 min Q&A -->

<!--
Outline:

- Evolutionary dynamics overview (Muller plots and evolution driving epidemiology)
- Tracking evolution via Nextstrain
- Early pandemic spread
- D614G and summer/fall variants
- Variants of concern, emergence and spread
- Current situation update
- Forecasts for antigenic evolution
- Scale of modern genomic surveillance (again comparisons to influenza)
- Data-driven vaccine updating
-->

<section data-background-iframe="/talks/viz/gradient/">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	Associate Professor, Fred Hutchinson Cancer Research Center
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<section>
	<h3>Rapid replacement of existing diversity by lineage B.1.1.7</h3>
	<img class="stretch" data-src="/images/talks/ncov_frequencies_b117.png">
</section>

<section class="left-align">
	<h3>1. Real-time tracking of SARS-CoV-2 evolution</h3>
	<h3>2. Emergence of variants of concern</h3>
	<h3>3. Current circulation patterns</h3>
	<h3>4. Expectations for antigenic evolution</h3>
	<h3>5. Scale of genomic surveillance</h3>
</section>

<section data-background="#000000">
	<h1 class="title">Real-time tracking of SARS-CoV-2 evolution</h1>
</section>

<section>
	<h3>Spike protein is critical for cell invasion by the virus and is the primary target for human immune response</h3>
	<img class="stretch" data-src="/images/talks/sars_cov_2_spike_structure.png">
	<div class="citation">
		<a href="https://science.sciencemag.org/content/367/6483/1260">Wrapp et al. 2020. Science.</a>
	</div>
</section>

<section>
	<h3>Over 600k SARS-CoV-2 genomes shared to GISAID and evolution tracked in real-time at nextstrain.org</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_map_2021_01_12_global.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>SARS-CoV-2 lineages establish globally in February and March</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_early_pandemic.png">
	<div class="citation">
		<a href="/papers/bedford-ncov-cryptic-transmission/">Bedford et al. 2020. Science.</a>
	</div>
</section>

<section>
	<h3>Limited early mutations like D614G spread globally during initial wave</h3>
	<img class="stretch" src="/images/talks/ncov_d614g_european_introduction.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h4 class="inset-top-right">
		After initial wave, with mitigation <br> efforts and decreased travel, <br> regional clades emerge
	</h4>
	<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_global_jun_oct.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Summer and fall mutations were confined to regional dominance</h3>
	<img class="stretch" data-src="/images/talks/ncov_a222v_tree_map.jpg">
	<div class="citation">
		<a href="https://www.medrxiv.org/content/10.1101/2020.10.25.20219063v1">Hodcroft et al. 2020. medRxiv.</a>
	</div>
</section>

<section data-background="#000000">
	<h1 class="title">Emergence of variants of concern</h1>
</section>

<!-- The S1 subunit contains a receptor-binding domain that recognizes and binds to the
host receptor angiotensin-converting enzyme 2 ACE-2, while the S2 subunit mediates viral
cell membrane fusion by forming a six-helical bundle via the two-heptad repeat domain. -->

<section>
	<h3>484K and 501Y repeatedly emerging across the world</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_484K_501Y_2021_02_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_484,501">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Emergence of 501Y.V1 (B.1.1.7) in the UK</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_501Y.V1_clock_2021_02_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/groups/neherlab/ncov/united-kingdom?c=clade_membership&f_country=United%20Kingdom&l=clock&r=division">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Emergence of 501Y.V2 (B.1.351) in the South Africa</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_501Y.V2_clock_2021_02_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/africa?f_country=South%20Africa&l=clock&r=division">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Emergence of 501Y.V3 (P.1) in the Brazil</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_501Y.V3_clock_2021_02_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/south-america?f_country=Brazil&l=clock&r=division">nextstrain.org</a>
	</div>
</section>

<section>
	<div class="left" style="float:left; width:54%">
		<h3>Substantial convergent evolution</h3>
	</div>
	<div class="right" style="float:right; width:44%">
		<img data-src="/images/talks/ncov_variants_shared_mutations_2021_02_05.png">
	</div>
	<div class="citation">
		<a href="https://covariants.org">covariants.org</a>
	</div>
</section>

<section class="left-align">
	<h3>
		Working hypothesis of within-host evolution occurring during prolonged infection,
		driven by natural selection for immune escape.
	</h3>
</section>

<section>
	<h3>Rapid within-host evolution during persistent infection</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_choi_chronic_tree.png">
	<div class="citation">
		<a href="https://www.nejm.org/doi/full/10.1056/NEJMc2031364">Choi et al. 2020. NEJM.</a>
	</div>
</section>

<section>
	<h3>484K and 501Y observed during this evolution</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_choi_chronic_alignment.png">
	<div class="citation">
		<a href="https://www.nejm.org/doi/full/10.1056/NEJMc2031364">Choi et al. 2020. NEJM.</a>
	</div>
</section>

<section data-background="#000000">
	<h1 class="title">Current circulation patterns</h1>
</section>

<section>
	<h3>VOCs are growing in frequency with B.1.1.7 leading the curve</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_484K_501Y_2021_02_10.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_484,501">nextstrain.org</a>
	</div>
</section>

<section data-transition="slide-in none-out">
	<h3>Repeated convergent evolution across sites in spike (452R)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_452R_2021_02_16.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_452">nextstrain.org</a>
	</div>
</section>

<section data-transition="none">
	<h3>Repeated convergent evolution across sites in spike (681H)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_681H_2021_02_16.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_681">nextstrain.org</a>
	</div>
</section>

<section data-transition="none">
	<h3>Repeated convergent evolution across sites in spike (69del)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_69del_2021_02_16.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_69">nextstrain.org</a>
	</div>
</section>

<section data-transition="none-in slide-out">
	<h3>Repeated convergent evolution across sites in spike (combined)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_across_spike_2021_02_16.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/global?c=gt-S_18,69,152,452,501,677,681">nextstrain.org</a>
	</div>
</section>

<section data-background="#000000">
	<h1 class="title">Expectations for antigenic evolution</h1>
</section>

<section>
	<h3>Up to December, my expectation was evolution as seen in seasonal coronaviruses</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_adaptation_accumulation_oc43_elife.png">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2021. eLife.</a>
	</div>
</section>

<section>
	<h3>OC43 and 229E show flu B-like rates of adaptive evolution in S1</h3>
	<img class="stretch" data-src="/images/talks/seasonal_cov_adaptation_rate_comparison_elife.jpg">
	<div class="citation">
		<a href="/papers/kistler-hcov-adaptive-evolution/">Kistler and Bedford. 2021. eLife.</a>
	</div>
</section>

<section>
	<h3>~23 mutations per year across SARS-CoV-2 genome</h3>
	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_nuc_clock_2021_02_05.png">
</section>

<section>

	<section>
		<h3>Substantial increase in spike S1 amino acid substitutions in VOCs</h3>
		<img class="stretch" data-src="/images/talks/ncov_mutation_rate_spike_clock_2021_02_05.png">
	</section>

	<section>
		<h3>Accumulation of substitutions faster in NTD than RBD</h3>
		<img class="stretch" data-src="/images/talks/ncov_mutation_rate_spike_ntd_rbd_clock_2021_02_05.png">
	</section>

</section>

<section>
	<h3>Spike S1 rate of 2.9 subs per year similar to rate in HA1 of flu A</h3>
	<img class="stretch" data-src="/images/talks/flu_mutation_rate_ha1_clock_across_subtypes.png">
</section>

<section>
	<h3>Influenza H3N2 drifts at ~1 two-fold HI dilution per year</h3>
	<img class="stretch" data-src="/images/talks/bmds_lineages_drift_stats.png">
	<div class="citation">
		<a href="http://bedford.io/papers/bedford-flux/">Bedford et al. 2014. eLife.</a>
	</div>
</section>

<section class="left-align">
	<h3>
		SARS-CoV-2 VOCs have evolved ~10 amino acid sites in S1 in just over a year.
		This is rapid even for influenza A. 501Y.V2 shows an ~8-fold drop in neutralization
		titer, equivalent to ~3 years of H3N2 evolution.
	</h3>
</section>

<section data-background="#000000">
	<h1 class="title">Genomic surveillance</h1>
</section>

<section>
	<h3>Sequences generated and shared at an unprecedented pace with 125k genomes shared in Jan 2021 alone</h3>
	<img class="stretch" data-src="/images/talks/ncov_genomic_surveillance_dotplot_global_2021_02_25.png">
	<div class="citation">
		Data from gisaid.org
	</div>
</section>

<section>
	<h3>My favorite metric is number of sequences available from samples collected in the past 30 days</h3>
	<img class="stretch" data-src="/images/talks/ncov_genomic_surveillance_recent_sequences_global_2021_02_25.png">
	<div class="citation">
		Data from gisaid.org
	</div>
</section>

<section>
	<h3>Front-runners of the UK, Denmark and the US have dramatically increased throughput, but other regions lag</h3>
	<img class="stretch" data-src="/images/talks/ncov_genomic_surveillance_recent_sequences_comparison_2021_02_25.png">
	<div class="citation">
		Data from gisaid.org
	</div>
</section>

<section class="left-align">
	<h3>
		If the goal is early detection of emerging variants to lay groundwork for
		vaccine updates, another 1000 genomes per month would be far more valuable
		from South America or Africa than 1000 more genomes from the US or the UK
	</h3>
</section>

<section class="left-align">
	<h3>Generally, I suspect we're looking at a data-rich version of the influenza strain selection process</h3>
	<p>
	<ol>
		<li>mRNA vaccines allow faster turnarounds and may push for more regional strategies</li>
		<li>Work directly from human serological data rather than naive animals</li>
		<li>Multiple valencies possible, clinical trials should be conducted now comparing regimens</li>
	</ol>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="tiny">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>
	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/allison-black.jpg"> Alli Black,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/kairsten-fay.jpg"> Kairsten Fay,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/misja-ilcisin.jpg"> Misja Ilcisin,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola MÃ¼ller,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span>
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/eli-harkins.jpg"> Eli Harkins</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_osp.jpg">
	</div>
</section>
