---
layout: talk_reveal4
title: Continuing SARS-CoV-2 evolution under population immune pressure
venue: FDA
event: Vaccines and Related Biological Products Advisory Committee Meeting
author: Trevor Bedford
widescreen: true
draft: true
---

<!-- ~10 min -->

<!--
- talk about modeling of variants and how they emerge, and likelihood of emerging
- to discuss considerations for use of COVID-19 vaccine booster doses and the process for COVID-19 vaccine strain selection to address current and emerging variant viruses
- want to know about Omicron, but it is more looking forward about what we should expect for viral evolution (not that anyone can confidently state this)
- they will be thinking about how to pick strains for the fall, considering Omicron and whatever comes next
-->

<!--
Outline:
- overview of variant emergence and spread
- SARS-CoV-2 shows substantial adaptive potential
- continued variant emergence and high burden of disease at endemicity
- strategies for vaccine strain selection
-->

<!-- Color ramp: [#5097BA", "#60AA9E", "#75B681", "#8EBC66", "#AABD52", "#C4B945", "#D9AD3D", "#E59637", "#E67030", "#DF4327"] -->

<section data-transition="fade" data-background="#000">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	Fred Hutchinson Cancer Center / Howard Hughes Medical Institute
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<section class="left-align">
	<h3>Disclosures</h3>
	<p>
	<ul>
		<li>
			I receive grant support from the National Institutes of Health and the Howard Hughes Medical Institute to research methods for evolutionary forecasting of SARS-CoV-2.
		</li>
	</ul>
	</p>
</section>

<!-- <section data-transition="fade" data-background="#5097BA">
	<h1 class="title">SARS-CoV-2 evolution</h1>
</section> -->

<section>
	<h3>Repeated emergence of variant of concern viruses</h3>
	<div class="left" style="float:left; width:49%">
		<img data-src="/images/talks/ncov_variants_501Y.V1_clock_2021_02_10.png">
	</div>
	<div class="right" style="float:right; width:49%">
		<img data-src="/images/talks/ncov_variants_501Y.V3_clock_2021_03_16.png">
	</div>
</section>

<section>
	<h3>Rapid adaptive evolution focused on S1 subunit of spike protein</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_dnds_h3n2.png">
	<div class="citation">
		<a href="/papers/kistler-sarscov2-adaptive-evolution/">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section>

<section>
	<h3>Genetic relationships of globally sampled SARS-CoV-2 to present</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_unrooted_2022_03_13.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?l=unrooted&m=div">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Rapid displacement of existing diversity by emerging variants</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_frequencies_2022_03_13.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Omicron shows particular excess of mutations at S1</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_s1_mutations_2022_02_20.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?l=scatter&scatterY=S1_mutations">nextstrain.org</a>
	</div>
</section>

<!-- Update with new adaptive evolution run when complete -->
<section>
	<h3>S1 evolved at a rate of 12 amino acid changes per year in 2021</h3>
 	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_spike_clock_2022_02_20.png">
</section>

<section>
	<h3>This is remarkably fast relative to seasonal influenza</h3>
 	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_compared_to_flu_2022_02_20.png">
</section>

<!-- <section class="left-align">
	<div class="left" style="float:left; width:62%">
		<h3>Primary hypotheses for Omicron's origin</h3>
		<p>
		<ol>
			<li>
				<i>(More likely) Evolution during chronic infection in an immunocompromised individual</i>
				<ul>
					<li>Supported by large excess of S1 mutations, but paucity of non-spike mutations</li>
					<li>Hard to explain simultaneous BA.1, BA.2 and BA.3 sublineages otherwise</li>
				</ul>
			</li>
			<li>
				<i>(Less likely) Human to animal spillover mid-2020 and spillback in late 2021</i>
				<ul>
					<li>Mutational spectra and particular mutations such as 493R and 498R suggest rodent intermediary (Wei et al)</li>
				</ul>
			</li>
		</ol>
		</p>
	</div>
	<div class="right" style="float:right; width:35%">
		<img data-src="/images/talks/ncov_omicron_sublineages_tree.png">
	</div>
</section> -->

<section>
	<h3>Omicron spike mutations substantially drop VE against infection</h3>
 	<img class="stretch" data-src="/images/talks/ncov_omicron_ve_ukhsa_2022_01_14.png">
	<div class="citation">
 		<a href="https://www.gov.uk/government/publications/investigation-of-sars-cov-2-variants-technical-briefings">UKHSA. 2022. Technical Report.</a>
 	</div>
</section>

<!-- Update with new Rt results when complete -->
<section>
	<h3>Significant immune escape drove large-scale epidemics</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_rt_log_cases_2022_03_13.png">
	<div class="citation">
		<a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2021. medRxiv.</a>
	</div>
</section>

<!-- <section>
	<h3>Starting Rt approached that of the initial wave in spring 2020</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_rt_2022_03_13.png">
	<div class="citation">
		<a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2021. medRxiv.</a>
	</div>
</section> -->

<section class="left-align">
	<h3>Omicron attack rate</h3>
	<p>
	<ol>
		<li>
			We estimate US has seen 9.8% of the population as confirmed cases of Omicron through Mar 1, with the
			large majority accumulating after Dec 15
		</li>
		<li>
			Assuming a case detection rate of 1 in 5 infections, we estimate almost 50% of the US infected with Omicron, most in the span of ~10 weeks
		</li>
	</ol>
	</p>
</section>

<!-- <section>
	<h3>Reduced intrinsic severity relative to other VOC viruses</h3>
	<img class="stretch" data-src="/images/talks/ncov_severity_variants_wa_2022_02_16.png">
	<div class="citation">
		<a href="/papers/paredes-sarscov2-variant-outcomes/">Paredes et al. 2022. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Immunity and reduced intrinsic severity has dropped CFR</h3>
	<img class="stretch" data-src="/images/talks/ncov_severity_lagged_cfr_prediction_2022_03_29.png">
</section> -->

<!-- <section data-transition="fade" data-background="#60AA9E">
	<h1 class="title">Future perspectives</h1>
</section> -->

<section>
	<h3>SARS-CoV-2 will continue to evolve to escape population immunity, though with multiple potential avenues</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_tree_2022_03_13_scenarios.png">
</section>

<section>
	<h3>With 1 observation in 2.35 years of virus evolution, it's currently unclear how common Omicron-like events will be</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_wait_time_distribution_2022_03_21.png">
</section>

<section>
	<h3>This rate distribution gives a naive prediction of Omicron-like emergence events occurring in the next 12 months</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_prediction_distribution_2022_03_21.png">
</section>

<section class="left-align">
	<h3>Likely scenarios over the next 12 months</h3>
	<p>
	<ol>
		<li>
			(More likely) Evolution within Omicron BA.2 to further increase intrinsic transmission and to escape from Omicron-derived immunity. This scenario sees lower attack rates with 2022-2023 epidemic driven by drift + waning + seasonality.
		</li>
		<li>
			(Less likely) Another Omicron-like emergence event in which a chronic infection initiated in ~2021 incubates a new wildly divergent virus. This scenario sees high attack rates with epidemic driven by variant emergence.
		</li>
	</ol>
	</p>
</section>

<!-- <section class="left-align">
	<h3>Broad expectations of endemicity based on comparison with flu</h3>
	<p>
	<ol>
		<li>
			<i>R</i><sub class="even-smaller">0</sub> of Omicron is >6 compared to
			<i>R</i><sub class="even-smaller">0</sub> of flu of ~2. At same rates of evolution and waning
			expect more COVID circulation.
		</li>
		<li>
			Before Omicron rates of evolution in SARS-CoV-2 S1 had been similar to H3N2 flu.
			Omicron equivalent to 5 years of H3N2 evolution.
		</li>
		<li>
			Infection fatality rate (IFR) of COVID is roughly comparable to flu once you have prior immunity.
		</li>
	</ol>
	</p>
	<p>
		1, 2 and 3 suggest a virus that circulates at higher levels than flu, but individual
		infections aren't much more severe in terms of mortality than H3N2 influenza.
	</p>
	<p>
		Seasonality suggests winter "COVID season", but Omicron-like events could overcome seasonality.
	</p>
</section> -->

<!-- <section data-transition="fade" data-background="#75B681">
	<h1 class="title">Vaccine strain selection</h1>
</section> -->

<!-- <section>
	<h3>Influenza vaccine strain selection strategy</h3>
	<p>
		General strategy for antigenically evolving seasonal influenza viruses is attempt to match vaccine strain as close as possible to circulating viruses
	</p>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines_2014_2022.png">
	<div class="citation">
		<a href="https://nextstrain.org/flu/seasonal/h3n2/ha/6y">nextstrain.org</a>
	</div>
</section> -->

<!-- <section>
	<h3>Influenza vaccine strain selection timeline</h3>
	<p>
		Due to manufacturing and distribution, vaccine strain selection occurs in Feb for an fall vaccination campaign ahead of seasonal influenza epidemic
	</p>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section> -->

<!-- <section class="left-align">
	<h3>Considerations for SARS-CoV-2 vaccine strain selection</h3>
	<p>
	<ol>
		<li>
			SARS-CoV-2 will continue to evolve under pressure to escape from population immunity with a baseline expectation of a rate similar to H3N2 flu
		</li>
		<li>
			Seasonality will always be present and will likely drive an annual winter COVID season
		</li>
		<li>
			We likely need yearly boosters to sustain protection against infection and disease and should time booster campaign to immediately precede winter season
		</li>
		<li>
			We should have a Northern Hemisphere decision sometime between ~April and ~June (given lead time required for mRNA platform, I don't know what these are however)
		</li>
	</ol>
	</p>
</section> -->

<!-- <section>
	<h3>Higher titers from Omicron-specific boost in mice</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_titers_ying_omicron.png">
	<div class="citation">
		<a href="https://www.cell.com/cell/fulltext/S0092-8674(22)00387-7">Ying et al. 2022. Cell.</a>
	</div>
</section> -->

<!----------------------------------------------------------------------------------->

<section>
	<h3>Acknowledgements</h3>

	<p class="even-smaller">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola MÃ¼ller,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/denisse-sequeira.jpg"> Denisse Sequeira,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jennifer-chang.jpg"> Jennifer Chang</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
	</div>
</section>
