---
layout: talk_reveal4
title: Directions for SARS-CoV-2 evolutionary forecasting
venue: Fred Hutch
event: Comp Bio Faculty Lunch
author: Trevor Bedford
widescreen: true
draft: true
---

<!-- ~40 min -->

<!--
Outline:

- SARS-CoV-2 vaccine strain selection will be necessary
  - VRBPAC meeting and where we currently stand
- Influenza strain selection and history of fitness models
- SARS-CoV-2 and amazing consistency of frequency dynamics
- Multinomial logistic regression models
- Marlin's variant Rt work
- Obermeyer hierarchical MLR work
- But new variants are appearing too quickly and so prediction threshold is poor\
  - Analogy to weather fronts
- Project direction to systematize prediction thresholds
- Project direction to automate MLR and Rt models on Nextstrain clades / PANGO lineages
- Project direction to revisit influenza forecasting using MLR models and Nextclade calls
-->

<!-- Color ramp: [#5097BA", "#60AA9E", "#75B681", "#8EBC66", "#AABD52", "#C4B945", "#D9AD3D", "#E59637", "#E67030", "#DF4327"] -->

<section data-transition="fade" data-background="#000">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<p class="title">
	Trevor Bedford (@trvrb)
	<br>
	Fred Hutchinson Cancer Research Center / Howard Hughes Medical Institute
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<!-- Introduction -->

<section data-transition-speed="slow" data-background="/images/talks/ncov_tree_unrooted_2021_09_11.png">
</section>

<!-- Revise -->
<section class="left-align">
	<h3>1. SARS-CoV-2 evolution and variant-specific vaccines</h3>
	<h3>2. Forecasting seasonal influenza evolution</h3>
	<h3>3. SARS-CoV-2 variant frequency dynamics</h3>
	<h3>4. Recent modeling advances</h3>
	<h3>5. Project directions</h3>
</section>

<!-- Need for SARS-CoV-2 variant vaccines -->

<section data-transition="fade" data-background="#5097BA">
	<h1 class="title">SARS-CoV-2 evolution and variant-specific vaccines</h1>
</section>

<section>
	<h3>Genetic relationships of globally sampled SARS-CoV-2 to present</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_unrooted_2022_03_13.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?l=unrooted&m=div">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Rapid displacement of existing diversity by emerging variants</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_frequencies_2022_03_13.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Omicron shows particular excess of mutations at S1</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_s1_mutations_2022_02_20.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global?l=scatter&scatterY=S1_mutations">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>S1 evolved at a rate of 12 amino acid changes per year in 2021</h3>
 	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_spike_clock_2022_02_20.png">
</section>

<section>
	<h3>This is remarkably fast relative to seasonal influenza</h3>
 	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_compared_to_flu_2022_02_20.png">
</section>

<section>
	<h3>Omicron spike mutations substantially drop VE against infection</h3>
 	<img class="stretch" data-src="/images/talks/ncov_omicron_ve_ukhsa_2022_01_14.png">
	<div class="citation">
 		<a href="https://www.gov.uk/government/publications/investigation-of-sars-cov-2-variants-technical-briefings">UKHSA. 2022. Technical Report.</a>
 	</div>
</section>

<section>
	<h3>Higher titers from Omicron-specific boost in mice</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_titers_ying_omicron.png">
	<div class="citation">
		<a href="https://www.cell.com/cell/fulltext/S0092-8674(22)00387-7">Ying et al. 2022. Cell.</a>
	</div>
</section>

<section>
	<img class="stretch" data-src="/images/talks/sarscov2_vrbpac_2022_04_06_weir.png">
	<div class="citation">
		<a href="https://www.fda.gov/advisory-committees/advisory-committee-calendar/vaccines-and-related-biological-products-advisory-committee-april-6-2022-meeting-announcement">Weir. 2022. Meeting presentation.</a>
	</div>
</section>

<section>
	<h3>Influenza vaccine strain selection strategy</h3>
	<p>
		General strategy for antigenically evolving seasonal influenza viruses is attempt to match vaccine strain as close as possible to circulating viruses
	</p>
	<img class="stretch" data-src="/images/talks/flu_h3n2_tree_vaccines_2014_2022.png">
	<div class="citation">
		<a href="https://nextstrain.org/flu/seasonal/h3n2/ha/6y">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Influenza vaccine strain selection timeline</h3>
	<p>
		Due to manufacturing and distribution, vaccine strain selection occurs in Feb for an fall vaccination campaign ahead of seasonal influenza epidemic
	</p>
	<img class="stretch" data-src="/images/talks/flu_vaccine_schedule.jpg">
</section>

<section class="left-align">
	<h3>Considerations for SARS-CoV-2 vaccine strain selection</h3>
	<p>
	<ol>
		<li>
			SARS-CoV-2 will continue to evolve under pressure to escape from population immunity with a baseline expectation of a rate similar to H3N2 flu
		</li>
		<li>
			Seasonality will always be present and will likely drive an annual winter COVID season
		</li>
		<li>
			We likely need yearly boosters to sustain protection against infection and disease and should time booster campaign to immediately precede winter season
		</li>
		<li>
			We should have a Northern Hemisphere decision sometime between ~April and ~June (guess at lead time required for mRNA platform)
		</li>
	</ol>
	</p>
</section>

<!-- Flu forecasting -->

<section data-transition="fade" data-background="#60AA9E">
	<h1 class="title">Forecasting seasonal influenza evolution</h1>
</section>

<section>
	<h3>Fitness models project strain frequencies</h3>
	<p>Future frequency $x_i(t+\Delta t)$ of strain $i$ derives from strain fitness $f_i$ and present day frequency $x_i(t)$, such that</p>
	<p>$$x_i(t+\Delta t) = \frac{1}{Z(t)} \, x_i(t) \, \mathrm{exp}(f_i \, \Delta t)$$</p>
	<p>
		Strain frequencies at each timepoint are normalized by total frequency $Z(t)$.
		This captures clonal interference between competing lineages.
	</p>
	<img class="stretch" data-src="/images/talks/clonal_interferance.jpg">
	<div class="citation">
		<a href="http://www.nature.com/nature/journal/v507/n7490/full/nature13087.html">&#321;uksza and L&auml;ssig. 2014. Nature.</a>
	</div>
</section>

<!-- <section>
	<h3>Integrating genotypes and phenotypes improves long-term forecasts of seasonal influenza A/H3N2 evolution</h3>
	<p class="smaller"><i>
	with <img width=40 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston, Richard Neher, Dave Wentworth, Becky Kondor, John McCauley, Hideki Hasegawa, Kanta Subbarao and others
	</i></p>
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section> -->

<section>
	<h3>Match strain forecast to retrospective circulation</h3>
	<img class="stretch" data-src="/images/talks/prediction_fitness_model_schematic.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section style="text-align: left;">
	<h3>Two inputs</h3>
	<p>
	<ul>
		<li>Estimate of present-day strain frequencies $x_i(t)$</li>
		<li>Estimate of present-day strain fitnesses $f_i$</li>
	</ul>
	</p>
</section>

<section>
	<h3>Strain frequency estimated via region-weighted KDE</h3>
	<img class="stretch" data-src="/images/talks/prediction_strain_frequencies.png">
</section>

<section>
	<h3>Strain fitness estimated from viral attributes</h3>
	<p>The fitness $f$ of strain $i$ is estimated as</p>
	<p>$$f_i = \beta^\mathrm{A} \, f_i^\mathrm{A} + \beta^\mathrm{B} \, f_i^\mathrm{B} + \ldots$$</p>
	<p>where $f^A$, $f^B$, etc... are different standardized viral attributes and
		$\beta^A$, $\beta^B$, etc... coefficients are trained based on historical evolution</p>
	<br>
	<table>
		<thead>
			<tr>
				<th width="28%">Antigenic drift</th>
				<th width="38%">Intrinsic fitness</th>
				<th width="33%">Recent growth</th>
			</tr>
		</thead>
		<tbody>
			<tr>
				<td>epitope mutations</td>
				<td>non-epitope mutations</td>
				<td>local branching index</td>
			</tr>
			<tr>
				<td>HI titers</td>
				<td>DMS data (via Bloom lab)</td>
				<td>delta frequency</td>
			</tr>
		</tbody>
	</table>
</section>

<!-- <section>
	<h3>Future population depends on frequency and fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_strain_projection.png">
</section> -->

<section>
	<h3>Forecast assessed based on weighted distance match to observed future population (earth mover's distance)</h3>
	<div class="left" style="float:left; width:49%">
		<p>Poor fit</p>
		<img data-src="/images/talks/prediction_weighted_distances_poor.png">
	</div>
	<div class="right" style="float:right; width:49%">
		<p>Good fit</p>
		<img data-src="/images/talks/prediction_weighted_distances_good.png">
	</div>
</section>

<section>
	<h3>Train in 6-year sliding windows from 1995 to 2015 with most recent years held out as test</h3>
	<img class="stretch" data-src="/images/talks/prediction_cross_validation_strategy_v2.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Composite models favor combinations of HI drift, local branching index and non-epitope fitness</h3>
	<img class="stretch" data-src="/images/talks/prediction_performance_composite.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<section>
	<h3>Model successfully predicts clade growth and best pick from model is generally close to best possible retrospective pick</h3>
	<img class="stretch" data-src="/images/talks/prediction_validation_clade_growth_ranking.png">
	<div class="citation">
		<a href="/papers/huddleston-flu-forecasting/">Huddleston et al. 2020. eLife.</a>
	</div>
</section>

<!-- <section>
	<h3>Forecasts are live at <a href="https://nextstrain.org/flu">nextstrain.org/flu</a>, but are currently fraught due to lack of recent isolates</h3>
	<img class="stretch" data-src="/images/talks/nextflu_oct2020_h3n2_tree_clades.png">
</section> -->

<section class="left-align">
	<h3>Two main issues</h3>
	<p>
	<ol>
		<li>
			We swapped from assessing clade frequencies to earth mover's distance because our clade assignments were not
			stable across trees built at different timepoints, while clade frequencies are the more natural metric.
		</li>
		<li>
			Strain fitness $f_i$ is largely fixed by the "fundamentals" of the strain rather than being learned from frequency behavior.
		</li>
	</ol>
	</p>
</section>

<!-- SARS-CoV-2 frequency dynamics -->

<section data-transition="fade" data-background="#75B681">
	<h1 class="title">SARS-CoV-2 variant frequency dynamics</h1>
</section>

<section>
	<h3>Population genetic expectation of variant frequency under selection</h3>
	<p>$x' = \frac{x \, (1+s)}{x \, (1+s) + (1-x)}$ for frequency $x$ over one generation with selective advantage $s$</p>
	<p>$x(t) = \frac{x_0 \, (1+s)^t}{x_0 \, (1+s)^t + (1-x_0)}$ for initial frequency $x_0$ over $t$ generations</p>
	<p>Trajectories are linear once logit transformed via $\mathrm{log}(\frac{x}{1 - x})$</p> 
	<img class="stretch" src="/images/talks/ncov_variants_selection_logit_trajectories.png">
</section>

<section>
	<h3>Variants show consistent frequency dynamics in logit space</h3>
	<img class="stretch" src="/images/talks/ncov_variants_omicron_countries_frequencies_logit_2022_04_08.png">
</section>

<section>
	<h3>Variants show consistent frequency dynamics in logit space</h3>
	<img class="stretch" src="/images/talks/ncov_variants_delta_countries_frequencies_logit_2021_09_07.png">
</section>

<section>
	<h3>Variants show consistent frequency dynamics in logit space</h3>
	<img class="stretch" src="/images/talks/ncov_variants_delta_states_frequencies_logit_2021_09_07.png">
</section>

<section>
	<h3>Multinomial logistic regression</h3>
	<p>
		Multinomial logistic regression models the probability of a virus sampled at time $t$ belonging to variant $i$ as
	</p>
	<p>
		$$\mathrm{Pr}(X = i) = x_i(t) = \frac{p_i \, \mathrm{exp}(f_i \, t)}{\sum_{1 \le j \le n} p_j \, \mathrm{exp}(f_j \, t) }$$
	</p>
	<p>
		where the model has $2n$ parameters consisting of $p_i$ the frequency of variant $i$ at initial timepoint and $f_i$ the
		growth rate or fitness of variant $i$.
	</p>
	<p>
		The model is fit to minimize "log loss" of predicted variant class vs observed variant class across dataset.
	</p>
</section>

<section>
	<h3>Multinomial logistic regression fits variant frequencies well</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_rt_logistic_regression_2021_09_01.png">
</section>

<section class="left-align">
	<h3>Multinomial logistic regression should work well for SARS-CoV-2 prediction, except new variants have been emerging
		fast enough that the prediction horizon is really quite short</h3>
</section>

<!-- Recent modeling advances -->

<section data-transition="fade" data-background="#8EBC66">
	<h1 class="title">Recent modeling advances</h1>
</section>

<section>
	<h3>Hierarchical MLR model incorporates specific mutations to predict fitness of Pango lineages</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_mutational_fitness_obermeyer_model.png">
	<div class="citation">
		<a href="https://www.medrxiv.org/content/10.1101/2021.09.07.21263228">Obermeyer et al. 2022. medRxiv.</a>
	</div>
</section>

<section>
	<h3>However, mutational makeup did not predict Omicron fitness</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_mutational_fitness_2022_01_16.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/2022-01-17?branchLabel=none&l=scatter&scatterY=mutational_fitness">nextstrain.org</a>
	</div>
</section>

<section class="left-align">
	<div class="left" style="float:left; width:35%">
		<h3>Estimation of variant-specific Rt through time using state-level data</h3>
		<p>State-level case counts are partitioned based on frequencies of sequenced cases</p>
	</div>
	<div class="right" style="float:right; width:60%">
		<img data-src="/images/talks/sarscov2_variant_rt_method_wa_2021_09_01.png">
	</div>
	<div class="citation">
		<a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2021. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Omicron BA.2 has been steadily displacing BA.1 across the world</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_rt_log_cases_split_2022_03_13.png">
	<div class="citation">
		<a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2021. medRxiv.</a>
	</div>
</section>

<section>
	<h3>BA.2 shows a transmission advantage of ~30% over BA.1</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_rt_split_2022_03_13.png">
	<div class="citation">
		<a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2021. medRxiv.</a>
	</div>
</section>

<!-- Project directions -->

<section data-transition="fade" data-background="#AABD52">
	<h1 class="title">Project directions</h1>
</section>

<section class="left-align">
	<h3>Project directions</h3>
	<p>
	<ol>
		<li>
			Revisit influenza forecasting using MLR models and Nextclade calls
		</li>
		<li>
			(With Matsen Lab) Simulate sequence evolution and antigenic dynamics to provide synthetic data for forecasting models
		</li>
		<li>
			(With Bloom Lab) Use DMS data to inform prediction of intrinsic strain fitness
		</li>
		<li>
			Quantify prediction horizon for MLR model (frequencies) and for variant Rt model (case counts and frequencies) through
			accuracy assessment of sliding window forecasts
		</li>
		<li>
			Automate MLR and variant Rt models to run on Nextstrain clades / Pango lineages and create a dashboard at something
			like forecasts.nextstrain.org
		</li>
	</ol>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="even-smaller">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/louise-moncla.jpg"> Louise Moncla,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/maya-lewinsohn.jpg"> Maya Lewinsohn,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola MÃ¼ller,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/denisse-sequeira.jpg"> Denisse Sequeira,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=35 class="img-avatar" src="/images/team/jennifer-chang.jpg"> Jennifer Chang</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
	</div>
</section>
