---
layout: talk_reveal4
title: Evolutionary dynamics of SARS-CoV-2
venue: ASV Annual Meeting
event: Symposium on Understanding How Viruses Spread and Evolve
author: Trevor Bedford
widescreen: true
---

<!-- 30 min total, no Q&A -->

<!--
Outline

1. Variant viruses introduction
 - variant viruses and population turnover
 - what does this mean for vaccination and burden of disease?

2. Forecasting seasonal influenza evolution (maybe)

3. SARS-CoV-2 genome evolution
 - mutation accumulation and clade growth
 - rate of evolution by dN/dS
 - comparison of rate to other antigenically evolving viruses

4. Variant frequency dynamics
 - SARS-CoV-2 and amazing consistency of frequency dynamics
 - multinomial logistic regression models
 - growth advantages of recent variants
 - Obermeyer hierarchical MLR work
 - (ideally) comparison of growth advantages to influenza

5. Evolution driving epidemics
 - variant Rt model
 - overall incidence from UK Infection Survey
 - describe seasonality-gating vs immunity-gating
 - potential for seasonality driven epidemics

6. Forecasting
 - MLR models generate accurate short-term forecasts
 - but new variants are appearing too quickly and so prediction threshold is poor
 - analogy to weather fronts
 - DMS escape prior and improved forecasting horizon

7. Big picture of continued evolution
 - potential for another Omicron-like event
   - based on simple statistical model
   - evidence from persistent infections and cryptic lineages
 - should we expect evolutionary rate to slow as immunity broadens?
 - expectations for continued incidence and burden of disease

8. Research priorities going forward
 - continued genomic surveillance and modeling
 - development of new antivirals
 - vaccination strategies to combat antigenic evolution and OAS
	 - universal vaccines?
   - cocktail vaccines?
	 - reducing imprinted response?

-->


<!-- Color ramp: ["#4377CD", "#61AB9D", "#94BD61", "#CDB642", "#E68133"] -->

<!-- This lectureship was established by John Holland's former trainees and colleagues in 2014 to
honor his pioneering research discoveries and his legacy as an outstanding mentor. John made major
contributions to our fundamental understanding of positive- and negative-strand RNA virus molecular
biology. His work on RNA virus evolution led directly to the quasispecies concept for RNA virus
populations. -->

<!-- I'd like to first acknowledge the John J. Holland Lectureship. John Holland's pioneering research
lead directly to the quasispecies concept for RNA virus evolution -->

<section data-transition="fade" data-background="#000">
	<h2 class="title">{{ page.title }}</h2>
	<br>&nbsp;<br>
	<h4 class="title">Trevor Bedford</h4>
	<p class="title">
	Fred Hutchinson Cancer Center / Howard Hughes Medical Institute
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	John J. Holland Lecture
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<!-- Variant viruses introduction -->

<section>
	<h3>Population immunity has driven a dramatic shift in death toll</h3>
	<img class="stretch" data-src="/images/talks/ncov_deaths_owid_2023_06_17.png">
	<div class="citation">
		<a href="https://ourworldindata.org/covid-deaths">ourworldindata.org</a>
	</div>
</section>

<section>
	<h3>But this immunity is now driving evolution of the virus</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_unrooted_2023_06_18.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18?l=unrooted">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>In which new variants emerge that escape from existing population immunity and spread rapidly</h3>
	<img class="stretch" data-src="/images/talks/ncov_frequencies_2023_06_18.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Novel variants sweep globally in months rather than years</h3>
	<div class="left" style="float:left; width:20%; margin-top: 15px;">SARS-CoV-2</div>
	<img width="80%" data-src="/images/talks/ncov_frequencies_2023_06_18.png">
	<div class="left" style="float:left; width:20%; margin-top: 15px;">Influenza H3N2</div>
	<img width="80%" data-src="/images/talks/flu_h3n2_frequencies_2023_06_15.png">
	<div class="citation">
		<a href="https://nextstrain.org/flu/seasonal/h3n2/ha/3y">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>We really should be at something like Psi at this point</h3>
	<img class="stretch" data-src="/images/talks/ncov_frequencies_greek_letters_2023_06_18.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18">nextstrain.org</a>
	</div>
</section>

<section class="left-align">
	<h3>1. SARS-CoV-2 genome evolution</h3>
	<h3>2. Variant frequency dynamics</h3>
	<h3>3. Evolution driving epidemics</h3>
	<h3>4. Forecasting</h3>
	<h3>5. Continued evolution</h3>
</section>

<!-- SARS-CoV-2 genome evolution -->

<section data-transition="fade" data-background="#4377CD">
	<h1 class="title">SARS-CoV-2 genome evolution</h1>
</section>

<section>
	<h3>Phylogeny of 2k viruses sampled globally in June and July 2022</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_revisited_phylogeny.png">
	<div class="citation">
		Following <a href="https://github.com/blab/sars2-adaptive-evo-revisited">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section>

<section>
	<h3>Measure clade growth as a proxy for viral fitness</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_revisited_phylogeny_logistic_growth.png">
	<div class="citation">
		Following <a href="https://github.com/blab/sars2-adaptive-evo-revisited">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section>

<section>
	<h3>Across 2020 to 2023, clades with more S1 mutations grow faster</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_revisited_muts_vs_growth.png">
	<div class="citation">
		Following <a href="https://github.com/blab/sars2-adaptive-evo-revisited">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section>

<section>
	<div class="left" style="float:left; width:35%">
		<h4 class="left-align">Rate of amino acid evolution and correlation between mutation and clade growth strongest in S1</h4>
	</div>
	<div class="right" style="float:right; width:63%">
		<img data-src="/images/talks/sarscov2_adaptive_evolution_revisited_rate_vs_r.png">
	</div>
	<div class="citation">
		Following <a href="https://github.com/blab/sars2-adaptive-evo-revisited">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section>

<section>
	<h3>Strength of adaptive evolution consistently high through time</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_revisited_s1_r_through_time.png">
	<div class="citation">
		Following <a href="https://github.com/blab/sars2-adaptive-evo-revisited">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section>

<!-- <section>
	<h3>dN/dS to root further highlights adaptive evolution in S1</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_adaptive_evolution_revisited_dnds.png">
	<div class="citation">
		Following <a href="https://github.com/blab/sars2-adaptive-evo-revisited">Kistler et al. 2022. Cell Host Microbe.</a>
	</div>
</section> -->

<section>
	<h3>Mutations at spike S1 propel escape from population immunity</h3>
 	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_spike_clock_2023_06_18.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18?branches=hide&l=scatter&scatterY=S1_mutations">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>These mutations are accumulating just as quickly post-Omicron</h3>
 	<img class="stretch" data-src="/images/talks/adaptive_evolution_rates_sars_cov_2.png">
	<div class="citation">
		<a href="https://bedford.io/papers/kistler-atlas-viral-evolution/">Kistler and Bedford. 2023. bioRxiv.</a>
	</div>
</section>

<section>
	<h3>And are accruing much more rapidly than other endemic viruses</h3>
 	<img class="stretch" data-src="/images/talks/adaptive_evolution_rates_comparison.png">
	<div class="citation">
		<a href="https://bedford.io/papers/kistler-atlas-viral-evolution/">Kistler and Bedford. 2023. bioRxiv.</a>
	</div>
</section>

<!-- Variant frequency dynamics -->

<section data-transition="fade" data-background="#61AB9D">
	<h1 class="title">Variant frequency dynamics</h1>
</section>

<section>
	<h3>Population genetic expectation of variant frequency under selection</h3>
	<p>$x' = \frac{x \, (1+s)}{x \, (1+s) + (1-x)}$ for frequency $x$ over one generation with selective advantage $s$</p>
	<p>$x(t) = \frac{x_0 \, (1+s)^t}{x_0 \, (1+s)^t + (1-x_0)}$ for initial frequency $x_0$ over $t$ generations</p>
	<p>Trajectories are linear once logit transformed via $\mathrm{log}(\frac{x}{1 - x})$</p>
	<img class="stretch" src="/images/talks/ncov_variants_selection_logit_trajectories.png">
</section>

<section>
	<h3>Variants show consistent frequency dynamics in logit space</h3>
	<img class="stretch" src="/images/talks/ncov_variants_omicron_countries_frequencies_logit_2022_04_08.png">
</section>

<section>
	<h3>Variants show consistent frequency dynamics in logit space</h3>
	<img class="stretch" src="/images/talks/ncov_variants_ba5_countries_frequencies_logit_2022_08_01.png">
</section>

<section>
	<h3>Multinomial logistic regression</h3>
	<p>
		Multinomial logistic regression across $n$ variants models the probability of a virus sampled at time $t$ belonging to
		variant $i$ as
	</p>
	<p>
		$$\mathrm{Pr}(X = i) = x_i(t) = \frac{p_i \, \mathrm{exp}(f_i \, t)}{\sum_{1 \le j \le n} p_j \, \mathrm{exp}(f_j \, t) }$$
	</p>
	<p>
		with $2n$ parameters consisting of $p_i$ the frequency of variant $i$ at initial timepoint
		and $f_i$ the growth rate or fitness of variant $i$.
	</p>
	<!-- <p>
		The model is fit to minimize "log loss" of predicted variant vs observed variant across observations in
		dataset.
	</p> -->
</section>

<!-- <section>
	<h3>Multinomial logistic regression fits variant frequencies well</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_rt_logistic_regression_2021_09_01.png">
</section> -->

<section>
	<h3>Variant frequencies across countries from Feb 2022 to present</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_mlr_frequencies_2023_06_17.png">
</section>

<section>
	<h3>We find that recent variants like XBB.1.5 are ~300% fitter than original Omicron BA.1</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_mlr_growth_advantages_2023_06_17.png">
</section>

<!-- <section>
	<h3>Population turnover driven by emergence of high fitness variants that displace existing diversity</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_mutational_fitness_2023_06_18.png">
	<div class="citation">
		Model from <a href="https://www.science.org/doi/10.1126/science.abm1208">Obermeyer et al. 2022. Science.</a>
		Figure from <a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18?l=scatter&scatterY=mutational_fitness">nextstrain.org</a>
	</div>
</section> -->

<!-- Evolution driving epidemics -->

<section data-transition="fade" data-background="#94BD61">
	<h1 class="title">Evolution driving epidemics</h1>
</section>

<section>
	<h3>Many fewer reported cases in England post-Omicron</h3>
	<img class="stretch" src="/images/talks/sarscov2_uk_attack_rate_cases.png">
	<div class="citation">
		Data from <a href="https://coronavirus.data.gov.uk/details/cases?areaType=nation&areaName=England">UKHSA</a>
	</div>
</section>

<section>
	<h3>ONS Infection Survey provides rare source of ground truth</h3>
	<p>Roughly 1 in 3 infections detected in 2021, while 1 in 40 in 2023</p>
	<img class="stretch" src="/images/talks/sarscov2_uk_attack_rate_cases_vs_infections.png">
	<div class="citation">
		Data from <a href="https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/datasets/coronaviruscovid19infectionsurveydata">ONS</a>
	</div>
</section>

<section>
	<div class="left" style="float:left; width:28%">
		<h4 class="left-align">Partitioning ONS incidence based on sequencing data shows variant-driven epidemics</h4>
	</div>
	<div class="right" style="float:right; width:70%">
		<img data-src="/images/talks/sarscov2_uk_attack_rate_partitioned_cases.png">
	</div>
	<div class="citation">
		Model from <a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2022. medRxiv.</a>
	</div>
</section>

<section>
	<h3>~110% population attack rate from March 2022 to March 2023</h3>
	<p>~27k deaths in ~61M infections yields IFR of 0.04%</p>
	<img class="stretch" src="/images/talks/sarscov2_uk_attack_rate_cumulative.png">
	<div class="citation">
		Data from <a href="https://coronavirus.data.gov.uk/details/deaths?areaType=nation&areaName=England">UKHSA</a>
		and <a href="https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/datasets/coronaviruscovid19infectionsurveydata">ONS</a>
	</div>
</section>

<!-- Forecasting -->

<!--
- MLR models generate accurate short-term forecasts
- but new variants are appearing too quickly and so prediction threshold is poor
- analogy to weather fronts
- DMS escape prior and improved forecasting horizon
-->

<section data-transition="fade" data-background="#CDB642">
	<h1 class="title">Forecasting</h1>
</section>

<section>
	<h3>Assessing MLR models for short-term frequency forecasting</h3>
	<img class="stretch" src="/images/talks/ncov_forecasting_fit_overview.png">
	<div class="citation">
		<a href="https://github.com/blab/ncov-forecasting-fit">Eslam Abousamra and Marlin Figgins</a>
	</div>
</section>

<section>
	<h3>MLR models generate accurate short-term forecasts</h3>
	<img class="stretch" src="/images/talks/ncov_forecasting_fit_accuracy_window.png">
	<div class="citation">
		<a href="https://github.com/blab/ncov-forecasting-fit">Eslam Abousamra and Marlin Figgins</a>
	</div>
</section>

<section>
	<div class="left" style="float:left; width:35%">
		<h4 class="left-align">Now have clade and lineage forecasts continuously updated</h4>
	</div>
	<div class="right" style="float:right; width:63%">
		<img data-src="/images/talks/ncov_forecasts_clades_2023_06_19.png">
	</div>
	<div class="citation">
		<a href="https://nextstrain.org/sars-cov-2/forecasts/">Figgins et al. nextstrain.org</a>
	</div>
</section>

<!-- Maybe include weather fronts figure -->
<!-- Or a figure that shows series of winner-takes-all forecasts -->
<section class="left-align">
	<h3>Multinomial logistic regression should work well for SARS-CoV-2 prediction, except new variants have been emerging
		fast enough that the prediction horizon is really quite short</h3>
</section>

<section>
	<h3>Could we predict the spread of new mutations using DMS data?</h3>
	<p>Escape from antibodies that potently neutralize BA.2</p>
	<img class="stretch" src="/images/talks/ncov_escape_cao_dms_array_vs_ba2.png">
	<div class="citation">
		<a href="https://www.nature.com/articles/s41586-022-05644-7">Cao et al. 2023. Nature.</a>
	</div>
</section>

<section>
	<h3>Can calculate escape of arbitrary RBD against antibodies known to neutralize BA.2</h3>
	<div class="left" style="float:left; width:58%">
		<img data-src="/images/talks/ncov_escape_calculator_bloom_2023_06_19.png">
	</div>
	<div class="right" style="float:right; width:40%">
		<img data-src="/images/talks/ncov_escape_tree_2023_06_19.png">
	</div>
	<div class="citation-left">
		<a href="https://jbloomlab.github.io/SARS2-RBD-escape-calc/">Greaney, Starr, Bloom. 2022. Virus Evol.</a>
	</div>
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/21L/global/all-time?c=immune_escape">Roemer and Neher. nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Strong correlation between DMS immune escape and lineage-level MLR growth advantage</h3>
	<img class="stretch" src="/images/talks/ncov_escape_binding_escape_growth_advantage.png">
	<div class="citation">
		<a href="https://github.com/blab/ncov-escape">Figgins, Roemer et al.</a>
	</div>
</section>

<!-- Continued evolution -->

<!--
7. Big picture of continued evolution
 - potential for another Omicron-like event
   - based on simple statistical model
   - evidence from persistent infections and cryptic lineages
 - should we expect evolutionary rate to slow as immunity broadens?
 - expectations for continued incidence and burden of disease

8. Research priorities going forward
 - continued genomic surveillance and modeling
 - development of new antivirals
 - vaccination strategies to combat antigenic evolution and OAS
	 - universal vaccines?
   - cocktail vaccines?
	 - reducing imprinted response?
-->

<section data-transition="fade" data-background="#E68133">
	<h1 class="title">Continued evolution</h1>
</section>

<section>
	<h3>How likely are further "Omicron-like" events?</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_scatter_s1_mutations_2023_06_18.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18?l=scatter&scatterY=S1_mutations">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>With 1 observation in 3.6 years of virus evolution, it's currently unclear how common Omicron-like events will be</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_wait_time_distribution_2023_06_20.png">
</section>

<section>
	<h3>This rate distribution gives a naive prediction of ~23% probability of an Omicron-like event occurring in the next 12 months</h3>
	<img class="stretch" data-src="/images/talks/ncov_omicron_prediction_distribution_2023_06_20.png">
</section>

<section>
	<h3>1 in 4 seems high, but "cryptic lineages" are commonly being found in wastewater</h3>
	<img class="stretch" data-src="/images/talks/ncov_cryptic_lineages_ohio_bronx.png">
	<div class="citation">
		<a href="https://twitter.com/SolidEvidence/status/1661491860010045440">Marc Johnson. Twitter</a>
	</div>
</section>

<section>
	<h3>Even without further "saltational" events, ongoing evolution is rapid and so far shows little evidence of slowing</h3>
 	<img class="stretch" data-src="/images/talks/adaptive_evolution_rates_comparison.png">
	<div class="citation">
		<a href="https://bedford.io/papers/kistler-atlas-viral-evolution/">Kistler and Bedford. 2023. bioRxiv.</a>
	</div>
</section>

<section class="left-align">
	<h3>This pace of evolution contributes to large burden of disease at endemicity</h3>
	<p>
	<ul>
		<li>
			Assume yearly attack rate of ~50% (even without the potential for Omicron-like emergence this may be an underestimate)
		</li>
		<li>
			Assume IFR of 0.05% (similar to influenza, compare to early pandemic IFR of 0.5%)
		</li>
		<li>
			This would give somewhere around 80K yearly deaths in the US at endemicity
		</li>
		<li>
			For comparison, this year the US has seen 40k COVID-19 deaths from Jan 1 to Jun 10, which would extrapolate to 90k deaths in 2023
		</li>
	</ul>
	</p>
</section>

<section class="left-align">
	<h3>Research priorities to deal with this rapid evolution</h3>
	<p>
	<ol>
		<li>
			Continued genomic surveillance and evolutionary forecasting
		</li>
		<li>
			Development of new antivirals
		</li>
		<li>
			Vaccination strategies to combat antigenic evolution and OAS
			<ul>
				<li>Universal vaccines?</li>
				<li>Cocktail vaccines?</li>
				<li>Masking epitope sites to reduce imprinted response?</li>
			</ul>
		</li>
	</ol>
	</p>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="even-smaller">John J. Holland Lectureship</p>

	<p class="even-smaller">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID and the Nextstrain team
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola Müller,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jennifer-chang.jpg"> Jennifer Chang,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/allison-li.jpg"> Allison Li,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/eslam-abousamra.jpg"> Eslam Abousamra,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/donna-modrell.jpg"> Donna Modrell,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/nashwa-ahmed.jpg"> Nashwa Ahmed,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg"> Cécile Tran Kiem</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="11%" data-src="/images/talks/logo_fred_hutch_vertical.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
	</div>
</section>
