---
layout: talk_reveal4
title: Data rich phylodynamics
venue: HHMI
event: Science Meeting
author: Trevor Bedford
widescreen: true
draft: true
---

<!-- 20 min total, with 15 min talk and 5 min Q&A -->

<!-- Goal is to give a overview of historical phylogenetic approach to phylodynamics and why this is breaking down with deluge of SARS-CoV-2 genomic data. With only 15 minutes the goal is to give flavor of scalable non-phylogenetic approaches rather real technical detail

Outline:

1. Key insights during pandemic from genomic data
2. Data deluge
3. New approaches for both epidemiology and evolution
4. Identical genomes for estimation of R0 and dispersal (5 min)
5. Lineages and MLR models for estimation of fitness (5 min) -->

<!-- To do:
1. Make an engaging visualization of expanding sequence data -->

<!-- Color ramp: [#5097BA", "#60AA9E", "#75B681", "#8EBC66", "#AABD52", "#C4B945", "#D9AD3D", "#E59637", "#E67030", "#DF4327"] -->

<section data-transition="fade" data-background="#000">
	<h1 class="title">{{ page.title }}</h1>
	<br>&nbsp;<br>
	<h4 class="title">Trevor Bedford</h4>
	<p class="title">
	Fred Hutchinson Cancer Center / Howard Hughes Medical Institute
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
</section>

<section data-background="#000000">
	<h1 class="title">Genomic epidemiology during the COVID-19 pandemic</h1>
</section>

<section>
	<h3>Over 16M SARS-CoV-2 genomes shared to GISAID and evolution tracked in real-time at nextstrain.org</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_map_2021_09_12_global.png">
	<br>
	<p class="tiny"><i>
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/richard-neher.jpg"> Richard Neher,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/ivan-aksamentov.jpg"> Ivan Aksamentov,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/jennifer-chang.jpg"> Jennifer Chang</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/emma-hodcroft.jpg"> Emma Hodcroft,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/cornelius-roemer.jpg"> Cornelius Roemer,</span> &hairsp;
		<span class="nobreak"><img width=30 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley</span>
	</i></p>
</section>

<section style="text-align: left;">
	<h3>Three key insights that genomic epi provided during pandemic</h3>
	<p>
	<ol>
		<li>Rapid human-to-human spread in Wuhan beyond initial market outbreak</li>
		<li>Extensive local transmission while testing was rare</li>
		<li>Identification of variants of concern and mapping of increased transmission rates</li>
	</ol>
	</p>
</section>

<!-- <section>
	<h3>Jan 11: First five genomes showed a novel SARS-like coronavirus</h3>
	<p>
		Initially thought clustering due to epi investigation of linked cases at
		Huanan seafood market
	</p>
	<img class="stretch" data-src="/images/talks/ncov_sars_like_tree.png">
	<div class="citation">
		Data from CAMS, China CDC, Fudan University, WIV;
		Figure from <a href="https://nextstrain.org/groups/blab/sars-like-cov">nextstrain.org</a>
	</div>
</section> -->

<section>
	<h3>Jan 19: First 12 genomes from Wuhan (blue) and Bangkok (red) showed lack of genetic diversity</h3>
	<img class="stretch" data-src="/images/talks/ncov_tree_2020_01_19.png">
	<div class="citation">
		Data from CAMS, China CDC, Fudan University, Hubei CDC, Thai MOPH, WIV;
		Figure from <a href="https://nextstrain.org/ncov/2020-01-22">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>
		Jan 23: Introduction into the human population between Nov 15 and Dec 15 and
		subsequent rapid human-to-human spread
	</h3>
	<img class="stretch" data-src="/images/talks/ncov_tmrca_r0_2020_01_23.png">
	<div class="citation">
		<a href="https://nextstrain.org/narratives/ncov/sit-rep/2020-01-23">Bedford et al. 2020. Technical Report.</a>
	</div>
</section>

<!-- <section>
	<h3>Rapid global epidemic spread from China</h3>
	<video data-autoplay>
		<source data-src="/images/talks/ncov_early_tree_map_animation.mp4" type="video/mp4"/>
	</video>
</section> -->

<section>
	<h3>Epidemic in the USA was introduced from China in late Jan and from Europe during Feb</h3>
	<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_usa_1.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Early sequencing provided best estimate of extent of local outbreak</h3>
	<img class="stretch" src="/images/talks/ncov_cryptic_transmission_outbreak_clade.png">
	<div class="citation">
		<a href="/papers/bedford-ncov-cryptic-transmission/">Bedford et al. 2020. Science.</a>
	</div>
</section>

<section>
	<h4 class="inset-top-right">
		After initial wave, with mitigation <br> efforts and decreased travel, <br> regional clades emerge
	</h4>
	<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_global_jun_oct.png">
	<div class="citation">
		<a href="https://nextstrain.org">nextstrain.org</a>
	</div>
</section>

<!-- Todo: Cut initial frames and add additional final frames -->
<section>
	<h3>Emergence and spread of initial VOC viruses</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_animation_2021_09_12.gif">
</section>

<section>
	<h3>Continued evolution post-Omicron</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_unrooted_2023_06_18.png">
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time/2023-06-18?l=unrooted">nextstrain.org</a>
	</div>
</section>

<section data-background="#000000">
	<h1 class="title">Future of genomic epidemiology</h1>
</section>

<section style="text-align: left;">
	<h3>The COVID-19 pandemic has pushed the field perhaps ~5 years into the future</h3>
	<div style="float: left; width: 80%">
		<br>
		<table style="line-height: 2.5em;">
			<tr>
				<td style="border: none;">2013-16 Ebola in West Africa</td>
				<td style="border: none;">29k confirmed cases</td>
				<td style="border: none;">1610 genomes</td>
			</tr>
			<tr>
				<td style="border: none;">2015-17 Zika in the Americas</td>
				<td style="border: none;">223k confirmed cases</td>
				<td style="border: none;">942 genomes</td>
			</tr>
			<tr>
				<td style="border: none;">2018-19 seasonal flu in US</td>
				<td style="border: none;">290k confirmed cases</td>
				<td style="border: none;">8864 genomes</td>
			</tr>
			<tr>
				<td style="border: none;">2020-22 COVID-19 pandemic</td>
				<td style="border: none;">732M confirmed cases</td>
				<td style="border: none;">14.5M genomes</td>
			</tr>
		</table>
	</div>
	<div style="float: right; width: 14%">
		<img src="/images/talks/ncov_data_quantity.png">
	</div>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 75%">
		<h3>Scable approaches to genomic epidemiology</h3>
		<p>
			<ul>
				<li>Separate threads for epidemiology and evolution</li>
				<ol>
					<li>
						Evolutionary forecasting: focus on seasonal influenza and SARS-CoV-2 for impact on vaccine strain selection
					</li>
					<li>
						Genomic outbreak investigation: across pathogens with focus on spatial dynamics and recontructing spread
					</li>
				</ol>
				<li>Continue to build out the Nextstrain software platform for broader use by the community</li>
			</ul>
		</p>
	</div>
	<div style="float: right; width: 16%">
		<img data-src="/images/talks/genomic_epi_frequency_dynamics.png">
		<br>
		<img data-src="/images/talks/genomic_epi_spatial_dynamics.png">
	</div>
</section>

<section>
	<h3>Clade and lineage forecasts continuously updated</h3>
	<img class="stretch" data-src="/images/talks/ncov_forecasts_clades_2023_08_25.png">
	<div class="citation">
		<a href="https://nextstrain.org/sars-cov-2/forecasts/">Figgins, Lee, Hadfield. nextstrain.org</a>
	</div>
</section>

<section>
	<div class="left" style="float:left; width:35%">
		<h4 class="left-align">Detailed geographic transmission patterns from identical sequences</h4>
	</div>
	<div class="right" style="float:right; width:63%">
		<img data-src="/images/talks/phylo_kernel_spatial_mixing_2023_08_25.png">
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		Tran Kiem et al.
	</div>
</section>

<section data-auto-animate>
	<h3>Phylogeography infers migration matrix via phylogeny</h3>
	<img class="stretch" src="/images/talks/ncov_king_county_mcc_clusters.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/miguel-paredes.jpg">
		<a href="/papers/paredes-sarscov2-king-county/">Paredes et al. 2022. medRxiv.</a>
	</div>
</section>

<section data-auto-animate>
	<h3>Phylogeography infers migration matrix via phylogeny</h3>
	<p>However, this approach faces issues with scalability and with sampling bias</p>
	<img class="stretch" src="/images/talks/ncov_king_county_mcc_clusters.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/miguel-paredes.jpg">
		<a href="/papers/paredes-sarscov2-king-county/"> Paredes et al. 2022. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Identical sequences contain the most signal for transmission linkage</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_mutation_vs_transmission.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		Tran Kiem et al.
	</div>
</section>

<section data-transition="none">
	<h3>Spatial signal readily evident in Washington State clusters of identical sequences</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_example_clusters.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		Tran Kiem et al.
	</div>
</section>

<section data-transition="none">
	<h3>Spatial signal readily evident in Washington State clusters of identical sequences</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_example_clusters.png">
	<div class="citation">
		Tran Kiem et al.
	</div>
</section>

<section>
	<h2>Lay ground-work for near ubiquitous pathogen sequencing</h2>
</section>

<section>
	<h3>Acknowledgements</h3>

	<p class="even-smaller">
		<b>Flu</b>: WHO Global Influenza Surveillance and Response System, other data producers, GISAID, John Huddleston, Richard Neher, Jennifer Chang, Jover Lee
	</p>

	<p class="even-smaller">
		<b>SARS-CoV-2</b>: Data producers from all over the world, GISAID, the Nextstrain team, Katie Kistler, Marlin Figgins, Eslam Abousamra, Jover Lee, James Hadfield
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/cassia-wagner.jpg"> Cassia Wagner,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/nicola-mueller.jpg"> Nicola Müller,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jennifer-chang.jpg"> Jennifer Chang,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/allison-li.jpg"> Allison Li,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/eslam-abousamra.jpg"> Eslam Abousamra,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/donna-modrell.jpg"> Donna Modrell,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/nashwa-ahmed.jpg"> Nashwa Ahmed,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg"> Cécile Tran Kiem,</span>  &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/kim-andrews.jpg"> Kim Andrews</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="10%" data-src="/images/talks/logo_fred_hutch_vertical_v2.png">
		&nbsp; &nbsp;
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp;
		<img width="14%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp;
		<img width="12%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_pew.png">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_cdc.png">
		&nbsp; &nbsp;
		<img width="11%" data-src="/images/talks/logo_wellcome_trust.jpg">
		&nbsp; &nbsp;
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
	</div>
</section>
