---
layout: talk_reveal4
title: Data rich phylodynamics
venue: University of Chicago
event: Cobey Lab Seminar
author: Trevor Bedford
widescreen: true
---

<!-- 1hr timeslot for ~40 min talk and discussion -->

<!-- Goal is to give a overview of historical phylogenetic approach to
phylodynamics and why this is breaking down with deluge of SARS-CoV-2 genomic
data.

Outline:

1. Key insights during pandemic from genomic data
2. Data deluge
3. New approaches for both epidemiology and evolution
4. Identical genomes for estimation of R0 and dispersal (5 min)
5. Lineages and MLR models for estimation of fitness (5 min) -->

<!-- General Todo with specifics inline:
1. Make an engaging visualization of expanding sequence data
  - Run tree with ~30k tips collected in the early pandemic period
	- Plot animation frames as analysis date updates
-->

<!--             Blue     Yellow   Purple   Red     -->
<!-- Color ramp: #4C91C1, #CFB642, #7F62A4, #E04328 -->

<section data-transition="fade" data-background="#000">
	<h1 class="title">{{ page.title }}</h1>
	<br>&nbsp;<br>
	<h4 class="title">Trevor Bedford</h4>
	<p class="title">
	Fred Hutchinson Cancer Center / Howard Hughes Medical Institute
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>&nbsp;<br>
	Slides at: bedford.io/talks
</section>

<!-- This first section should be a brief overview of how genomic data is so
useful early in an outbreak, focusing on SARS-CoV-2, H5N1 and perhaps Zika -->

<section data-background="#4C91C1">
	<h1 class="title">Genomic epidemiology in emerging outbreaks</h1>
</section>

<section>

	<section style="text-align: left;">
		<h3>Three key insights from genomic epi during COVID-19 pandemic</h3>
		<p>
		<ol>
			<li>Rapid human-to-human spread in Wuhan beyond initial market outbreak</li>
			<li>Extensive local transmission while testing was rare</li>
			<li>Identification of variants of concern and mapping of increased transmission rates</li>
		</ol>
		</p>
		<div style="float: left; width:35%">
			<img data-src="/images/talks/ncov_tree_2020_01_19.png">
		</div>
		<div style="float: left; width:29%">
			<img data-src="/images/talks/ncov_cryptic_transmission_introductions.png">
		</div>
		<div style="float: left; width:35%">
			<img data-src="/images/talks/ncov_variants_501Y.V1_clock_2021_02_10.png">
		</div>
	</section>

	<section>
		<h3>Jan 11: First five genomes showed a novel SARS-like coronavirus</h3>
		<p>
			Initially thought clustering due to epi investigation of linked cases at
			Huanan seafood market
		</p>
		<img class="stretch" data-src="/images/talks/ncov_sars_like_tree.png">
		<div class="citation">
			Data from CAMS, China CDC, Fudan University, WIV;
			Figure from <a href="https://nextstrain.org/groups/blab/sars-like-cov">nextstrain.org</a>
		</div>
	</section>

	<section>
		<h3>Jan 19: First 12 genomes from Wuhan (blue) and Bangkok (red) showed lack of genetic diversity</h3>
		<img class="stretch" data-src="/images/talks/ncov_tree_2020_01_19.png">
		<div class="citation">
			Data from CAMS, China CDC, Fudan University, Hubei CDC, Thai MOPH, WIV;
			Figure from <a href="https://nextstrain.org/ncov/2020-01-22">nextstrain.org</a>
		</div>
	</section>

	<section>
		<h3>
			Jan 23: Introduction into the human population between Nov 15 and Dec 15 and
			subsequent rapid human-to-human spread
		</h3>
		<img class="stretch" data-src="/images/talks/ncov_tmrca_r0_2020_01_23.png">
		<div class="citation">
			<a href="https://nextstrain.org/narratives/ncov/sit-rep/2020-01-23">Bedford et al. 2020. Technical report.</a>
		</div>
	</section>

	<section>
		<h3>Rapid global epidemic spread from China</h3>
		<video data-autoplay>
			<source data-src="/images/talks/ncov_early_tree_map_animation.mp4" type="video/mp4"/>
		</video>
	</section>

	<section>
		<h3>Epidemic in the USA was introduced from China in late Jan and from Europe during Feb</h3>
		<img class="stretch" src="/images/talks/ncov_tree_map_2020_04_20_usa_1.png">
		<div class="citation">
			<a href="https://nextstrain.org">nextstrain.org</a>
		</div>
	</section>

	<section>
		<h3>Early sequencing provided best estimate of extent of local outbreak</h3>
		<img class="stretch" src="/images/talks/ncov_cryptic_transmission_outbreak_clade.png">
		<div class="citation">
			<a href="/papers/bedford-ncov-cryptic-transmission/">Bedford et al. 2020. Science.</a>
		</div>
	</section>

	<section>
		<h4 class="inset-top-right">
			After initial wave, with mitigation <br> efforts and decreased travel, <br> regional clades emerge
		</h4>
		<img class="stretch" src="/images/talks/ncov_tree_2020_10_20_global_jun_oct.png">
		<div class="citation">
			<a href="https://nextstrain.org">nextstrain.org</a>
		</div>
	</section>

	<section>
		<h3>Emergence and spread of initial VOC viruses</h3>
		<div class="left" style="float:left; width:49%">
			Alpha
			<img data-src="/images/talks/ncov_variants_501Y.V1_clock_2021_02_10.png">
		</div>
		<div class="right" style="float:right; width:49%">
			Gamma
			<img data-src="/images/talks/ncov_variants_501Y.V3_clock_2021_03_16.png">
		</div>
		<div class="citation-left">
			Alpha described in <a href="https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563">Rambaut et al. 2020. Virological.org</a>
			Figure from <a href="https://nextstrain.org/ncov/europe/2021-02-10?c=clade_membership&f_country=United%20Kingdom&l=clock&r=division">nextstrain.org</a>
		</div>
		<div class="citation">
			Gamma described in <a href="https://www.science.org/doi/10.1126/science.abh2644">Faria et al. 2021. Science.</a>
			Figure from <a href="https://nextstrain.org/ncov/south-america/2021-03-16?f_country=Brazil&l=clock&r=division">nextstrain.org</a>
		</div>
	</section>

</section>

<section>
	<h2>Similar story for other emerging pathogens</h2>
</section>

<section>
	<h3>Zika epidemic in the Americas originated ~1.5 years before initial case detection</h3>
	<img class="stretch" src="/images/talks/zika_zibra_tree.png">
	<div class="citation">
		<a href="/papers/faria-zika-in-brazil/">Faria et al. 2017. Nature.</a>
	</div>
</section>

<section>
	<h3>Mpox epidemic of 2022 stems from human-to-human transmission that began in Nigeria in 2016</h3>
	<img class="stretch" src="/images/talks/mpxv_otoole_h2h_tmrca.png">
	<div class="citation">
		<a href="https://www.science.org/doi/10.1126/science.adg8116">O'Toole et al. 2023. Science.</a>
	</div>
</section>

<section>
	<h3>Ongoing H5N1 epizootic in cattle was spreading for months prior to initial case detection in late March</h3>
	<img class="stretch" data-src="/images/talks/h5n1_cattle_genome_host_2024_09_07.png">
	<div class="citation">
		<a href="https://nextstrain.org/avian-flu/h5n1-cattle-outbreak/genome@2024-09-07?m=num_date">nextstrain.org</a>
	</div>
</section>

<section>
	<h2>After initial lag, we've seen ramp up of molecular diagnostics and sequencing</h2>
</section>

<section style="text-align: left;">
	<h3>The COVID-19 pandemic pushed the field perhaps ~5 years into the future</h3>
	<div style="float: left; width: 80%">
		<br>
		<table style="line-height: 2.5em;">
			<tr>
				<td style="border: none;">2013-16 Ebola in West Africa</td>
				<td style="border: none;">29k confirmed cases</td>
				<td style="border: none;">1610 genomes</td>
			</tr>
			<tr>
				<td style="border: none;">2015-17 Zika in the Americas</td>
				<td style="border: none;">223k confirmed cases</td>
				<td style="border: none;">942 genomes</td>
			</tr>
			<tr>
				<td style="border: none;">2018-19 seasonal flu in US</td>
				<td style="border: none;">290k confirmed cases</td>
				<td style="border: none;">8864 genomes</td>
			</tr>
			<tr>
				<td style="border: none;">2020-22 COVID-19 pandemic</td>
				<td style="border: none;">732M confirmed cases</td>
				<td style="border: none;">14.5M genomes</td>
			</tr>
		</table>
	</div>
	<div style="float: right; width: 14%">
		<img src="/images/talks/ncov_data_quantity.png">
	</div>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 70%">
		<h3>Scalable approaches to genomic epidemiology</h3>
		<p>
		<ul>
			<li>
				<b>Determinants of transmission:</b> Use high volume sequencing datasets to reconstruct transmission dynamics across (spatial) compartments
			</li>
			<li>
				<b>Evolutionary forecasting:</b> Use high volume sequencing datasets to track fine-scale lineage frequencies and infer selective dynamics
			</li>
		</ul>
	</div>
	<div style="float: right; width: 25%">
		<img data-src="/images/talks/genomic_epi_spatial_dynamics_crop.png">
		<br>
		<img data-src="/images/talks/genomic_epi_frequency_dynamics.png">
	</div>
</section>

<section data-background="#CFB642">
	<h1 class="title">Determinants of transmission</h1>
</section>

<section data-auto-animate>
	<h3>Phylogeography infers migration matrix via phylogeny</h3>
	<div class="left" style="float:left; width:68%">
		<img data-src="/images/talks/flu_geo_h3_tree_map.png">
	</div>
	<div class="right" style="float:right; width:28%">
		<img data-src="/images/talks/ncov_king_county_mcc_clusters_vertical.png">
	</div>
	<div class="citation-left">
		<a href="/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/miguel-paredes.jpg">
		<a href="/papers/paredes-sarscov2-king-county/">Paredes et al. 2024. PLoS Pathog.</a>
	</div>
</section>

<section data-auto-animate>
	<h3>Phylogeography infers migration matrix via phylogeny</h3>
	<p>However, this approach faces significant issues with scalability and sampling bias</p>
	<div class="left" style="float:left; width:68%">
		<img data-src="/images/talks/flu_geo_h3_tree_map.png">
	</div>
	<div class="right" style="float:right; width:28%">
		<img data-src="/images/talks/ncov_king_county_mcc_clusters_vertical.png">
	</div>
	<div class="citation-left">
		<a href="/papers/bedford-global-circulation/">Bedford et al. 2015. Nature.</a>
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/miguel-paredes.jpg">
		<a href="/papers/paredes-sarscov2-king-county/">Paredes et al. 2024. PLoS Pathog.</a>
	</div>
</section>

<section>
	<h3>Model emergence, spread and dissolution of clusters of identical sequences</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_tree_diagram.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Identical sequences generally closely linked</h3>
	<p>One mutation every ~13 days vs duration of infection of ~5 days</p>
	<div class="left" style="float:left; width:53%">
		<img data-src="/images/talks/ncov_mutation_rate_nuc_clock_2023_04_15.png">
	</div>
	<div class="right" style="float:right; width:45%">
		<img data-src="/images/talks/phylo_kernel_mutation_vs_transmission.png">
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<!-- For longer version of this talk, include 2-3 slides on tran-kiem-transmission-heterogeneity
1. Logic of R and k from distribution of cluster sizes
2. R and k of SARS-CoV-2 in New Zealand
3. R of SARS-CoV-2 variants through time -->

<section>
	<h3>Spatial and social determinants of transmission from clusters of identical sequences</h3>
	<p>114k SARS-CoV-2 genomes from Washington State sentinel surveillance annotated with <br> geographic location and age</p>
	<img class="stretch" src="/images/talks/phylo_kernel_example_clusters.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Pairs of identical sequences 5.1 times more likely to come from same county</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_within_county.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Relative risk score robust to sampling intensity</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_sampling_bias.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Between county enrichment corresponds well with geography</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_maps.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>East-vs-west split due to geographic barrier in Washington State</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_geo_distance.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Sequence-derived mixing patterns correlate with cell-phone mobility data</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_mobility_correlation.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Between age group enrichment shows social mixing patterns</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_age.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Sequence-derived mixing patterns correlate well with expectations from journaling studies</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_age_correlation.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Signal for differential local vs long-distance age mixing</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_relative_risk_age_by_geography.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section>
	<h3>More methods development to be done, but accuracy compared to traditional
		discrete traits phylogeography is promising</h3>
	<img class="stretch" src="/images/talks/phylo_kernel_dta_comparison_bias.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg">
		<a href="/papers/tran-kiem-sarscov2-phylo-kernel/"> Tran Kiem et al. 2024. medRxiv.</a>
	</div>
</section>

<section class="left-align">
	<h3>Future work</h3>
	<p>
	<ol>
		<li>Incorporate assymetry in transmission direction between groups</li>
		<li>Estimate group-level contributions to epidemic growth</li>
		<li>Apply more broadly</li>
	</ol>
</section>

<section data-background="#7F62A4">
	<h1 class="title">Evolutionary forecasting</h1>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 25%">
		<h4>SARS-CoV-2 evolution since 2020</h4>
	</div>
	<div style="float: right; width: 75%">
		<img data-src="/images/talks/ncov_tree_freq_2024_07_08.png">
	</div>
	<div class="citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time@2024-07-08">nextstrain.org</a>
	</div>
</section>

<section class="left-align">
	<h3>Recent advances</h3>
	<p>
	<ol>
		<li>Granular nomenclature and rapid classification</li>
		<li>Detailed frequency data allowing direct models of selection</li>
	</ol>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 44%">
		<h4>Currently 3328 Pango lineages with rapid assignment of samples to lineages by Nextclade or UShER</h4>
	</div>
	<div style="float: right; width: 54%">
		<img data-src="/images/talks/ncov_nextclade_tree_2024_09_07.png">
	</div>
	<div class="citation">
		<a href="https://nextstrain.org/staging/nextclade/sars-cov-2">Roemer and Neher. nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Variants that are just 1-2 mutations different will get a label</h3>
	<img class="stretch" data-src="/images/talks/ncov_nextclade_tree_kp311_2024_09_07.png">
	<div class="citation">
		<a href="https://nextstrain.org/staging/nextclade/sars-cov-2?label=clade:24C">Roemer and Neher. nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Population genetic expectation of variant frequency under selection</h3>
	<p>$x' = \frac{x \, (1+s)}{x \, (1+s) + (1-x)}$ for frequency $x$ over one generation with selective advantage $s$</p>
	<p>$x(t) = \frac{x_0 \, (1+s)^t}{x_0 \, (1+s)^t + (1-x_0)}$ for initial frequency $x_0$ over $t$ generations</p>
	<p>Trajectories are linear once logit transformed via $\mathrm{log}(\frac{x}{1 - x})$</p>
	<img class="stretch" src="/images/talks/ncov_variants_selection_logit_trajectories.png">
</section>

<section>
	<h3>Consistent frequency dynamics in logit space (BA.2 Mar 2022)</h3>
	<img class="stretch" src="/images/talks/ncov_variants_omicron_countries_frequencies_logit_2022_04_08.png">
</section>

<section>
	<h3>Consistent frequency dynamics in logit space (BA.5 Jul 2022)</h3>
	<img class="stretch" src="/images/talks/ncov_variants_ba5_countries_frequencies_logit_2022_08_01.png">
</section>

<section>
	<h3>Consistent frequency dynamics in logit space (JN.1 Dec 2023)</h3>
	<img class="stretch" src="/images/talks/ncov_variants_jn1_countries_frequencies_logit_2022_12_13.png">
</section>

<section>
	<h3>Multinomial logistic regression</h3>
	<p>
		Multinomial logistic regression across $n$ variants models the probability of a virus sampled at time $t$ belonging to
		variant $i$ as
	</p>
	<p>
		$$\mathrm{Pr}(X = i) = x_i(t) = \frac{p_i \, \mathrm{exp}(f_i \, t)}{\sum_{1 \le j \le n} p_j \, \mathrm{exp}(f_j \, t) }$$
	</p>
	<p>
		with $2n$ parameters consisting of $p_i$ the frequency of variant $i$ at initial timepoint
		and $f_i$ the growth rate or fitness of variant $i$.
	</p>
</section>

<section>
	<h3>Original VOC viruses had substantially increased transmissibility</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_rt_growth_advantage_2021_09_23.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/marlin-figgins.jpg">
		Model from <a href="/papers/figgins-rt-from-frequency-dynamics/">Figgins and Bedford. 2022. medRxiv.</a>
	</div>
</section>

<section>
	<h3>Clade success from 2022 to 2024 driven by antigenic evolution</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_mlr_frequencies_2024_02_01.png">
</section>

<section>
	<h3>We find that recent variants are ~250% fitter than original Omicron BA.1</h3>
	<img class="stretch" src="/images/talks/sarscov2_variant_mlr_growth_advantages_2024_02_01.png">
</section>

<section>
	<h3>Assessing MLR models for short-term frequency forecasting</h3>
	<p>Retrospective projections twice monthly during 2022</p>
	<img class="stretch" src="/images/talks/ncov_forecasting_fit_overview.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/eslam-abousamra.jpg">
		<a href="/papers/abousamra-ncov-forecasting-fit/">Abousamra, Figgins and Bedford. 2024. PLoS Comput Biol.</a>
	</div>
</section>

<section>
	<h3>+30 day short-term forecasts across different countries</h3>
	<img class="stretch" src="/images/talks/ncov_forecasting_fit_projections.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/eslam-abousamra.jpg">
		<a href="/papers/abousamra-ncov-forecasting-fit/">Abousamra, Figgins and Bedford. 2024. PLoS Comput Biol.</a>
	</div>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 30%">
		<h4>MLR models generate accurate short-term forecasts</h4>
		<p>30 days out, countries range from 5 to 15% mean absolute error</p>
	</div>
	<div style="float: right; width: 68%">
		<img data-src="/images/talks/ncov_forecasting_fit_accuracy_window_8_countries.png">
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/eslam-abousamra.jpg">
		<a href="/papers/abousamra-ncov-forecasting-fit/">Abousamra, Figgins and Bedford. 2024. PLoS Comput Biol.</a>
	</div>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 30%">
		<h4>Hierarchical MLR model pools variant fitness estimates across countries</h4>
		<p>This approach improves poor model accuracy in countries with less intensive genomic surveillance</p>
	</div>
	<div style="float: right; width: 68%">
		<img data-src="/images/talks/ncov_forecasting_fit_accuracy_window_8_countries_hier.png">
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/eslam-abousamra.jpg">
		<a href="/papers/abousamra-ncov-forecasting-fit/">Abousamra, Figgins and Bedford. 2024. PLoS Comput Biol.</a>
	</div>
</section>

<section>
	<h3>Clade and lineage forecasts continuously updated at nextstrain.org</h3>
	<p>Rapid sweep of JN.1 over Dec to Jan 2024</p>
	<img class="stretch" src="/images/talks/ncov_forecasts_clades_2024_01_28.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/marlin-figgins.jpg">
		<a href="https://nextstrain.org/sars-cov-2/forecasts/">Figgins et al. nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Clade and lineage forecasts continuously updated at nextstrain.org</h3>
	<p>Serial replacement of descendants of JN.1 with KP.3 the current winner</p>
	<img class="stretch" src="/images/talks/ncov_forecasts_clades_2024_09_09.png">
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/marlin-figgins.jpg">
		<a href="https://nextstrain.org/sars-cov-2/forecasts/">Figgins et al. nextstrain.org</a>
	</div>
</section>

<section>
	<h3>Assess currently circulating lineages by comparing frequency to population weighted growth advantage</h3>
	<img class="stretch" src="/images/talks/ncov_forecasts_lineages_freq_vs_ga_2024_09_11.png">
</section>

<section>
	<h3>Eventual lineage success determined by initial fitness</h3>
	<img class="stretch" src="/images/talks/ncov_forecasts_lineages_freq_vs_ga_2023_11_15_to_2024_01_15.png">
</section>

<section>
	<h3>Eventual lineage success determined by initial fitness</h3>
	<img class="stretch" src="/images/talks/ncov_forecasts_lineages_freq_vs_ga_2024_01_01_to_2024_05_15.png">
</section>

<section>
	<h3>Eventual lineage success determined by initial fitness</h3>
	<img class="stretch" src="/images/talks/ncov_forecasts_lineages_freq_vs_ga_2024_04_14_2024_09_11.png">
</section>

<section class="left-align">
	<h3>Multinomial logistic regression should work well for SARS-CoV-2 prediction, except new variants have been emerging
		fast enough that the prediction horizon is limited to 2-3 months</h3>
</section>

<section>
	<div class="left" style="float:left; width:31%">
		<h4 class="left-align">Ongoing work to lengthen prediction horizon by incorporating high-throughput experimental
		measurements of ACE2 binding and immune escape</h4>
	</div>
	<div class="right" style="float:right; width:67%">
		<img data-src="/images/talks/ncov_escape_dadonaite_delta_growth_vs_dms.png">
	</div>
	<div class="citation">
		<a href="/papers/dadonaite-sarscov2-spike-dms-fitness/">Dadonaite et al. 2024. Nature.</a>
	</div>
</section>

<section style="text-align: left;">
	<div style="float: left; width: 67%">
		<h4>Prediction of variant fitness from empirical priors</h4>
		<p>
			Rather than estimate variant specific fitness $f_i$ directly, we instead
			parameterize as the "innovation" in fitness in going from parent lineage
			$p$ to child lineage $i$ as $\psi_i = (f_i - f_p)$.
		</p>
		<p>
			We then compare a non-informative model of
			$$\psi_i = (f_i - f_p) \sim \mathrm{Normal}(0, \sigma)$$
			to a model where each "innovation" value has an informed prior based on
			a linear combination of predictors such as ACE2 binding, immune escape
			and S1 mutations
			$$\psi_i = (f_i - f_p) \sim \mathrm{Normal}\left(\sum_p \beta_p \, x_p, \sigma\right)$$
		</p>
	</div>
	<div style="float: right; width: 31%">
		<img data-src="/images/talks/ncov_escape_pango_variant_tree.png">
	</div>
	<div class="citation-with-avatar">
		<img width=30 class="img-avatar" src="/images/team/marlin-figgins.jpg">
		Figgins et al. In prep.
	</div>
</section>

<section>
	<h3>Exciting developments in applying protein language models to estimate sequence-level fitness</h3>
	<img class="stretch" src="/images/talks/ncov_escape_ito_model_diagram.png">
	<div class="citation">
		<a href="https://www.biorxiv.org/content/10.1101/2024.03.15.584819v1">Ito et al. 2024. bioRxiv.</a>
	</div>
</section>

<section class="left-align">
	<h3>Continued research</h3>
	<p>
	<ul>
		<li>
			Application of MLR models to seasonal influenza and other pathogens
		</li>
		<li>
			Assessing and improving accuracy of "live" models at nextstrain.org/sars-cov-2/forecasts/
		</li>
		<li>
			Implementing DMS priors to predict fitness of emerging and yet-to-emerge lineages
		</li>
		<li>
			Deep learning approaches to incorporate structural information and avoid classification
		</li>
	</ul>
	</p>
</section>

<section style="text-align: left;">
	<h3>Acknowledgements</h3>

	<p class="tiny">
		<b>SARS-CoV-2 genomic epi</b>: Data producers from all over the world, GISAID, UW Virology, BBI, WA PHL
	</p>
	<p class="tiny">
		<b>Nextstrain</b>: Richard Neher, Ivan Aksamentov, John SJ Anderson, Kim Andrews, Jennifer Chang,
		James Hadfield, Emma Hodcroft, John Huddleston, Jover Lee, Victor Lin, Cornelius Roemer, Thomas Sibley
	</p>
	<p class="tiny">
		<b>Determinants of transmission</b>: Cécile Tran Kiem, Amanda Perofsky,
		Miguel Paredes, Lauren Frisbie, Allison Black, Cécile Viboud
	</p>
	<p class="tiny">
		<b>Evolutionary forecasting</b>: Marlin Figgins, Jover Lee, James Hadfield, John Huddleston,
		Eslam Abousamra, Jesse Bloom, Cornelius Roemer, Richard Neher
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/thomas-sibley.jpg"> Thomas Sibley,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/miguel-paredes.jpg"> Miguel Paredes,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jennifer-chang.jpg"> Jennifer Chang,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/nashwa-ahmed.jpg"> Nashwa Ahmed,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/cecile-tran-kiem.jpg"> Cécile Tran Kiem,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/kim-andrews.jpg"> Kim Andrews,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/cristian-ovadiuc.jpg"> Cristian Ovaduic,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/philippa-steinberg.jpg"> Philippa Steinberg,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/jacob-dodds.jpg"> Jacob Dodds,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/john-sj-anderson.jpg"> John SJ Anderson</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" src="/images/team/amin-bemanian.jpg"> Amin Bemanian</span>
	</p>

	<div class="pull-right" style="vertical-align: bottom; display: table-cell;">
		<img width="12%" data-src="/images/talks/logo_fred_hutch_vertical_v2.png">
		&nbsp; &nbsp; &nbsp; &nbsp;
		<img width="9%" data-src="/images/talks/logo_nih.jpg">
		&nbsp; &nbsp; &nbsp; &nbsp;
		<img width="18%" data-src="/images/talks/logo_bmgf.png">
		&nbsp; &nbsp; &nbsp; &nbsp;
		<img width="15%" data-src="/images/talks/logo_gates_ventures.png">
		&nbsp; &nbsp; &nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_cdc.png">
		&nbsp; &nbsp; &nbsp; &nbsp;
		<img width="10%" data-src="/images/talks/logo_hhmi.jpg">
	</div>
</section>
