---
layout: talk_reveal5
title: Fitness flux in SARS-CoV-2
venue: HHMI
event: Science Meeting
author: Trevor Bedford
draft: true
---

<!-- 20 min total, with 15 min talk and 5 min Q&A -->
<!-- strictly enforcing the 15-minute time limit -->

<!-- Color ramp: -->
<!-- blue     yellow,  purple,  green,   red      orange  -->
<!-- #799BB3, #CEB541, #7F62A4, #96BD60, #DF4327, #E68B35 -->

<!--
Summary: SARS-CoV-2 and seasonal influenza rapidly evolve to escape from human
immunity. This manifests as the emergence and spread of variant viruses. Here I
characterize this process of viral population turnover, quantify the rate of
fitness change in the viral population and relate this to the influx of new
fitness altering mutations.
-->

<!--
Todo:
 - Run a new ncov all-time analysis to update screenshots from
 - Directly plot EvEscape score vs spike mutations
-->

<section data-transition="fade" data-background="#000">
	<h1 class="title">{{ page.title }}</h1>
	<br>
	<div style="float:right; width:20%">
		<img width=55% src="/images/talks/gemini_sarscov2.png">
	</div>
	<h4 class="title left-align">Trevor Bedford</h4>
	<p class="title left-align">
	Fred Hutchinson Cancer Center / Howard Hughes Medical Institute
	<br>
	{{ page.date | date: "%-d %b %Y" }}
	<br>
	{{ page.event }}
	<br>
	{{ page.venue }}
	<br>
	Slides at: bedford.io/talks
</section>

<!-- TODO: Update with latest data -->
<section>
	<h3>Genetic relationships of globally sampled SARS-CoV-2 from 2020 to present</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_tree_unrooted_2025_08_05.png">
	<template class="slide-citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time@2025-08-05?l=unrooted">nextstrain.org</a>
	</template>
</section>

<!-- TODO: Update with latest data -->
<section style="text-align: left;">
	<div style="float: left; width: 40%">
		<h4>Rapid displacement of existing diversity by emerging variants</h4>
	</div>
	<div style="float: right; width: 55%">
		<img data-src="/images/talks/ncov_tree_freq_2025_08_05.png">
	</div>
	<template class="slide-citation-left">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time@2025-08-05">nextstrain.org</a>
	</template>
</section>

<!-- TODO: Update with latest data -->
<section>
	<h3>Mutations in S1 domain of spike protein driving displacement</h3>
	<img class="stretch" data-src="/images/talks/ncov_mutation_rate_spike_clock_2025_08_05.png">
	<template class="slide-citation">
		<a href="https://nextstrain.org/ncov/gisaid/global/all-time@2025-08-05?branches=hide&l=scatter&scatterY=S1_mutations">nextstrain.org</a>
	</template>
</section>

<section>
	<h3>SARS-CoV-2 evolution fast relative to previous endemic viruses</h3>
	<img class="stretch" data-src="/images/talks/adaptive_evolution_rates_comparison_2025_08_01.png">
	<template class="slide-citation">
		<img width=32 class="img-avatar" src="/images/team/katie-kistler.jpg">
		<a href="/papers/kistler-atlas-viral-evolution/">Kistler and Bedford. 2023.</a>
	</template>
</section>

<section class="left-align">
	<h3>This talk</h3>
	<ul>
		<li>Frequency dynamics</li>
		<li>Fitness flux</li>
		<li>Mutational fitness effects</li>
	</ul>
</section>

<!----------------------------------------------->
<!-- Frequency dynamics and fitness estimation -->
<!----------------------------------------------->

<!-- <section data-background="#799BB3">
	<h1 class="title">Frequency dynamics and fitness estimation</h1>
</section> -->

<section data-background="#000">
  <h1 class="title">
    <i class="fa-solid fa-chart-line fa-fw c-blue"></i>
    Frequency dynamics
  </h1>
</section>

<!-- TODO: Maybe remove this slide -->
<section>
	<h3>Population genetic expectation of variant frequency under selection</h3>
	<p>$x' = \frac{x \, (1+s)}{x \, (1+s) + (1-x)}$ for frequency $x$ over one generation with selective advantage $s$</p>
	<p>$x(t) = \frac{x_0 \, (1+s)^t}{x_0 \, (1+s)^t + (1-x_0)}$ for initial frequency $x_0$ over $t$ generations</p>
	<p>Trajectories are linear once logit transformed via $\mathrm{log}(\frac{x}{1 - x})$</p>
	<img class="stretch" data-src="/images/talks/ncov_variants_selection_logit_trajectories.png">
</section>

<!-- TODO: Maybe remove this slide -->
<section>
	<h3>Consistent frequency dynamics in logit space (BA.2 Mar 2022)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_omicron_countries_frequencies_logit_2022_04_08.png">
</section>

<!-- TODO: Maybe remove this slide -->
<section>
	<h3>Consistent frequency dynamics in logit space (JN.1 Dec 2023)</h3>
	<img class="stretch" data-src="/images/talks/ncov_variants_jn1_countries_frequencies_logit_2022_12_13.png">
</section>

<!-- TODO: Maybe update with simple figure and pop gen language -->
<section>
	<h3>Multinomial logistic regression</h3>
	<p>
		Multinomial logistic regression across $n$ variants models the probability of a virus sampled at time $t$ belonging to
		variant $i$ as equal to its frequency $x_i(t)$
	</p>
	<p>
		$$\mathrm{Pr}(X = i) = x_i(t) = \frac{p_i \, \mathrm{exp}(f_i \, t)}{\sum_j p_j \, \mathrm{exp}(f_j \, t) }$$
	</p>
	<p>
		with $2n$ parameters consisting of $p_i$ the frequency of variant $i$ at initial timepoint
		<br> and $f_i$ the growth rate or fitness of variant $i$.
	</p>
</section>

<section>
	<h3>Multinomial logistic regression fits variant frequencies well</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_variant_rt_logistic_regression_2021_09_01.png">
</section>


<!------------------>
<!-- Fitness flux -->
<!------------------>

<!-- <section data-background="#7F62A4">
	<h1 class="title">Fitness flux</h1>
</section> -->

<section data-background="#000">
  <h1 class="title">
    <i class="fa-solid fa-arrows-left-right fa-fw c-blue"></i>
    Fitness flux
  </h1>
</section>

<section>
	<h3>Fitness flux is the rate of change of mean population fitness</h3>
	<p>
		With variant frequency $x_i(t)$ and constant variant fitness $f_i$
	</p>
	<p>
		Mean population fitness $\bar{f}(t) = \sum_i x_i(t) \, f_i$
		&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
		Fitness flux $\phi(t) = \Delta \bar{f}(t) / \Delta t$
	</p>
	<img class="stretch" data-src="/images/talks/mustonen_fitness_flux_diagram.png">
	<template class="slide-citation">
		<a href="https://doi.org/10.1073/pnas.0907953107">Mustonen and Lässig. 2010.</a>
	</template>
</section>

<section data-background-image="/images/talks/ocean_wave.jpg">
</section>

<section>
	<h3>Clade-level frequency dynamics and MLR fits in sliding windows</h3>
	<p>Constant clade fitness within each window, USA data only, ignores within-clade fitness variation</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_clade_frequencies_2025_08_13.png">
</section>

<section>
	<h3>SARS-CoV-2 roughly doubled in fitness every year</h3>
	<p>Line thickness is proportional to variant frequency, 40 total variants</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_clades_fitness_2025_08_13.png">
</section>

<!-- <section>
	<h3>Analogous frequency dynamics and MLR fits for H3N2</h3>
	<p>Constant clade fitness within each window, USA data only, ignores within-clade fitness variation</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_h3n2_clade_frequencies_2025_03_01.png">
</section> -->

<!-- <section>
	<h3>H3N2 roughly doubled in fitness every 10 years</h3>
	<p>Line thickness is proportional to variant frequency, 32 total variants</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_h3n2_clades_fitness_2025_03_01.png">
</section> -->

<!-- <section>
	<h3>Dramatically faster fitness flux of SARS-CoV-2</h3>
	<div style="float: left; width: 49%">
		<img data-src="/images/talks/fitness_dynamics_sarscov2_clades_fitness_narrow_2025_08_13.png">
	</div>
	<div style="float: right; width: 49%">
		<img data-src="/images/talks/fitness_dynamics_h3n2_clades_fitness_narrow_2025_03_01.png">
	</div>
</section> -->

<!----------------------------->
<!-- Traveling fitness waves -->
<!----------------------------->

<!-- <section data-background="#96BD60">
	<h1 class="title">Traveling fitness waves</h1>
</section> -->

<!-- <section data-background="#000">
  <h1 class="title">
    <i class="fa-solid fa-wave-square fa-fw c-green"></i>
    Traveling fitness waves
  </h1>
</section> -->

<section>
	<h3>Traveling wave as viral population increases in fitness</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_clades_location_joyplot_2025_08_13.png">
</section>

<section>
	<h3>Slow down of fitness flux after 2022, but still rapid evolution</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_clades_flux_2025_08_13.png">
</section>

<!-- <section>
	<h3>Multistrain models produce traveling waves in antigenic space</h3>
	<img class="stretch" data-src="/images/talks/canalization_dynamics_grid.png">
	<template class="slide-citation">
		<a href="/papers/bedford-canalization/">Bedford, Rambaut and Pascual. 2012.</a>
	</template>
</section> -->

<section>
	<h3>Many mutations of small effect create traveling fitness waves</h3>
	<p>Richard Neher and others have analytically characterized these waves</p>
	<p>Diffusion constant $D = \mu \, \langle \delta^2 \rangle/2$, where the average $\langle \ldots \rangle$ is over the distribution of mutational effects $K(\delta)$</p>
	<img class="stretch" data-src="/images/talks/neher_fitness_wave_diagram.png">
	<template class="slide-citation">
		<a href="https://doi.org/10.1073/pnas.1213113110">Neher and Hallatschek. 2013.</a>
	</template>
</section>

<!-- <section>
	<h3>Using empirical frequencies and MLR fitness to characterize fitness wave in SARS-CoV-2</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_clades_location_joyplot_2025_08_13.png">
</section> -->

<section>
	<h3>Fitness variance correlates well with fitness flux</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_clades_variance_vs_flux_2025_08_13.png">
</section>

<!-- <section>
	<h3>Fitness variance correlates well with fitness flux</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_h3n2_clades_variance_vs_flux_2025_03_01.png">
</section> -->

<section>
	<h3>This is a specific example of Fisher's fundamental theorem</h3>
	<p>
		"The rate of increase in fitness of any organism at any time is equal to <br> its genetic variance in fitness at that time," ie
		$$\frac{d\bar{f}}{dt} = Var(f)$$
	</p>
	<template class="slide-citation">
		Fisher. 1930.
	</template>
</section>

<!-------------------------------->
<!-- Mutational fitness effects -->
<!-------------------------------->

<!-- <section data-background="#DF4327">
	<h1 class="title">Mutational fitness effects</h1>
</section> -->

<section data-background="#000">
  <h1 class="title">
    <i class="fa-solid fa-square-virus fa-fw c-blue"></i>
    Mutational fitness effects
  </h1>
</section>

<section>
	<h3>Analyze MLR fitness between parent/child lineages</h3>
	<p>Expand to 367 Pango lineages with at least 1000 sequence counts in the US from 2020 to 2025</p>
	<p>Similar concept to Obermeyer et al</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_pango_relationships.png">
</section>

<section>
	<h3>Most Pango branches have 0-1 spike mutations and change log fitness by ±0.1</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_spike_nonspike_fitness_hist_2025_08_13.png">
</section>

<section>
	<h3>Spike mutations tend to increase fitness</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_spike_count_vs_fitness_2025_08_13.png">
</section>

<section>
	<h3>Non-spike mutations do not impact fitness on average</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_non_spike_count_vs_fitness_2025_08_13.png">
</section>

<!-- The S1 subunit contains a receptor-binding domain that recognizes and binds to the
host receptor angiotensin-converting enzyme 2 ACE-2, while the S2 subunit mediates viral
cell membrane fusion by forming a six-helical bundle via the two-heptad repeat domain. -->

<section>
	<h3>Looking across the genome shows that spike is the focus for positive selection, but accessory genes have some signal</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_genome_structure.png">
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_genome_regions_vs_fitness_2025_08_13.png">
</section>

<section>
	<h3>Some attenuation of fitness effects over time</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_genome_regions_vs_fitness_split_2025_08_13.png">
</section>

<!-------------------------------->
<!-- Predicting fitness effects -->
<!-------------------------------->

<!-- <section data-background="#E68B35">
	<h1 class="title">Predicting fitness effects</h1>
</section> -->

<!-- <section data-background="#000">
  <h1 class="title">
    <i class="fa-solid fa-meteor fa-fw c-orange"></i>
    Predicting fitness effects
  </h1>
</section> -->

<section>
	<h3>Simple linear model to combine predictors into a fitness estimate</h3>
	<p>In this case, a very simple model of mutation counts</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_genome_regions_vs_fitness_lm_split_2025_08_13.png">
</section>

<section>
	<h2>Framework to compare predictors of fitness</h2>
</section>

<section>
	<h3>EvEscape combines a variational autoencoder for mutation effect + antibody accessibility + biochemical dissimilarity</h3>
	<img class="stretch" data-src="/images/talks/sarscov2_thadani_evescape.png">
	<template class="slide-citation">
		EvEscape from <a href="https://doi.org/10.1038/s41586-023-06617-0">Thadani et al. 2023.</a>
	</template>
</section>

<!-- TODO: Update to directly plot EvEscape score vs spike mutations ending up with six panels -->
<section>
	<h3>EvEscape does no better than counting spike mutations</h3>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_spike_count_evescape_vs_fitness_2025_08_13.png">
	<template class="slide-citation">
		EvEscape from <a href="https://doi.org/10.1038/s41586-023-06617-0">Thadani et al. 2023.</a>
	</template>
</section>

<section>
	<h3>CoVFit uses spike protein embeddings from ESM-2 to predict MLR fitnesses</h3>
	<img class="stretch" data-src="/images/talks/ncov_escape_ito_model_diagram.png">
	<template class="slide-citation">
		CoVFit from <a href="https://doi.org/10.1038/s41467-025-59422-w">Ito et al. 2025.</a>
	</template>
</section>

<section>
	<h3>CoVFit performs poorly outside of training window</h3>
	<p>Note also the importance of properly assessing independent parent/child lineage deltas</p>
	<img class="stretch" data-src="/images/talks/fitness_dynamics_sarscov2_lineage_delta_spike_count_covfit_vs_fitness_2025_08_13.png">
	<template class="slide-citation">
		CoVFit from <a href="https://doi.org/10.1038/s41467-025-59422-w">Ito et al. 2025.</a>
	</template>
</section>

<!-- <section>
	<div class="left" style="float:left; width:31%">
		<h4 class="left-align">Ongoing work by Bloom Lab to conduct high-throughput experimental
		measurements of ACE2 binding and immune escape</h4>
		<p class="left-align">Challenging to do experiments fast enough to directly improve fitness
			estimates over simple MLR and extrapolation to future backgrounds is non-trivial</p>
	</div>
	<div class="right" style="float:right; width:67%">
		<img data-src="/images/talks/ncov_escape_dadonaite_delta_growth_vs_dms.png">
	</div>
	<template class="slide-citation">
		<a href="/papers/dadonaite-sarscov2-spike-dms-fitness/">Dadonaite et al. 2024.</a>
	</template>
</section> -->

<section style="text-align: left;">
	<h3>
		<dl>
			<dt><i class="fa-solid fa-circle-check c-green"></i></dt>
			<dd>Estimate fitness of circulating lineages</dd>
			&nbsp;
			<dt><i class="fa-solid fa-circle-minus c-yellow"></i></dt>
			<dd>Predict fitness of proximal out-of-sample lineages</dd>
			&nbsp;
			<dt><i class="fa-solid fa-circle-xmark c-red"></i></dt>
			<dd>Predict emergence and spread of out-of-sample lineages</dd>
		</dl>
	</h3>
</section>

<section style="text-align: left;">
	<h3>Acknowledgements</h3>

	<p class="tiny">
		<b>Seasonal influenza and SARS-CoV-2 genomics</b>: Data producers from all over the world, GISAID
	</p>
	<p class="tiny">
		<b>Nextstrain</b>: Richard Neher, Ivan Aksamentov, John SJ Anderson, Kim Andrews, Jennifer Chang,
		James Hadfield, Emma Hodcroft, John Huddleston, Jover Lee, Victor Lin, Cornelius Roemer, Thomas Sibley
	</p>
	<p class="tiny">
		<b>MLR and fitness modeling</b>: Marlin Figgins, Eslam Abousamra, Jover Lee, James Hadfield,
			John Huddleston, Philippa Steinberg, Jesse Bloom, Cornelius Roemer, Richard Neher
	</p>

	<p class="tiny"><b>Bedford Lab</b>:
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/john-huddleston.jpg"> John Huddleston,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/james-hadfield.jpg"> James Hadfield,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/katie-kistler.jpg"> Katie Kistler,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/jover-lee.jpg"> Jover Lee,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/marlin-figgins.jpg"> Marlin Figgins,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/victor-lin.jpg"> Victor Lin,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/jennifer-chang.jpg"> Jennifer Chang,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/nashwa-ahmed.jpg"> Nashwa Ahmed,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/cecile-tran-kiem.jpg"> Cécile Tran Kiem,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/kim-andrews.jpg"> Kim Andrews,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/philippa-steinberg.jpg"> Philippa Steinberg,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/jacob-dodds.jpg"> Jacob Dodds,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/amin-bemanian.jpg"> Amin Bemanian,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/carlos-avendano.jpg"> Carlos Avendano,</span> &hairsp;
		<span class="nobreak"><img width=38 class="img-avatar" data-src="/images/team/aayush-verma.jpg"> Aayush Verma</span>
	</p>

	<div width="100%" style="display: flex; justify-content: space-around; align-items: center;">
		<img width="10%" data-src="/images/talks/logo_fred_hutch_vertical_v2.png">
		<img width="7%" data-src="/images/talks/logo_nih.jpg">
		<img width="8%" data-src="/images/talks/logo_cdc.png">
		<img width="8%" data-src="/images/talks/logo_hhmi.jpg">
	</div>

</section>
