---
layout: paper
title: Timely vaccine strain selection and genomic surveillance improve evolutionary forecast accuracy of seasonal influenza A/H3N2
image: /images/papers/huddleston-delays.png
authors: Huddleston J, Bedford T.
year: 2025
ref: Huddleston and Bedford. 2025. eLife.
journal: "eLife 14: RP104282."
pdf: /pdfs/papers/huddleston-delays.pdf
supplement: /pdfs/papers/huddleston-delays-supp.pdf
doi: 10.7554/eLife.104282
github: https://github.com/blab/flu-forecasting-delays
---

# Abstract

Evolutionary forecasting models inform seasonal influenza vaccine design by predicting which current genetic variants will dominate in the influenza season 12 months later.
Forecasting models depend on hemagglutinin sequences from global public health networks to identify current genetic variants (clades) and estimate clade fitnesses.
The lag between collection of a clinical sample and public availability of its sequence averages ~3 months, complicating the 12-month forecasting problem by reducing our understanding of current clade frequencies.
Despite continued methodological improvements to forecasting models, these constraints of a 12-month forecast horizon and 3-month submission lags impose an upper bound on any model's accuracy.
The SARS-CoV-2 pandemic revealed that modern vaccine technology reduces forecast horizons to 6 months and expanded sequencing support reduces submission lags to 1 month on average.
We quantified the potential effects of these public health policy changes on forecast accuracy for A/H3N2 populations.
Reducing forecast horizons to 6 months reduced average absolute forecasting errors to 25% of the 12-month average, while reducing submission lags decreased uncertainty in current clade frequencies by 50%.
These results show the potential to improve the accuracy of existing forecasting models through realistic changes to public health policy.
