---
layout: paper
title: Quantifying plasmid movement in drug-resistant Shigella species using phylodynamic inference
image: /images/papers/muller-plasmid-phylogenetic-networks.png
authors: Müller NF, Wick RR, Judd LM, Williamson DA, Bedford T, Howden B, Duchêne S, Ingle DJ
year: 2025
ref: Müller et al. 2025. bioRxiv.
journal: "bioRxiv: 2022.10.27.514108."
pdf: /pdfs/papers/muller-plasmid-phylogenetic-networks.pdf
doi: 10.1101/2022.10.27.514108
github: https://github.com/nicfel/Plasmids-Material
---

# Abstract

The "silent pandemic" of antimicrobial resistance (AMR) represents a significant global public health threat. AMR genes in bacteria are often carried on mobile elements, such as plasmids. The horizontal movement of plasmids allows AMR genes and resistance to key therapeutics to disseminate in a population. However, the quantification of the movement of plasmids remains challenging with existing computational approaches. Here, we introduce a novel method that allows us to reconstruct and quantify the movement of plasmids in bacterial populations over time. To do so, we model chromosomal and plasmid DNA co-evolution using a joint coalescent and plasmid transfer process in a Bayesian phylogenetic network approach. This approach reconstructs differences in the evolutionary history of plasmids and chromosomes to reconstruct instances where plasmids likely move between bacterial lineages while accounting for parameter uncertainty. We apply this new approach to a five-year dataset of _Shigella_, exploring the plasmid transfer rates of five different plasmids with different AMR and virulence profiles. In doing so, we reconstruct the co-evolution of the large _Shigella_ virulence plasmid with the chromosome DNA. We quantify higher plasmid transfer rates of three small plasmids that move between lineages of _Shigella sonnei_. Finally, we determine the recent dissemination of a multidrug-resistant plasmid between _S. sonnei_ and _S. flexneri_ lineages in multiple independent events and through steady growth in prevalence since 2010. This approach has a strong potential to improve our understanding of the evolutionary dynamics of AMR-carrying plasmids as they are introduced, circulate, and are maintained in bacterial populations.
