---
layout: paper
title: Nomenclature for tracking of genetic variation of seasonal influenza viruses
image: /images/papers/neher-seasonal-flu-nomenclature.png
authors: Neher RA, Huddleston J, Bedford T, Lewis NS, Harvey R, Galiano M, Byrne AMP, James S, Smith D, Łuksza M, Ruchnewitz D, Lässig M, Fujisaki S, Watanabe S, Hasegawa H, Hassell N, Wentworth DE, Kondor R, Deng Y-M, Dapat C, Subbarao K, Barr I.
year: 2026
ref: Neher et al. 2026. Influenza Other Respir Viruses.
journal: "Influenza Other Respir Viruses 20 :e70230."
pdf: /pdfs/papers/neher-seasonal-flu-nomenclature.pdf
doi: 10.1111/irv.70230
github: https://github.com/influenza-clade-nomenclature
---

# Abstract

**Background:** Genomic surveillance of human seasonal influenza viruses is an essential component of the Global Influenza Surveillance and Response system (GISRS) and informs the recommendations for the seasonal influenza vaccine composition. Phylogenetic analysis of viral genome sequences is used to identify groups of viruses sharing potential antigenic change and computational models are used to predict which viral variants are likely to circulate at high levels in upcoming seasons. To facilitate discussion and reporting of genetic diversity, as well as to communicate antigen recommendations, up-to-date and sufficiently granular definitions of genetic clades are important.

**Methods:** A nomenclature system for segments 4 (haemagglutinin) and 6 (neuraminidase) of human A(H3N2), A(H1N1)pdm09, and influenza B.

**Results:** We devised a clade suggestion algorithm that proposes new subclades based on criteria including (i) the number of sequences in the group, (ii) the divergence from the directly ancestral clade, and (iii) the number and quality of amino acid substitutions on the branch leading to the common ancestor of the subclade. Algorithmic clade proposals were reviewed and assigned a systematic hierarchical label consisting of a leading letter, followed by numbers (e.g., G.1.3). Names are kept short by aliasing, that is collapsing prefixes into unique letters. Subclade definitions are shared openly to promote adoption and tool development. Nextclade is supporting this new nomenclature and it is being used routinely by the GISRS network.

**Conclusions:** With increasing genomic surveillance, the need for up-to-date classification schemes is growing and we hope that the current dynamic proposal will adapt to growing data volumes and aid in simplifying the interpretation of these data.
