---
layout: post
title: Full-stack developer and bioinformatician for an open-source genomic epidemiology research platform
author: Thomas Sibley
---

Positions for a __full-stack developer__ and __bioinformatician__ are available immediately in the Bedford lab at the Fred Hutch to work on an open-source genomic epidemiology research platform enabling a large-scale study of respiratory illness in Seattle.

In collaboration with groups at the Fred Hutch, the University of Washington, Seattle Children's, the Institute for Disease Modeling, and area hospitals, we're embarking on a high-resolution, multi-year study of influenza and other respiratory illnesses in Seattle.
Through the study thousands of influenza and other respiratory pathogens will be sampled and sequenced in near-real time and from these viral genome sequences, transmission dynamics uncovered.
Initial work this year will set the course for future years.
The primary task of both positions will be to develop the information and analysis systems underlying the study's research aims, with potential expansion to new studies around the world in the future.
This new platform will build upon the software behind [Nextstrain](https://nextstrain.org), an award-winning tool for tracking infectious disease epidemics developed in collaboration with the [Neher lab](https://neherlab.org/) at the University of Basel.
The platform will ingest subject and sample metadata, lab assay results, and raw and processed genome data as well as provide access to this curated data to streamline downstream analysis.
This platform will be open-source, adaptable, and designed with future outbreak surveillance in mind, regardless of if the targeted pathogen is viral, bacterial, or eukaryotic in nature.
On top of this data warehousing platform, we will deploy analytic pipelines that align sequences, build phylogenies and reconstruct city-scale transmission chains.

The ideal candidate for the __full-stack developer__ position would have expertise in web development, relational database systems, cloud infrastructure, and software engineering best practices.
Experience working with genomic data or in systems integration is a plus but not a requirement.

The ideal candidate for the __bioinformatician__ position would have expertise in genomics, molecular biology, pipeline automation, and software development practices.
Experience working with cloud infrastructure and web technology is a plus but not a requirement.

Candidates for both positions should be fluent in at least one high-level programming language, such as Python, JavaScript, Ruby, or Perl.
Candidates should also have excellent written and verbal communication skills.
Interfacing with project collaborators in-person and online is a key aspect of both positions.
Both positions will work within a small team of existing members of the Bedford lab.

The [Fred Hutch](https://fredhutch.org) is located in South Lake Union in Seattle, WA and offers a dynamic work environment with cutting-edge science and computational resources.
The position is available immediately with flexible starting dates.
Informal inquires are welcome.
Applications will be accepted until the position is filled.
We offer a competitive salary commensurate with skills and experience, along with benefits.
The Fred Hutch and the Bedford lab are committed to improving diversity in the computational sciences.
Applicants of diverse backgrounds are particularly encouraged to apply.
Depending on the applicant, this position could be a full-time salaried employee, a part-time employee, or a contracted consultant.

For more information about the lab, please see our website at [bedford.io](http://bedford.io).
To apply for the position please send (a) your current resume, (b) code samples or links to published/distributed code you've written, and (c) contact information for two references to [tsibley@fredhutch.org](mailto:tsibley@fredhutch.org).
