---
layout: post
title: Preliminary analysis of the US H5N1 cattle outbreak
authors:
  - Trevor Bedford
  - James Hadfield
  - Louise Moncla
date: 2024-04-29
---

The 2.3.4.4b clade of influenza subtype H5N1 has been remarkably successful in its global spread with arrival into Europe in mid-2020, into North America in late 2021 and arrival into South America in mid-2022 (Fig. 1).
Although sustained transmission and spread has been largely within birds, these viruses have infected a number of species of non-human mammals, including [foxes, skunks and domestic cats](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=species_group&f_species_group=Mammal-%20Terrestrial&r=division&transmissions=hide) which appear to be dead-end hosts to infection, but also [seals in North America](https://nextstrain.org/groups/moncla-lab/h5nx/north-america/ha?c=species_group&f_species_group=Mammal-%20Marine&r=division&transmissions=hide) and [sea lions in South America](https://www.nature.com/articles/s41467-023-41182-0), where sustained transmission outside of birds is highly likely.
More recently, this clade of H5N1 has been discovered circulating in dairy cows in the US with cases [first reported in Texas in March 25, 2024](https://www.aphis.usda.gov/news/agency-announcements/federal-state-veterinary-public-health-agencies-share-update-hpai).
On April 21, the [first large tranche of genomic data from the outbreak was shared](https://www.aphis.usda.gov/livestock-poultry-disease/avian/avian-influenza/usda-publishes-h5n1-influenza-virus-genetic) by the APHIS National Veterinary Services lab at the USDA.
However, these data were shared as raw sequencing reads rather than the standard assembled consensus genomes.
Karthik Gangavarapu, Mike Worobey and colleagues assembled these sequences and shared consensus genomes to [GitHub](https://github.com/andersen-lab/avian-influenza/tree/master).
Notably, these genomes still lack critical metadata on date of collection and location of collection that are generally needed to support phylodynamic inferences.

We combined these USDA sequences with other H5N1 sequences in GISAID to analyze the outbreak.
This analysis is visible as 
