---
layout: member
title: Kim Andrews
position: Bioinformatician
github: kimandrews
scholar: M2_G2OQAAAAJ
image: /images/team/kim-andrews.jpg
website: https://kimberlyandrews.net/
joined: 2023
departed: 2025
alumni: true
---

I joined the lab in December 2023 as a Bioinformatician. I obtained my PhD in 2009 from the University of Hawaiâ€˜i specializing in ecology and evolution, and I have broad interest in using genomic tools to explore evolutionary questions. Previously I held research and bioinformatics positions at Durham University, University of Idaho, and University of Washington.

Most of my research has focused on using population genetic and phylogenetic techniques to investigate the ecological and evolutionary forces driving population structure, adaptation, and speciation across a wide taxonomic range of species. For example, I have used whole genome sequence data to investigate transmission dynamics of [SARS-CoV-2 in a university town](https://www.nature.com/articles/s41598-023-34625-7) and adaptive variation within and between host species in a [bacterial pathogen of sheep and goats](https://github.com/kimandrews/Movi) (*Mycoplasma ovipneumoniae*). I have also conducted studies characterizing the population genetic structure of [dolphins](https://onlinelibrary.wiley.com/doi/full/10.1111/j.1365-294X.2010.04521.x), [copepods](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.12950), [marine fish](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0091665), and [beetles](https://www.nature.com/articles/s42003-020-01169-9); investigating the genomic basis of adaptation in [dolphins](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.15865) and [freshwater fish](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.16810); characterizing ancestry for [wolves, bighorn sheep](https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.12910), and [laboratory mice](https://link.springer.com/article/10.1186/s12864-021-07698-9); and using phylogenetic inference to identify new species of [fish](https://www.sciencedirect.com/science/article/pii/S1055790316300380), [beetles](https://www.nature.com/articles/s42003-020-01169-9), and [copepods](https://onlinelibrary.wiley.com/doi/full/10.1111/mec.12950). I have experience working with a wide range of genomic data types, including shotgun sequencing, [RADseq](https://www.nature.com/articles/nrg.2015.28), RNAseq, and amplicon sequencing data generated with Illumina, PacBio, and Oxford Nanopore technologies. I was also the primary instructor for a graduate-level Genomics workshop and an undergraduate-level Marine Biology lab, and co-instructor for graduate-level courses in Conservation Genomics and Landscape Genetics.
